<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=vzxKn6nZtIGKCOgTKQG8-3e6S71Rvb7XAk8J16JjOPI);.lst-kix_obdmgy1vscwr-2>li:before{content:"\0025a0   "}.lst-kix_obdmgy1vscwr-1>li:before{content:"\0025cb   "}.lst-kix_obdmgy1vscwr-3>li:before{content:"\0025cf   "}.lst-kix_obdmgy1vscwr-0>li:before{content:"\0025cf   "}.lst-kix_obdmgy1vscwr-4>li:before{content:"\0025cb   "}.lst-kix_obdmgy1vscwr-6>li:before{content:"\0025cf   "}.lst-kix_obdmgy1vscwr-5>li:before{content:"\0025a0   "}.lst-kix_obdmgy1vscwr-7>li:before{content:"\0025cb   "}ul.lst-kix_de1lyxmd30ln-8{list-style-type:none}ul.lst-kix_j2ztfdahgekg-4{list-style-type:none}ul.lst-kix_j2ztfdahgekg-5{list-style-type:none}ul.lst-kix_j2ztfdahgekg-6{list-style-type:none}ul.lst-kix_j2ztfdahgekg-7{list-style-type:none}ul.lst-kix_j2ztfdahgekg-0{list-style-type:none}ul.lst-kix_j2ztfdahgekg-1{list-style-type:none}ul.lst-kix_j2ztfdahgekg-2{list-style-type:none}ul.lst-kix_j2ztfdahgekg-3{list-style-type:none}ul.lst-kix_de1lyxmd30ln-0{list-style-type:none}.lst-kix_j2ztfdahgekg-8>li:before{content:"\0025a0   "}ul.lst-kix_de1lyxmd30ln-1{list-style-type:none}ul.lst-kix_obdmgy1vscwr-7{list-style-type:none}ul.lst-kix_de1lyxmd30ln-2{list-style-type:none}ul.lst-kix_obdmgy1vscwr-8{list-style-type:none}ul.lst-kix_de1lyxmd30ln-3{list-style-type:none}ul.lst-kix_obdmgy1vscwr-5{list-style-type:none}ul.lst-kix_de1lyxmd30ln-4{list-style-type:none}ul.lst-kix_j2ztfdahgekg-8{list-style-type:none}ul.lst-kix_obdmgy1vscwr-6{list-style-type:none}ul.lst-kix_de1lyxmd30ln-5{list-style-type:none}ul.lst-kix_obdmgy1vscwr-3{list-style-type:none}ul.lst-kix_de1lyxmd30ln-6{list-style-type:none}ul.lst-kix_obdmgy1vscwr-4{list-style-type:none}ul.lst-kix_de1lyxmd30ln-7{list-style-type:none}.lst-kix_d1go8u71kpqj-6>li:before{content:"\0025cf   "}.lst-kix_d1go8u71kpqj-7>li:before{content:"\0025cb   "}.lst-kix_d1go8u71kpqj-8>li:before{content:"\0025a0   "}.lst-kix_obdmgy1vscwr-8>li:before{content:"\0025a0   "}.lst-kix_w5ja0u4v7ehl-1>li:before{content:"\0025cb   "}.lst-kix_w5ja0u4v7ehl-0>li:before{content:"\0025cf   "}.lst-kix_de1lyxmd30ln-0>li:before{content:"\0025cf   "}.lst-kix_7f1aadtxzvwv-5>li:before{content:"\0025a0   "}.lst-kix_7f1aadtxzvwv-7>li:before{content:"\0025cb   "}.lst-kix_de1lyxmd30ln-1>li:before{content:"\0025cb   "}.lst-kix_7f1aadtxzvwv-4>li:before{content:"\0025cb   "}.lst-kix_7f1aadtxzvwv-8>li:before{content:"\0025a0   "}ul.lst-kix_d1go8u71kpqj-8{list-style-type:none}.lst-kix_de1lyxmd30ln-3>li:before{content:"\0025cf   "}ul.lst-kix_d1go8u71kpqj-7{list-style-type:none}.lst-kix_de1lyxmd30ln-2>li:before{content:"\0025a0   "}.lst-kix_de1lyxmd30ln-4>li:before{content:"\0025cb   "}.lst-kix_w5ja0u4v7ehl-2>li:before{content:"\0025a0   "}.lst-kix_w5ja0u4v7ehl-3>li:before{content:"\0025cf   "}.lst-kix_7f1aadtxzvwv-6>li:before{content:"\0025cf   "}.lst-kix_w5ja0u4v7ehl-4>li:before{content:"\0025cb   "}ul.lst-kix_d1go8u71kpqj-0{list-style-type:none}.lst-kix_de1lyxmd30ln-7>li:before{content:"\0025cb   "}ul.lst-kix_d1go8u71kpqj-2{list-style-type:none}.lst-kix_de1lyxmd30ln-6>li:before{content:"\0025cf   "}.lst-kix_de1lyxmd30ln-8>li:before{content:"\0025a0   "}ul.lst-kix_d1go8u71kpqj-1{list-style-type:none}ul.lst-kix_d1go8u71kpqj-4{list-style-type:none}.lst-kix_de1lyxmd30ln-5>li:before{content:"\0025a0   "}ul.lst-kix_d1go8u71kpqj-3{list-style-type:none}.lst-kix_w5ja0u4v7ehl-5>li:before{content:"\0025a0   "}ul.lst-kix_d1go8u71kpqj-6{list-style-type:none}ul.lst-kix_d1go8u71kpqj-5{list-style-type:none}.lst-kix_7f1aadtxzvwv-0>li:before{content:"\0025cf   "}.lst-kix_w5ja0u4v7ehl-8>li:before{content:"\0025a0   "}.lst-kix_7f1aadtxzvwv-1>li:before{content:"\0025cb   "}.lst-kix_7f1aadtxzvwv-3>li:before{content:"\0025cf   "}.lst-kix_w5ja0u4v7ehl-6>li:before{content:"\0025cf   "}.lst-kix_w5ja0u4v7ehl-7>li:before{content:"\0025cb   "}.lst-kix_7f1aadtxzvwv-2>li:before{content:"\0025a0   "}ul.lst-kix_7f1aadtxzvwv-8{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-4{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-5{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-6{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-7{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-0{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-1{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-2{list-style-type:none}ul.lst-kix_7f1aadtxzvwv-3{list-style-type:none}.lst-kix_d1go8u71kpqj-5>li:before{content:"\0025a0   "}.lst-kix_d1go8u71kpqj-4>li:before{content:"\0025cb   "}ul.lst-kix_w5ja0u4v7ehl-5{list-style-type:none}ul.lst-kix_obdmgy1vscwr-1{list-style-type:none}ul.lst-kix_w5ja0u4v7ehl-6{list-style-type:none}ul.lst-kix_obdmgy1vscwr-2{list-style-type:none}.lst-kix_j2ztfdahgekg-4>li:before{content:"\0025cb   "}.lst-kix_j2ztfdahgekg-5>li:before{content:"\0025a0   "}ul.lst-kix_w5ja0u4v7ehl-3{list-style-type:none}ul.lst-kix_w5ja0u4v7ehl-4{list-style-type:none}ul.lst-kix_obdmgy1vscwr-0{list-style-type:none}.lst-kix_d1go8u71kpqj-1>li:before{content:"\0025cb   "}.lst-kix_d1go8u71kpqj-3>li:before{content:"\0025cf   "}.lst-kix_j2ztfdahgekg-3>li:before{content:"\0025cf   "}ul.lst-kix_w5ja0u4v7ehl-7{list-style-type:none}.lst-kix_j2ztfdahgekg-7>li:before{content:"\0025cb   "}ul.lst-kix_w5ja0u4v7ehl-8{list-style-type:none}.lst-kix_d1go8u71kpqj-2>li:before{content:"\0025a0   "}.lst-kix_j2ztfdahgekg-6>li:before{content:"\0025cf   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_d1go8u71kpqj-0>li:before{content:"\0025cf   "}.lst-kix_j2ztfdahgekg-0>li:before{content:"\0025cf   "}.lst-kix_j2ztfdahgekg-1>li:before{content:"\0025cb   "}ul.lst-kix_w5ja0u4v7ehl-1{list-style-type:none}ul.lst-kix_w5ja0u4v7ehl-2{list-style-type:none}.lst-kix_j2ztfdahgekg-2>li:before{content:"\0025a0   "}ul.lst-kix_w5ja0u4v7ehl-0{list-style-type:none}ol{margin:0;padding:0}table td,table th{padding:0}.c16{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:234pt;border-top-color:#000000;border-bottom-style:solid}.c30{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:58.5pt;border-top-color:#000000;border-bottom-style:solid}.c26{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:158.2pt;border-top-color:#000000;border-bottom-style:solid}.c14{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:309.8pt;border-top-color:#000000;border-bottom-style:solid}.c0{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:156pt;border-top-color:#000000;border-bottom-style:solid}.c31{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:253.5pt;border-top-color:#000000;border-bottom-style:solid}.c11{padding-top:20pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c36{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c39{padding-top:18pt;padding-bottom:6pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c37{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c34{padding-top:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c29{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c1{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c4{color:#000000;text-decoration:none;vertical-align:baseline;font-size:12pt;font-style:normal}.c3{color:#000000;text-decoration:none;vertical-align:baseline;font-size:11pt;font-style:normal}.c18{color:#000000;text-decoration:none;vertical-align:baseline;font-size:20pt;font-style:normal}.c5{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c21{color:#000000;text-decoration:none;vertical-align:baseline;font-style:normal}.c19{color:#000000;text-decoration:none;vertical-align:baseline;font-size:11pt}.c24{border-spacing:0;border-collapse:collapse;margin-right:auto}.c20{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c15{padding-top:0pt;padding-bottom:0pt;line-height:1.5;text-align:left}.c22{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:center}.c44{text-decoration:none;vertical-align:baseline;font-style:normal}.c23{color:#000000;text-decoration:none;vertical-align:baseline}.c32{margin-left:36pt;text-indent:-36pt}.c49{max-width:468pt;padding:72pt 72pt 72pt 72pt}.c12{padding:0;margin:0}.c17{margin-left:36pt;padding-left:0pt}.c27{background-color:#ffffff;color:#212121}.c2{font-weight:400;font-family:"Manrope"}.c41{background-color:#ffffff;font-size:11pt}.c25{font-weight:700;font-family:"Manrope"}.c10{color:inherit;text-decoration:inherit}.c35{color:#212121}.c50{height:23.5pt}.c45{text-indent:36pt}.c42{font-size:20pt}.c40{vertical-align:super}.c46{height:20pt}.c9{font-style:italic}.c38{font-size:10.5pt}.c28{font-size:14pt}.c47{font-size:16pt}.c13{height:0pt}.c6{height:11pt}.c33{page-break-after:avoid}.c43{font-size:11pt}.c48{background-color:#ffffff}.c8{font-size:12pt}.c7{font-size:10pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c48 c49 doc-content"><h1 class="c11" id="h.uvar74b8hqkb"><span class="c18 c2">X-DNA Formation and Plasmid Linearization</span></h1><h2 class="c39 c33" id="h.6q1rvaevms7k"><span class="c21 c2 c47">Abbreviations Used</span></h2><ul class="c12 lst-kix_7f1aadtxzvwv-0 start"><li class="c1 c17 li-bullet-0"><span class="c4 c2">TE: Tris-EDTA Buffer</span></li><li class="c1 c17 li-bullet-0"><span class="c4 c2">TAE: Tris-Acetate-EDTA Buffer</span></li><li class="c1 c17 li-bullet-0"><span class="c4 c2">MWCO: Molecular Weight Cut-Off</span></li><li class="c1 c17 li-bullet-0"><span class="c4 c2">PNK: Polynucleotide Kinase</span></li><li class="c1 c17 li-bullet-0"><span class="c4 c2">ATP: Adenosine Triphosphate</span></li><li class="c1 c17 li-bullet-0"><span class="c4 c2">PCR: Polymerase Chain Reaction</span></li><li class="c1 c17 li-bullet-0"><span class="c4 c2">Bp: Base pairs</span></li></ul><h1 class="c11" id="h.b1eag0axuri7"><span class="c18 c2">Introduction</span></h1><p class="c1"><span class="c4 c2">The foundational monomers and building blocks of our DNA-based hydrogel are linearized plasmids containing the gene of interest and X-DNA units. The linearized plasmid serves as the template for protein expression. The plasmid was linearized to facilitate ligation into the hydrogel and enhanced accessibility for transcriptional machinery. Following linearization, the plasmid was dephosphorylated to prevent self-ligation. When needed, it was rephosphorylated to restore 5&rsquo; phosphate groups for compatibility with crosslinking. X-DNA is a synthetic DNA nanostructure composed of four single-stranded oligonucleotides that hybridize into a four-way junction due to the complementarity of their sequences. These junctions act as structural nodes that interconnect plasmid molecules through sequence-specific base pairing. As a result, a mechanically stable and transcriptionally active gel network was formed, capable of supporting cell-free protein expression within a confined 3D matrix.</span></p><h1 class="c11" id="h.7w6cwiy5w1ag"><span class="c2">Techniques</span></h1><p class="c1"><span class="c4 c25">Restriction enzyme digestion</span></p><p class="c1"><span class="c4 c2">Restriction digestion is a process that uses restriction endonucleases to cut DNA at specific sites. This was performed using XmaI, which cuts at a unique sequence within the plasmid encoding carbonic anhydrase. This converts the circular plasmid to a linear form, which is more suitable for incorporation into the DNA hydrogel (Park et al. 2009). A key reason for linearization is that it reduces the likelihood of unwanted topological effects, such as supercoiling, and makes hybridization with DNA crosslinkers more predictable. The reaction was carried out in the presence of a 10x restriction digestion buffer, which provided the optimal pH for enzyme activity. </span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c4 c25">DNA purification</span></p><p class="c1"><span class="c4 c2">DNA purification was used to remove residual enzymes, salts, and buffer components after digestion. A silica column-based purification kit was used, which relied on the selective affinity of DNA for silica under high-salt, chaotropic conditions. This selectivity arises because chaotropic agents such as guanidinium salts disrupt the hydrogen bonding in water, causing nucleic acids to become dehydrated and exposing their phosphate backbones. These negatively charged backbones formed electrostatic interactions and hydrogen bonds with the hydroxyl groups on the silica surface, enabling the DNA to bind tightly to the filter membrane (Vandeventer et al., 2013). In contrast, proteins and other contaminants either lack sufficient negative charge or are denatured by the chaotropic salts, preventing them from binding efficiently and allowing them to be washed away (Promega, 2025). Once contaminants were removed, the bound DNA was eluted in nuclease-free water, which lowers the salt concentration and disrupts the DNA-silica interaction, thereby releasing the DNA into solution. This step ensured that the plasmid used for downstream annealing and gel formation was clean and free of inhibitory impurities that could affect later analysis.</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c4 c25">Gel electrophoresis</span></p><p class="c1"><span class="c4 c2">Agarose gel electrophoresis was used to confirm both plasmid linearization and successful X-DNA formation. This technique separates DNA fragments based on size by applying an electric field to a gel matrix. DNA samples were loaded into wells of an agarose gel and submerged in 1x TAE running buffer, which provides the necessary ionic environment for current flow. When an electric field is applied, the negatively charged DNA migrates toward the positive electrode (anode). The agarose gel contains microscopic pores that act as a molecular sieve, allowing smaller DNA fragments to move more quickly through the matrix than larger ones (Lee et al., 2012). Linear DNA migrates differently from its circular or supercoiled counterparts, allowing confirmation of successful plasmid digestion through changes in band position. A DNA ladder (a molecular weight standard of known sizes) was used as a reference to assess fragment sizes, and shifts in migration patterns were analyzed to verify proper digestion and annealing. </span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c4 c25">Thermal Annealing of Oligonucleotides</span></p><p class="c1"><span class="c4 c2">DNA annealing was used to assemble the X-DNA nanostructure from four single-stranded oligonucleotides (X1-X4). Annealing refers to the process of hybridization, where complementary strands of DNA bind through complementary base pairing. The oligos were mixed in a buffer and subjected to a thermocycling program that gradually reduced the temperature from 95&deg;C to 25&deg;C. This slow cooling facilitated the correct folding and pairing of the strands into a stable four-way junction, known as X-DNA. Such junctions serve as crosslinking elements capable of hybridizing with specific sequences on the linearized plasmid, allowing them to form a physically interconnected network. </span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c4 c25">Buffer Exchange using Centrifugal Filtration</span></p><p class="c1"><span class="c4 c2">Buffer exchange was performed using centrifugal ultrafiltration to purify and concentrate the annealed X-DNA, thereby preparing it for integration into the hydrogel. This technique allows the replacement of the solution surrounding a biomolecule with a more suitable buffer, which is essential when transitioning between experimental conditions. A centrifugal filter unit with a 3,000 Da molecular weight cutoff (MWCO) was used. During high-speed centrifugation, small molecules, such as excess salts, unbound nucleotides, and annealing buffer components, passed through the membrane, while the larger X-DNA structures were retained. This process enabled simultaneous purification, buffer exchange, and concentration of X-DNA. Buffer exchange into nuclease-free water reduced ionic strength, enhancing compatibility with hydrogel formation and cell-free protein expression. Maintaining clean buffer conditions is crucial for reducing ionic strength, preserving structural integrity, and ensuring reproducibility in subsequent applications.</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c4 c25">Spectrophotometry (Nanodrop)</span></p><p class="c1"><span class="c4 c2">Spectrophotometry was used to quantify DNA concentration using absorbances at 260 nm. DNA maximally absorbs ultraviolet light at this wavelength due to its nucleotide bases, and an absorbance of 1.0 corresponds to 50&micro;g/mL of double-stranded DNA in a standard 1 cm path length. Using this ratio, the concentration of DNA in a sample can be deduced based on the absorbance of the sample. This quantification step ensures that we use standardized DNA concentrations during hydrogel formation, which affects both mechanical strength and expression yield, and allows for reproducibility in our results (Qiagen, n.d.).</span></p><h1 class="c11" id="h.h8pkpe35k3p7"><span class="c2">Reagents</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">10X CutSmart Buffer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">New England Biolabs</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">B6004S</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Restriction Enzyme XmaI</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">New England Biolabs</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">R0180S</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Nuclease-free water</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hallam Lab</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">NA</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Agarose</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Sigma-Aldrich</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c8 c48">A9539</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">1X TAE Buffer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hallam Lab</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">NA</span></p></td></tr><tr class="c50"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">SYBR Safe</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">ThermoFisher</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c38">S33102</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">5X DNA loading dye</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">ThermoFisher</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c38">R0611</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">QIAquick Miniprep Kit</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Qiagen</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">27104</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Oligonucleotides</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">IDT</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c2 c8">&mdash; </span><span class="c2 c9 c8">Custom Order</span><span class="c4 c2">&nbsp;&mdash;</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">1X TE buffer (pH 8.0)</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hallam Lab</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c4 c2">&mdash;</span></p></td></tr></table><h1 class="c11" id="h.inziny420ijo"><span class="c18 c2">Methods</span></h1><p class="c34"><span class="c2 c8">This protocol was adapted from Park et al. (2009). To begin, 1</span><img src="images/exp/image1.png"><span class="c4 c2">&nbsp;of circular plasmid DNA was digested with XmaI in 10x CutSmart buffer and incubated overnight at 37&#8239;&deg;C. After digestion, the sample was purified using the QIAquick Nucleotide Removal Kit and eluted in nuclease-free water. A portion of the linearized plasmid (0.5&#8239;&micro;g) was run on a 1% agarose gel in 1x TAE buffer, stained with SYBR Safe, and compared to the undigested plasmid to confirm successful digestion via a visible shift in band migration between the circular and linearized forms. Following confirmation, the DNA was treated with calf intestinal alkaline phosphatase (CIP) to remove 5&prime; phosphates and prevent self-ligation. The sample was purified again and re-phosphorylated using T4 polynucleotide kinase and ATP to restore functionality for crosslinking. The resulting linearized, phosphorylated plasmid was concentrated to ~500&#8239;ng/&micro;L using a vacuum concentrator. </span></p><p class="c1"><span class="c4 c2">To assemble the synthetic four-way DNA junction (X-DNA), four complementary single-stranded oligonucleotides (X1&ndash;X4), each modified with a 5&prime; phosphate group, were used. The sequences were resuspended in 1&times; TE buffer (pH 8.0) to a final concentration of 1.0&#8239;mM and combined in equimolar amounts. After thorough mixing, the oligonucleotide solution was annealed using a thermocycler under a gradual cooling program designed to promote precise and complete hybridization into the target nanostructure. The thermal profile used for annealing is outlined in Table 1 (shown in Appendix A).</span></p><p class="c1 c6"><span class="c4 c2"></span></p><table class="c24"><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Step #</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Action</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Duration</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">1</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hold at 95 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">2 min</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">2</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Decrease to 65 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">With maximum rate</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">3</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hold at 65 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">2 min</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">4</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Decrease to 60 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">With maximum rate</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">5</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hold at 60 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">5.5 min</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">6</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Decrease 1 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">With maximum rate</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">7</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hold</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">30 s</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">8</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Repeat steps 6 and 7 until temperature is 20 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">30 s each cycle</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">9</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hold at 20 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">30 s</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">10</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Decrease to 4 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">With maximum rate</span></p></td></tr><tr class="c13"><td class="c30" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">11</span></p></td><td class="c31" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Hold at 4 &deg;C before use or store at &ndash;20 &deg;C</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">&ndash;</span></p></td></tr></table><p class="c29"><span class="c25 c9 c8">Table 1:</span><span class="c2 c9 c8">&nbsp;Thermal profile for annealing oligonucleotides to form X-DNA</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2 c8">Following annealing, the X-DNA solution was purified by centrifugal ultrafiltration to remove excess salts, unbound strands, and buffer components. A centrifugal filter unit with a 3,000 Da molecular weight cutoff (MWCO) was used, and the solution was processed at 15,000g and 4&#8239;&deg;C overnight. The retained X-DNA was then rinsed with ice-cold nuclease-free water in three successive wash-and-recovery cycles. The final collected eluent, approximately 300&#8239;&micro;L, was pooled and stored at -20&#8239;&deg;C. To verify successful assembly, the X-DNA product was quantified by UV spectrophotometry at 260 nm and analyzed by agarose gel electrophoresis. A distinct band in the 100-150&#8239;bp range should be consistently observed on 1% agarose gels (Park et al., 2009), confirming the formation of the X-DNA.</span></p><h1 class="c11" id="h.70jwifs3qdf0"><span class="c18 c2">Results &amp; Discussion</span></h1><p class="c1"><span class="c4 c2">A 1% agarose gel was used to validate the formation of X-DNA (Fig. 1). The first two lanes, containing unannealed samples, showed faint bands slightly below the lowest molecular weight band of the ladder, consistent with the expected molecular weights of individual strands. In contrast, the annealed samples exhibited a bright, prominent band at ~100-150 bp, indicative of correctly assembled X-DNA. A downward smear beneath the major band was also observed, likely representing partially annealed intermediates. Fully hybridized X-DNA complexes contain more DNA and adopt conformations with reduced electrophoretic mobility; thus, the presence of a downward smear is expected. These results confirm the successful synthesis and structural integrity of the X-DNA construct.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(3.14rad) translateZ(0px); -webkit-transform: rotate(3.14rad) translateZ(0px); width: 259.15px; height: 224.57px;"><img alt="" src="images/exp/image6.jpg" style="width: 259.15px; height: 344.15px; margin-left: 0.00px; margin-top: -119.58px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c25">Figure 1: </span><span class="c3 c2">A 1% agarose gel shows faint bands for unannealed strands (lanes A &amp; B) and a strong ~100-150 bp band for annealed samples (lanes C &amp; D), suggesting successful X-DNA formation. The downward smear (lanes C &amp; D) indicates partially annealed intermediates.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c4 c2">To validate the formation of the linearized plasmid, samples were analyzed using a NanoDrop spectrophotometer to measure DNA concentration and purity, as shown in Table 2. The 260/280 ratio (indicating protein contamination) and the 260/230 ratio (indicating the presence of compounds that absorb at 230 nm, such as organic or salt contamination) were both approximately 2.0, which suggests possibly some RNA contamination but minimal presence of contaminants that absorb at 230 nm . The curve indicated that 19.3 ng/&micro;L of DNA was extracted from an OD 600 culture.</span></p><p class="c1 c6"><span class="c4 c2"></span></p><table class="c24"><tr class="c13"><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Quantity </span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Value</span></p></td></tr><tr class="c13"><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Measured Concentration (ng/&micro;L)</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">19.3</span></p></td></tr><tr class="c13"><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Absorbance At 260/280</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">2.02</span></p></td></tr><tr class="c13"><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Absorbance At 260/230</span></p></td><td class="c16" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">2.07</span></p></td></tr></table><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2">Table 2: NanoDrop spectrophotometry showing 260/280 and 260/230 ratios of ~2.0, and a final concentration of </span><span class="c4 c2">&nbsp;19.3 ng/&micro;L.</span></p><p class="c1 c6"><span class="c4 c2"></span></p><hr><p class="c1 c6"><span class="c4 c2"></span></p><h1 class="c11" id="h.tzabkeg77378"><span class="c18 c2">P-gel Formation</span></h1><h2 class="c33 c39" id="h.ac0lcjwhinpp"><span class="c21 c2 c47">Abbreviations Used</span></h2><ul class="c12 lst-kix_obdmgy1vscwr-0 start"><li class="c1 c17 li-bullet-0"><span class="c3 c2">P-Gel: Protein-Producing Hydrogel</span></li></ul><h1 class="c11" id="h.3o1zi25etbub"><span class="c18 c2">Introduction </span></h1><p class="c36"><span class="c3 c2">After forming the PDMS molds, the protein-producing DNA hydrogel (P-gel) could be synthesized. This process involved three critical steps. First, a gel precursor solution consisting of X-DNA, plasmid, ligase, and buffer was formed. &nbsp;Next, the precursor solution was placed on a glass slide and incubated, allowing for the gel to form (using DNA ligase to covalently crosslink the X-DNA and linearized plasmid components). Lastly, the P-gel pads were carefully detached from the PDMS and stored in 1x TAE buffer. Overall, this protocol took approximately 4 hours, and the steps were performed in parallel to produce multiple p-gels simultaneously (Park et al., 2009). </span></p><p class="c36 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.5dxkr2ruv2p7"><span class="c18 c2">Techniques</span></h1><p class="c36 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">Cross-Linking &amp; DNA Ligation</span></p><p class="c36"><span class="c3 c2">DNA ligation is the process of covalently linking DNA strands by forming a phosphodiester bond between a 3&prime;-OH and a 5&prime;-phosphate in an ATP-dependent reaction catalyzed by DNA ligase (Park et al., 2009). In our X-DNA hydrogel, monomers first assemble by sticky-end hybridization to create a temporary and weak DNA network of nicked X-DNA and linearized plasmid junctions. Ligation then seals these nicks, converting the annealed sticky ends into continuous backbones and locking the X-DNA and plasmid network into a covalently crosslinked gel. This step increases the hydrogel&rsquo;s stability and prevents dissociation under dilution. (Park et al., 2009).</span></p><p class="c36 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">Confocal Validation</span></p><p class="c36"><span class="c3 c2">Confocal microscopy is a high-resolution optical imaging technique that allows visualization of 3D structures within gels at the microscale level. In this protocol, it was used to confirm that the P-gel formed a continuous, interconnected network. To visualize DNA using confocal microscopy, intercalating fluorescent SYBR Green dyes were used. These dyes insert between the base pairs of double-stranded DNA and emit bright fluorescence when excited by a laser, effectively tracing the distribution and density of DNA strands throughout the gel. By scanning across the z-axis, the confocal microscope produced layered optical sections that revealed uniform fluorescence and a stable crosslinked matrix, validating successful hydrogel formation (Park et al., 2009).</span></p><h1 class="c11" id="h.2af76zbnnmdq"><span class="c2">Reagents</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">T4 DNA Ligase</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">New England Biolabs</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">M0202S</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c21 c2 c7">10X T4 Ligase Buffer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">New England Biolabs</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">B0202S </span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">Xmal</span><span class="c21 c2 c7">&nbsp;sticky ends X-DNA Oligomers</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c21 c2 c7">&mdash;</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c21 c2 c7">&mdash;</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c21 c2 c7">100 nM Linearized Plasmid</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c21 c2 c7">&mdash;</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c2 c7 c21">&mdash;</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c21 c2 c7">Nuclease-Free Water</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c21 c2 c7">&mdash;</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c21 c2 c7">&mdash;</span></p></td></tr></table><h1 class="c11" id="h.ale7wtyao0y"><span class="c18 c2">Methods</span></h1><p class="c36"><span class="c3 c2">To produce the P-gel pads, a P-gel precursor solution was prepared consisting of X-DNA stock solution, linearized plasmid solution, T4 DNA ligase, 10x DNA ligase buffer, and nuclease-free water (Park et al., 2009). The solution was prepared with an X-DNA to plasmid ratio of 3000:1 in the volumes and concentrations listed in Table 1. &nbsp;</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c29"><span class="c3 c25">Table 1. Components of P-Gel Precursor Solution for Various Molar Ratios</span></p><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c3 c25">Component</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c3 c25">Final Concentration</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c3 c25">Volume</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">X-DNA</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">0.12 M</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">10.5 &micro;L</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Linearized Plasmid</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">204 ng/&micro;L</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">5.25 &micro;L</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">T4 Ligase</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c3 c2">&mdash;</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">10.25 &micro;L</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">10x T4 Ligase Buffer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">1x</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">3 &micro;L</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Nuclease-Free Water</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c3 c2">&mdash;</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">1 &micro;L</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Total</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">NA</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">30 &micro;L</span></p></td></tr></table><p class="c1 c6"><span class="c3 c2"></span></p><p class="c36"><span class="c3 c2">To form the pads themselves, 8.0 &micro;L of p-gel precursor solution was placed onto a glass slide and sandwiched with the fabricated PDMS mold, with gentle pressure exerted to squeeze out any excess solution. This was incubated at room temperature for at least 4 hours to allow the P-gel to form by complete crosslinking by DNA ligase in line with previous work by Park et al. (2009). After the gel was formed, the PDMS mold was peeled off, and the P-gel pads were washed off the glass slide using nuclease-free water. Pads were then gently pipetted into a 1.5 mL microcentrifuge tube and stored in a 4&deg;C refrigerator for future addition to the lysate system. </span></p><p class="c36 c6"><span class="c3 c2"></span></p><p class="c36"><span class="c2">Confocal microscopy was used to validate the formation of a P-gel. The P-gel was stained with a DNA intercalating dye, SYBR Green, to visualize its internal structure. Specifically, 10 &micro;L of the hydrogel was incubated with the dye at room temperature for 10 minutes, allowing the fluorescent molecules to insert between double-stranded DNA regions within the network </span><span class="c2 c8">(ThermoFisher Scientific, 2022)</span><span class="c3 c2">. Excess dye was removed by gentle washing with a buffer to reduce background fluorescence. The stained sample was then transferred onto a glass slide and imaged using a confocal laser scanning microscope equipped with a 488 nm excitation laser. Z-stack images were collected across the full gel depth and reconstructed into a 3D projection to assess the connectivity of the crosslinked DNA network. </span></p><h1 class="c11" id="h.igeahjxamd6s"><span class="c18 c2">Results &amp; Discussion</span></h1><p class="c1"><span class="c3 c2">Upon mixing X-DNA monomers with DNA ligase at room temperature, the solution rapidly became viscous, indicating successful initiation of crosslinking (Figure 2a). After stamping with the mold for 4 hours, a square-shaped gel pad was obtained (Figures 2b-c). Of the 10 P-gel pads prepared, only 2 formed correctly and were extractable. This low yield is likely due to inadequate vortexing of the master solution, which may have led to uneven distribution of ligase and incomplete crosslinking. These observations suggest that gel formation occurred at the macroscopic scale.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 194.67px;"><img alt="" src="images/exp/image13.png" style="width: 624.00px; height: 194.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c25">Figure 2</span><span class="c3 c2">: Formation of P-gel following ligation of X-DNA monomers. (a) Viscous solution was observed immediately after mixing X-DNA monomers with DNA ligase at room temperature, indicating initiation of crosslinking. (b-c) Mold-stamped square-shaped P-gel pads obtained after 4 hours of incubation in solution (b) or on a glass slide (c).</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">To confirm the P-gel&rsquo;s formation and crosslinking at the microscopic level, confocal microscopy was performed as shown in Figures 3a and 3b. At the 20 &mu;m scale (Figure 3a), a green mesh-like structure is visible, indicating successful formation of a DNA hydrogel network. Several dense clumps are also observed, likely resulting from incomplete dissociation of the lyophilized X-DNA during rehydration or from self-association of supercoiled plasmids. At higher magnification (100 &mu;m scale, Figure 3b), the network appears as a fine, continuous mesh interspersed with bright aggregates, again consistent with partially dissociated DNA clusters. These findings confirm that crosslinking occurred and that the P-gel network formed as expected, though optimization of DNA solubilization may further improve uniformity and downstream use of the gel.</span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 312.00px;"><img alt="" src="images/exp/image4.png" style="width: 624.00px; height: 312.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c3 c2">Figure 3: Confocal microscopy images confirming P-gel formation and crosslinking. (a) A 20 &mu;m scale image showing a green mesh-like hydrogel network with visible DNA clumps, likely due to incomplete dissociation of lyophilized X-DNA or plasmid self-association. (b) A 100 &mu;m scale image at higher magnification reveals a finer, continuous DNA hydrogel mesh interspersed with bright aggregates corresponding to partially dissociated DNA regions.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><hr><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.k34u7xx67ut5"><span class="c18 c2">PDMS Mold Formation</span></h1><h2 class="c39 c33" id="h.wzstfgkd4mar"><span class="c21 c2 c47">Abbreviations Used</span></h2><ul class="c12 lst-kix_w5ja0u4v7ehl-0 start"><li class="c1 c17 li-bullet-0"><span class="c3 c2">PDMS: polydimethylsiloxane</span></li></ul><h1 class="c11" id="h.4by61xhe6nxh"><span class="c18 c2">Introduction</span></h1><p class="c37"><span class="c3 c2">Park et al. found that the global geometry of the hydrogel significantly influenced protein production efficiency (2009). Since most protein synthesis occurs at the gel&rsquo;s surface, increasing surface area is expected to enhance overall reaction yield. To align with these findings, our team designed PDMS molds to shape the hydrogel prior to cross-linking. Both the mold composition and geometry were critical design parameters and discussed in the design page of the website.</span></p><h1 class="c11" id="h.mmkymgnalygv"><span class="c18 c2">Techniques</span></h1><p class="c1"><span class="c3 c25">Resin Negative Mold</span></p><p class="c1"><span class="c3 c2">A resin negative mold was fabricated to serve as the master template for PDMS casting. The term &lsquo;negative&rsquo; refers to the fact that the mold&rsquo;s geometry is the inverse of the desired final structure, where raised regions in the resin correspond to channels or cavities in the resulting PDMS (polydimethylsiloxane) mold. This ensures that when PDMS is poured and cured over the resin surface, it forms a positive replica with features matching the intended design. These molds must be produced by 3D printing a high-resolution resin pattern, which is then post-cured under UV light to enhance hardness and surface fidelity. The cured resin master is cleaned with isopropanol to remove uncured residues and fully dried before PDMS casting, ensuring accurate feature transfer and easy demolding.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">PDMS Positive Mold</span></p><p class="c1"><span class="c3 c2">A PDMS mold was cast from the resin negative to shape the P-Gels into the correct geometry. PDMS was prepared by mixing the base and curing agent in a 10:1 ratio by weight, and degassing the mixture under vacuum to remove trapped air bubbles. The liquid PDMS was then poured over the resin negative mold, ensuring complete coverage of all surface features, and cured at 60 &deg;C for several hours until fully cross-linked. Once cured, the PDMS layer was carefully peeled off the resin master, yielding a soft, durable mold with feature dimensions identical to the original design. The resulting PDMS mold could then be used as a flexible template for subsequent hydrogel or microfluidic casting steps.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.truzd5j7dnef"><span class="c2">Reagents</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Sylgard 184 Silicone Elastomer Kit </span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Dow Corning</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c15"><span class="c4 c2">W292907</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Isopropanol (IPA)</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Sigma-Aldrich</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">I9516</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Petri Dish</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Thermo Fisher Scientific</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c22"><span class="c4 c25">&mdash;</span></p></td></tr></table><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.o2leos8kl6mk"><span class="c18 c2">Methods</span></h1><p class="c1"><span class="c3 c25">Resin Negative Mold</span></p><p class="c1"><span class="c3 c2">The negative mold was fabricated using a Formlabs Form 3+ stereolithography (SLA) printer with Formlabs Clear Resin v4. The design file was prepared using CAD software and sliced with Formlabs PreForm prior to printing. After printing, the resin mold was washed twice in isopropanol to remove uncured resin and cured under UV light for 30 minutes. The resulting resin structure served as a negative master, meaning its raised and recessed regions were geometrically inverted relative to the intended final PDMS mold. </span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">PDMS Positive Mold</span></p><p class="c1"><span class="c3 c2">Polydimethylsiloxane (PDMS; Sylgard 184, Dow Corning) was prepared by mixing the elastomer base and curing agent at a 10:1 (w/w) ratio. The mixture was stirred thoroughly to ensure homogeneity. The PDMS was then gently poured over the resin negative mold placed in a clean Petri dish, ensuring complete coverage of the surface features. The assembly was cured in an oven at 60 &deg;C for 4 hours until full crosslinking occurred. After cooling to room temperature, the PDMS layer was carefully peeled away from the resin by cutting around a dead space and peeling via tweezers. The mold was stored in a petri dish wrapped in tape to minimize drying.</span></p><h1 class="c11" id="h.b67gopp047ht"><span class="c18 c2">Results &amp; Discussion</span></h1><p class="c1"><span class="c3 c2">The results of the resin negative mold print and subsequent PDMS casting are shown in Figure 4. A well-defined mold was successfully fabricated in the orientation displayed. The lower structures serve as support elements, printed to enhance stability during the resin printing process. The structure on the left represents the cast and demolded PDMS mold, which exhibits a series of indentations corresponding to the individual P-gel compartments. Overall, both the resin negative mold fabrication and PDMS casting protocols were successful, yielding clean and structurally intact molds.</span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 398.50px; height: 173.02px;"><img alt="" src="images/exp/image11.png" style="width: 398.50px; height: 173.02px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 190.50px; height: 172.71px;"><img alt="" src="images/exp/image3.png" style="width: 190.50px; height: 207.76px; margin-left: 0.00px; margin-top: -16.24px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c2 c9">Figure 4:</span><span class="c3 c2">&nbsp;The 3D-printed resin negative mold (right) and the casted PDMS mold (left) show successful replication of P-gel compartments. Support structures (bottom) ensured print stability.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><hr><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.hxgpdlopk73o"><span class="c23 c2 c9 c42">Protein Expression &amp; Isolation</span></h1><h2 class="c39 c33" id="h.sipeqteiqrq0"><span class="c21 c2 c47">Abbreviations Used</span></h2><ul class="c12 lst-kix_j2ztfdahgekg-0 start"><li class="c1 c17 li-bullet-0"><span class="c3 c2">CA: Carbonic anhydrase</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">CaCl2: Calcium chloride</span></li><li class="c1 c17 li-bullet-0"><span class="c19 c2 c9">E. coli: Escherichia coli</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">IPTG: Isopropyl &beta;-D-1-thiogalactopyranoside</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">LB: Lysogeny broth</span></li><li class="c1 c17 li-bullet-0"><span class="c2">NaCl: Sodium chloride</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">OD600: Optical Density at 600 nm</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">SOC: Super optimal broth with catabolite repression</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">CA: Carbonic Anhydrase</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">E. coli: Escherichia coli</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">SDS-PAGE: Sodium Dodecyl Sulfate&ndash;Polyacrylamide Gel Electrophoresis</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">APS: Ammonium Persulfate</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">TEMED: N,N,N&#39;,N&#39;-Tetramethylethylenediamine</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">IAM: Iodoacetamide</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">IPTG: Isopropyl &beta;-D-1-thiogalactopyranoside</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">IPA: Isopropanol</span></li></ul><h1 class="c11" id="h.fnzme7iv1j2e"><span class="c18 c2">Introduction</span></h1><p class="c1"><span class="c2">Across multiple independent experiments, three protein production systems were evaluated for their ability to synthesize carbonic anhydrase (CA). Specifically, a cell-based </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;expression system, a traditional cell-free plasmid-based system, and a novel protein-producing hydrogel (P-gel)-based method were tested. All experiments followed a similar workflow involving induction, isolation, and assessment. Given the methodological overlap, the protocols are presented collectively by first outlining the induction and isolation methods for each system, followed by assessment of the isolates and a unified discussion of the results and next steps.</span></p><h1 class="c11" id="h.7sxfiyov3112"><span class="c2 c9">E. coli-Based</span><span class="c18 c2">&nbsp;Protein Expression</span></h1><p class="c1"><span class="c2">The purpose of this experiment is to induce and observe recombinant gene expression in </span><span class="c2 c9">Escherichia coli </span><span class="c2">(</span><span class="c2 c9">E. coli</span><span class="c2">) using a carbonic anhydrase (CA) encoding plasmid that is activated by isopropyl &beta;-D-1-thiogalactopyranoside (IPTG). The process involves preparing lysogeny broth (LB) agar plates, culturing </span><span class="c2 c9">E. coli </span><span class="c3 c2">transformed with the engineered plasmid, and inducing protein expression using IPTG. Expression levels are evaluated by comparing collected protein levels before and after induction to determine if CA production was successfully upregulated. The results from this protocol will be used to validate the successful expression of the gene of interest in a cellular system after downstream protein purification, and serve as a control comparison for cell-free protein-producing methods.</span></p><h1 class="c11" id="h.gkub319ks2p6"><span class="c2">Techniques: </span><span class="c2 c9">E. coli-Based</span><span class="c18 c2">&nbsp;Protein Expression</span></h1><p class="c1"><span class="c3 c25">LB Agar Plate Preparation</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c2">LB is a nutritionally rich medium often used to culture </span><span class="c2 c9">E. coli</span><span class="c3 c2">. It contains tryptone (a source of peptides), yeast extract (a source of vitamins and coenzymes), and sodium chloride (NaCl, to maintain osmotic pressure). Agar is also added as a gelling agent to prepare solid plates for culturing bacterial colonies. After preparing the media, it is autoclaved at 121 &ordm;C at 15 psi for 15 minutes to ensure sterility. To prevent degradation of the antibiotic, kanamycin is added to the media at a final concentration of 100 &mu;g/mL once the LB agar media has cooled to approximately 50 &ordm;C. The addition of an antibiotic to LB agar medium allows for the selection of only bacteria that are resistant to that antibiotic, which can be conferred by a plasmid carrying the antibiotic resistance gene (Addgene, 2016). This allows for the selective growth of only bacteria that contain a plasmid of interest, thereby minimizing contamination. Plates are then poured aseptically and stored inverted at 4 &ordm;C until use.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Cell Competency</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c2">Cell competency refers to a cell&rsquo;s ability to take up foreign DNA from its surrounding environment. In a lab setting, artificial cell competence is achieved through a chemical process or electroporation (Martin, n.d.). To ensure the uptake of plasmid DNA, </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;cells are often made chemically competent using calcium chloride (CaCl2). Treatment with Ca2+ ions neutralizes the negative charge of DNA and the cell membrane, promoting DNA entry (Asif et al., 2017). For a CaCl2 protocol, cells are incubated on ice with 0.1 M CaCl2, then washed and resuspended in a chilled solution of CaCl2 and 15% glycerol, which acts as a cryoprotectant. The competent cells are then aliquoted, flash frozen, and stored at -80 &ordm;C for later transformation.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Heat Shock Transformation</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c2">This method of transformation involves competent </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;cells being heat-shocked at 42 &ordm;C, which causes a disruption in the membrane potential and allows plasmid DNA to enter the cytoplasm. The cells are then rapidly cooled on ice and incubated in super-optimal broth with catabolite repression (SOC) medium to recover and allow expression of the antibiotic resistance gene before plating. This recovery step is crucial for maximizing transformation efficiency (Kroemer, n.d.-a).</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Antibiotic Selection using Kanamycin</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">After the plasmid of interest is uptaken by bacteria, only cells that have successfully incorporated the plasmid and are expressing the kanamycin resistance gene can survive on the kanamycin plates. Kanamycin inhibits protein synthesis by targeting the 30S ribosomal subunit present in bacteria, thereby disrupting translation and leading to cell death (&ldquo;Kanamycin,&rdquo; 2008). Using LB agar kanamycin plates allows for the selective growth of transformed colonies for this project.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Validation via New Generation Sequencing (NGS)</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c2">To confirm successful transformation of the plasmid, the transformed </span><span class="c2 c9">E. coli</span><span class="c2">&nbsp;were also sequenced using next-generation sequencing (NGS). The resulting FASTQ reads were quality-checked using FastQC, then aligned to the reference plasmid sequence using BWA-MEM. Alignment files were processed with SAMtools to generate sorted BAM files, and variant calls were obtained using bcftools. Mutation frequency was then assessed by comparing the variant calls to the reference plasmid, confirming successful incorporation of the plasmid into the host </span><span class="c2 c9">E. Coli</span><span class="c3 c2">.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Spectrophotometric Optical Density at 600 nm (OD600) Measurement</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c3 c2">Bacterial growth is quantified using spectrophotometry to measure the optical density (OD) at 600 nm (OD600). Absorbance at this wavelength correlates with cell density and can help indicate what stage of growth a bacterial colony has reached. An OD600 reading of 0.4-0.5 indicates the mid-log phase of bacterial growth (Kobayashi et al., 2006). Cells in this phase are actively dividing and are in their most healthy and uniform state, optimizing them for induced gene expression and other experimental protocols (Bruslind, 2021).</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c3 c25">IPTG-induced Gene Expression</span></p><p class="c1 c6"><span class="c3 c25"></span></p><p class="c1"><span class="c2">IPTG is a synthetic compound used to trigger the expression of genes controlled by the </span><span class="c2 c9">lac</span><span class="c2">&nbsp;operon. It functions by binding to the lac repressor protein, causing it to dissociate from the operator sequence and allowing transcription of downstream genes. Unlike natural inducers such as lactose, IPTG is not metabolized by </span><span class="c2 c9">E. coli</span><span class="c2">, providing a stable and sustained induction effect (Kroemer, n.d.-b). In this protocol, IPTG is added to a final concentration of 1 mM once cultures reach an OD600 of 0.4-0.5 to maximize protein expression.</span></p><h1 class="c11" id="h.9kzc8ktcyrma"><span class="c2">Reagents: </span><span class="c2 c9">E. coli-Based</span><span class="c2">&nbsp;Protein Expression</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Kanamycin 50 mg/mL stock</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">GoldBio</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">K-120-SL25</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Anhydrous CaCl2</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Sigma-Aldrich</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">C1016-100G</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">100% Glycerol</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">GLY004.1</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">IPTG</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">GoldBio</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">I2481C5</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">SOC Medium</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">GoldBio</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">CC-302-A</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c8">Transformed </span><span class="c2 c9 c8 c23">E. coli </span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">IDT</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Custom Order: DH5-&alpha; with PET-IDT</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">NaCl</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">SOD002.1</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Tryptone</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">TRP402</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Yeast extract</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">YEX401</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Agar</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c4">AGR001.100</span></p></td></tr></table><h1 class="c11" id="h.84ns2li5xjsu"><span class="c2">Methods: </span><span class="c2 c9">E. coli-Based</span><span class="c18 c2">&nbsp;Protein Expression</span></h1><p class="c1"><span class="c3 c2">The LB agar solution was prepared by dissolving premade LB mix (10 g NaCl, 10 g tryptone, and 5 g yeast extract for 1 L of media) in distilled water in Erlenmeyer flasks, following a protocol from Addgene (Addgene, 2016). After adding 12 g of agar per 1 L of media, the solution was sterilized by autoclaving at 121 &ordm;C and 15 psi for 15 minutes. Once cooled to a safe handling temperature (~50 &deg;C), 1 mL of a 50 mg/mL kanamycin stock solution was added to 500 mL of LB agar solution to achieve a final antibiotic concentration of 100 &mu;g/mL. Plates were poured under sterile conditions, allowed to solidify, inverted, labelled, and stored at 4 &ordm;C.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">The plasmid of interest was designed according to the rationale outlined in the Design Page. Features of interest include the CA-encoding gene which produces our desired protein, a 6x Histidine tag for downstream protein isolation, a kanamycin selection marker to facilitate selective growth, the LacI operon for protein induction, and components required for transcription and translation. The final plasmid sequence was submitted to IDT, an oligo manufacturer, for synthesis and transformation into competent DH5&alpha; </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;bacterial cells.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">The presence of the plasmid was confirmed by growing the received samples on LB agar supplemented with Kanamycin, and NGS analysis of extracted plasmid DNA. Selected colonies were inoculated into fresh LB broth and grown to mid-log phase (OD600 0.4-0.5). 100 &mu;L samples were taken for pre-induction measurements. IPTG was then added to a final concentration of 1 mM, and cultures were incubated at room temperature for at least 50 minutes. 100 &mu;L samples were taken for post-induction measurements. The remaining culture was centrifuged at room temperature and 4000 rpm for 5 minutes, and the samples were centrifuged at room temperature and 10,000 rpm for 1 minute. The supernatant was removed, and the pellets were flash-frozen and stored at -80 &ordm;C for future use.</span></p><h1 class="c11" id="h.jqndxaxv7sjm"><span class="c18 c2">Cell-Based Specific Results &amp; Discussion</span></h1><p class="c1"><span class="c2">Our custom pET-IDT plasmid was designed and sent to IDT for plasmid synthesis and transformation into DH5-</span><span class="c2 c8">&alpha;</span><span class="c2">&nbsp;</span><span class="c2 c9">E. coli.</span><span class="c2">&nbsp;The service included transformation, validation of plasmid incorporation via next-generation sequencing (NGS), and shipment of the verified transformant on ice. Upon receiving the transformed bacteria, cultures were plated on LB agar plates supplemented with kanamycin (100 &micro;g/mL). Distinct</span><span class="c2 c9">&nbsp;E. coli</span><span class="c2">&nbsp;colonies were observed, as shown in </span><span class="c2 c9">Figure 5</span><span class="c3 c2">. Individual colonies were then picked and cultured in LB with kanamycin medium for further validation.</span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 196.50px; height: 203.00px;"><img alt="" src="images/exp/image8.jpg" style="width: 196.50px; height: 261.15px; margin-left: 0.00px; margin-top: -32.43px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c2 c9">Figure 5</span><span class="c3 c2">: Transformed E. coli colonies growing on LB agar with kanamycin (100 &micro;g/mL).</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">NGS analysis of plasmid DNA extracted from overnight cultures confirmed transformation success. The NGS report showcases &gt;99% coverage and &lt;0.01% mutation frequency, suggesting a successful transformation, as illustrated in </span><span class="c2 c9 c19">Figure 6.</span></p><p class="c1 c6"><span class="c19 c2 c9"></span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 470.50px; height: 257.12px;"><img alt="" src="images/exp/image2.png" style="width: 470.50px; height: 257.12px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c1"><span class="c2 c9">Figure 6</span><span class="c2">: Depth of coverage (purple) and percent wild type (orange) across the sequence. Average depth of the read is 154 bp, while the percent wild type stays near 100%.</span></p><p class="c1 c6"><span class="c19 c2 c9"></span></p><p class="c1"><span class="c3 c2">Following IPTG induction, protein expression was detected, but at concentrations too low to be considered conclusive. BCA assay results showed a small increase in total protein compared with uninduced controls, suggesting that induction occurred but at insufficient levels for further analysis, such as with an SDS-PAGE (Figure 8). The low yield possibly reflects suboptimal IPTG concentration, limited induction time, toxic properties of overproducing CA, or poor expression efficiency of the plasmid. Further optimization of induction conditions, including IPTG concentration, temperature, and incubation time, is needed to achieve reproducible and detectable protein expression.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">Overall, these results confirmed:</span></p><ul class="c12 lst-kix_de1lyxmd30ln-0 start"><li class="c1 c17 li-bullet-0"><span class="c2">Successful transformation of </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;with the recombinant plasmid.</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">Effective antibiotic selection using kanamycin.</span></li><li class="c1 c17 li-bullet-0"><span class="c2">Sequence-verified plasmid integrity by NGS analysis.</span></li></ul><hr style="page-break-before:always;display:none;"><h1 class="c11 c46" id="h.k4nj16etrnmw"><span class="c18 c2"></span></h1><h1 class="c11" id="h.rvhxzupv29or"><span class="c18 c2">Cell-Free Based Induction</span></h1><p class="c1"><span class="c3 c2">In living cells, protein expression occurs through the central dogma of molecular biology: DNA is transcribed into messenger RNA (mRNA), which is then translated by ribosomes into proteins. This process requires a tightly regulated and resource-intensive environment, where enzymes, ribosomes, energy sources, and cofactors all work in concert within the cytoplasm.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Cell-free protein expression systems replicate this process outside of the cell using crude or purified lysates. In this protocol, </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;lysate is utilized, which contains ribosomes, RNA polymerases, tRNAs, amino acids, and various cofactors required for transcription and translation. However, it lacks genomic DNA, which allows for precise control of expression using the added plasmid. </span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">This protocol aims to express CA using a P-gel. The P-gel is a biomaterial embedded with plasmid DNA, enabling localized protein synthesis when placed in contact with cell-free E. coli lysate. Expression is induced in a two-compartment system using </span><span class="c2 c9">E. Coli </span><span class="c3 c2">lysate and IPTG, and functional protein production is validated through SDS-PAGE. The setup displays a simplified cell-free system, allowing in-gel transcription and translation reactions.</span></p><h1 class="c11" id="h.1f8obqqxipjd"><span class="c2">Techniques: Cell-Free Based Protein Production</span></h1><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">BCA Assay</span></p><p class="c1"><span class="c2">To quantify the total protein produced within the reaction mixture, a Bicinchoninic Acid (BCA) assay was performed. In this method, proteins in the sample reduce Cu&sup2;&#8314; ions to Cu&#8314; under alkaline conditions, and the resulting Cu&#8314; forms a purple-colored complex with bicinchoninic acid. The intensity of this colour is measured spectrophotometrically at 562 nm, and is directly proportional to protein concentration. A standard curve was generated using known concentrations of bovine serum albumin (BSA), and the protein content of the P-gel</span><span class="c2 c9">&ndash;E. coli </span><span class="c3 c2">lysate mixture was determined by interpolating sample absorbances from this curve. All samples and standards were measured in triplicate to ensure reproducibility.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">SDS-PAGE (Sodium Dodecyl Sulfate-Polyacrylamide Gel Electrophoresis)</span></p><p class="c1"><span class="c3 c2">This technique separates proteins based on their molecular weight. Proteins are denatured using the detergent SDS, which imparts a uniform negative charge across the denatured molecule&rsquo;s surface, and then loaded onto a polyacrylamide gel. When an electric current is applied, the proteins migrate through the gel, with smaller proteins moving faster than larger ones. This allows estimation of protein size and purity.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">IPTG and cAMP</span></p><p class="c1"><span class="c3 c2">IPTG is a lactose analog that induces gene expression by inactivating the lac repressor and enabling transcription from the lac promoter. cAMP enhances this effect by binding to CAP (catabolite activator protein), which increases RNA polymerase recruitment to the promoter. Together, these molecules promote strong expression of target genes under lac control.</span></p><h1 class="c11" id="h.tw9r28ea8e2u"><span class="c2">Reagents: Cell-Free Based Protein Production</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c25">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Nutrient Feed</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">New England Biolabs</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">E5360S</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Dialysis Membrane</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">ThermoFisher</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">69560</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">E. coli Lysate</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">New England Biolabs</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">E5360S</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">HEPES</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">HEP003.100</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Tris Base</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">TRS001.500</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">NaOH</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">Sigma-Aldrich</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">S5881</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Tris-HCl, pH=6.8</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">TRS002.500</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Isopropanol (IPA)</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">Hallam Lab</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">NA</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Acetic Acid</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">BioShop Canada</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">ACE333.500</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">cAMP </span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">Fisher Scientific</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c7">A23811G</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Pre-cast Stain Free SDS</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">BioRad</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c1"><span class="c4 c2">5932</span></p></td></tr></table><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.jc0mk7qytwyh"><span class="c18 c2">Methods: Cell-Free Based Protein Production</span></h1><p class="c1"><span class="c25">P-Gel expression</span></p><p class="c1"><span class="c3 c2">To initiate CA expression, a two-compartment reaction apparatus was assembled. The reaction mixture was prepared by combining E. coli lysate, buffer, and other components necessary for transcription and translation, and P-gel micropads were introduced into this mixture. The full reaction mixture was then transferred into the reaction compartment of the apparatus. In a separate reservoir segment, the feed mixture was prepared by combining the reagents listed in Table 1, which include energy sources, cofactors, and buffering agents required to support prolonged expression. </span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">The feed mixture was introduced into the reservoir compartment of the apparatus. If applicable, expression was induced by adding IPTG and possibly cAMP. The assembled reaction setup was incubated at 30&#8239;&deg;C with shaking at 750&ndash;900&#8239;rpm for a duration of 24 &ndash; 48 hours.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Table 1. Nutrient Feed Composition</span></p><table class="c24"><tr class="c13"><td class="c26" colspan="1" rowspan="1"><p class="c5"><span class="c3 c25">Component Category</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c5"><span class="c3 c25">Reagents</span></p></td></tr><tr class="c13"><td class="c26" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Primary energy sources</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">ATP, potassium phosphate or hexametaphosphate</span></p></td></tr><tr class="c13"><td class="c26" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Secondary energy sources</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Glucose, maltose/maltodextrin, 3-PGA, L-glutamate</span></p></td></tr><tr class="c13"><td class="c26" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">Cofactors</span></p></td><td class="c14" colspan="1" rowspan="1"><p class="c5"><span class="c3 c2">NAD&#8314;, CoA, ammonium, DTT (if applicable), spermidine</span></p></td></tr></table><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">BCA Assay</span></p><p class="c1"><span class="c2">To determine the total protein yield from the expression reaction, a BCA assay was performed following the manufacturer&rsquo;s protocol. First, aliquots of the reaction mixture containing P-gel and </span><span class="c2 c9">E. coli </span><span class="c3 c2">lysate were collected and diluted appropriately to fall within the linear detection range. A standard curve was prepared, using serial dilutions of bovine serum albumin (BSA) standards of known concentrations. 25 &nbsp;&micro;L of each standard and sample was mixed with 200 &micro;L of BCA working reagent in a 96-well plate and incubated at 37 &deg;C for 30 minutes. After incubation, absorbance was measured at 562 nm using a plate reader. Protein concentrations were calculated by fitting absorbance values to the BSA standard curve, and all measurements were performed in triplicate to ensure reproducibility.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">SDS-PAGE</span></p><p class="c1"><span class="c2">After incubation, the samples were analyzed using a precast</span><span class="c2 c8">&nbsp;SDS-P</span><span class="c3 c2">AGE gel with stain-free imaging technology that was purchased. This allowed for the visualization of bands via a short photoactivation reaction that makes proteins fluorescent without the use of traditional stain methodologies, such as the Fairbanks protocol. For sample preparation, 5&times; loading buffer (Tris-HCl pH&#8239;6.8, SDS, glycerol, bromophenol blue, and &beta;-mercaptoethanol) was added to the protein solutions, heated at 85&#8239;&deg;C for 2 minutes, and then cooled to room temperature. Samples and a protein mass marker were loaded into the gel, and electrophoresis was carried out using 1&times; running buffer (25&#8239;mM Tris, 240&#8239;mM glycine, 0.1% SDS). The run was stopped once the dye front reached the bottom. Following electrophoresis, the gel was illuminated under UV for 45 seconds and then imaged. </span></p><hr style="page-break-before:always;display:none;"><p class="c1 c32 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.jitq8cn9oiu0"><span class="c18 c2">Protein Isolation</span></h1><p class="c1"><span class="c2">Once carbonic anhydrase is produced, it must be purified and isolated. This was done using a histidine (His) tag that was engineered into the plasmid, which is a sequence of histidine residues, typically six long and has a high affinity for metal ions. The tag was designed to be located on the C terminus of the protein. For isolation of the protein using its His tag, a resin containing nickel-charged nitrilotriacetic acid (NTA), which is a tetradentate chelating agent that binds four Ni</span><span class="c2 c40">2+</span><span class="c2">&nbsp;ions, was utilized.</span><span class="c2 c40">1</span><span class="c2">&nbsp;The metal complex is immobilized onto a column, which can then bind the His-tagged protein, but can be eluted using a high concentration of imidazole. Imidazole has a higher affinity for Ni</span><span class="c2 c40">2+</span><span class="c3 c2">&nbsp;than the His tag, which results in imidazole outcompeting the tagged protein for binding. When this occurs, the protein is eluted from the column. Once the protein is eluted, the tag must be cleaved, which was designed to be done using His7-TEV protease. This is a viral protease with high specificity for the amino acid sequence ENLYFQ/G, cleaving between Q and G with high efficiency.</span></p><h1 class="c11" id="h.tzgcfct0to9w"><span class="c18 c2">Techniques: Protein Isolation</span></h1><p class="c1"><span class="c3 c2">Protein tags are commonly used in protein purification due to their efficiency and simplicity in isolating a protein of interest. Protein tags can be used to prevent the need for consecutive columns during purification, while still achieving high purity. Proteins are typically tagged on either the N or C terminus to prevent interference with their folding structure and function. Protein tags can also be either cleavable or non-cleavable, adding to their versatility. Generally, the tag will have a high affinity for ligands that can be immobilized on the column, while molecules with lower affinity for the column, such as non-target proteins still present from cell lysis, will flow through the column. One pitfall is that tags can sometimes interfere with the protein structure or function; in these instances, they must be cleaved. To do this, a cleavage recognition sequence can be added between the protein and the tag, allowing a protease to cleave the tag from the protein at this position. </span></p><h1 class="c11" id="h.3kq94ojz4gpj"><span class="c2">Reagents: Protein Isolation</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Ni-His Spin Column</span></p><p class="c5"><span class="c4 c2">Elution Buffer</span></p><p class="c5"><span class="c4 c2">Wash Buffer</span></p><p class="c5"><span class="c4 c2">Equilibrium Buffer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">ThermoFischer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">88224</span></p></td></tr></table><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.18r5cr7i258x"><span class="c18 c2">Methods: Protein Isolation</span></h1><p class="c1"><span class="c2">To compare the isolation of the cell-free protein system to a cell-based protein system, a positive control was performed using an </span><span class="c2 c9">Escherichia coli</span><span class="c2">&nbsp;(E.coli) based expression system. The </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;pellet was first lysed using a lysis buffer. The buffer was added to the pellet, and the culture was sonicated on ice, then added to a 1.5 mL Eppendorf tube and centrifuged. The supernatant was collected and added to a separate tube.</span></p><p class="c1 c45 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">&nbsp;Before purification was started, the columns were equilibrated to the temperature at which the purification was performed at, either at 4&deg;C or room temperature (22&deg;C). The supernatant was mixed with an equilibration buffer equal to two column volumes. First, the packing buffer was removed by placing the column into a micro-centrifuge tube that was spun at 700 g for 2 minutes. The flow-through was discarded. The column was then equilibrated using two column volumes of equilibration buffer. The column was then centrifuged at 700 g for 2 minutes, and the flow-through was once again discarded. </span></p><p class="c1 c45 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">The protein extract was then added to the column and incubated for 30 minutes at room temperature or 4&deg;C with shaking to allow for complete binding to the resin. The column was centrifuged at 700 g for 2 minutes, and the flow-through was collected in a centrifuge tube. Then, two column volumes of wash buffer were added to the column and centrifuged at 700 g for 2 minutes. The flow-through was collected in a centrifuge tube. This was repeated twice more, with each wash fraction being collected in a separate tube. The tagged protein of interest, carbonic anhydrase, was eluted by adding one column volume of elution buffer to the column. The column was then centrifuged at 700 g for 2 minutes. This fraction was collected, and the step was performed twice more, with each fraction being collected separately. The fractions were analyzed using absorbance by BCA assay, SDS-PAGE and total protein assays (Bradford, Bicinchoninic acid). The resin was regenerated and used again by first washing with 10 column volumes of MES buffer, then 10 column volumes of ultrapure water, and then stored as a 50% slurry in 20% ethanol. </span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">After isolation, the samples were analyzed using a prefor</span><span class="c2 c8">med SDS-P</span><span class="c3 c2">AGE. The purchased gel did not require a Fairbanks protocol or similar method for visualization. Protein samples were prepared by mixing with 5&times; loading buffer (Tris-HCl pH&#8239;6.8, SDS, glycerol, bromophenol blue, and &beta;-mercaptoethanol), heating at 85&#8239;&deg;C for 2 minutes, and cooling to room temperature. Samples and a protein mass marker were loaded into the gel, and electrophoresis was carried out using 1&times; running buffer (25&#8239;mM Tris, 240&#8239;mM glycine, 0.1% SDS). The run was stopped once the dye front reached the bottom. Following electrophoresis, the gel was illuminated under UV for 45 seconds, then imaged. </span></p><p class="c1 c45 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">To cleave the tag from the protein, 100 &mu;g of the protein containing a TEV protease recognition site in the linker region, was incubated overnight at 4-8 &deg;C in 50 &mu;L of standard reaction buffer. Then, 10 &mu;g of pure His7-TEV protease was added to 100 &mu;L of soluble crude extract containing the fusion protein and incubated at room temperature for 1 hour. The products were analyzed using SDS-PAGE.</span></p><p class="c1 c6 c45"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><hr style="page-break-before:always;display:none;"><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.9ngmtdcadwzn"><span class="c18 c2">Overall Results &amp; Discussion</span></h1><p class="c1"><span class="c2">Following induction and isolation of carbonic anhydrase (CA), five conditions were established to evaluate protein yield across expression platforms. This included 2 controls and 3 experimental conditions. For controls, a dilute flow-through fraction lacking detectable protein served as the negative control, while a positive control containing a known quantity of CA provided a reference for signal calibration. Two cell-free protein production (CFPP) systems were evaluated: a P-gel-based CFPP using a crosslinked DNA hydrogel matrix as its template scaffold, and a plasmid-based CFPP relying on free plasmid templates in solution. Finally, cell-based production using </span><span class="c2 c9">E. coli</span><span class="c3 c2">&nbsp;isolates served as the benchmark for conventional CA expression and purification.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">All samples were first quantified for total protein content using a BCA assay (Figure 7). Raw absorbance measurements showed a strong response in the positive control and minimal signal in the negative control. All experimental conditions contained significantly (p &lt; 0.05) more protein than the negative control and significantly less than the positive control. Upon conversion of absorbance values to concentration, all experimental groups yielded approximately 100 &micro;g/mL of protein. Because these readings were taken after nickel affinity purification and buffer exchange by size exclusion spin column, the measured protein most likely corresponds to carbonic anhydrase. No significant differences were observed among the experimental conditions for raw absorbance data.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 281.33px;"><img alt="" src="images/exp/image9.png" style="width: 624.00px; height: 281.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c3 c2">Figure 7: Raw absorbance values of BCA assay (left) and calculated concentrations (&micro;g/ml) from standard curve (right).</span></p><p class="c29 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">An SDS-PAGE gel was run for all experimental conditions described above. The positive control displayed a distinct band at 30 kD, whereas the negative control showed no detectable band, confirming that the gel and staining procedure functioned properly. None of the experimental samples exhibited visible bands, likely due to insufficient protein loaded into the gel or non-optimized experimental conditions. Notably, the 5X diluted positive control produced a faint band; given that the original sample contained approximately 1500 &micro;g/mL protein, this dilution corresponds to ~300 &micro;g/mL, roughly threefold higher than the experimental samples. Thus, the absence of visible bands in the experimental lanes is consistent with the expected lower protein concentrations despite the confirmed presence of protein via BCA. As well, the absence of bands in the expected lanes could indicate a need for further experimentation to optimize sample preparation and loading conditions of the SDS-PAGE.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 366.55px; height: 304.87px;"><img alt="" src="images/exp/image12.png" style="width: 366.55px; height: 304.87px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c2">Figure 8:</span><span class="c2 c9">&nbsp;</span><span class="c2">Lanes </span><span class="c25">A</span><span class="c2">&nbsp;and </span><span class="c25">B</span><span class="c2">: ladder; Lane </span><span class="c25">C</span><span class="c2">: positive control; Lane </span><span class="c25">D</span><span class="c2">: negative control; Lanes </span><span class="c25">E</span><span class="c2">&nbsp;and </span><span class="c25">F</span><span class="c2">: E. coli flow-through; Lane </span><span class="c25">G</span><span class="c2">: P-Gel; Lane </span><span class="c25">H</span><span class="c2">: plasmid-only cell-free protein production; Lanes </span><span class="c25">I</span><span class="c2">&nbsp;and </span><span class="c25">J</span><span class="c2">: flow-through from P-Gel and plasmid-only samples; Lanes </span><span class="c25">K</span><span class="c2">&nbsp;and </span><span class="c25">L</span><span class="c2">: 5&times; dilutions of the positive and negative controls; Lanes </span><span class="c25">M</span><span class="c2">&nbsp;and </span><span class="c25">N</span><span class="c2">: 5&times; dilutions of the P-Gel and plasmid-only samples; Lane </span><span class="c25">O</span><span class="c3 c2">.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1 c6"><span class="c3 c2"></span></p><hr><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.vs1q3d7xoss1"><span class="c18 c2">Carbonic Anhydrase Activity Assay</span></h1><h2 class="c39 c33" id="h.jsi4mr3jhaww"><span class="c21 c2 c47">Abbreviations Used</span></h2><ul class="c12 lst-kix_d1go8u71kpqj-0 start"><li class="c1 c17 li-bullet-0"><span class="c3 c2">CA: Carbonic Anhydrase</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">BC: Background control</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">S: Sample</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">NC: Negative Control</span></li><li class="c1 c17 li-bullet-0"><span class="c3 c2">PC: Positive Control</span></li></ul><h1 class="c11" id="h.zi2snifetu0s"><span class="c18 c2">Introduction</span></h1><p class="c1"><span class="c2">The aim of this protocol is to determine if the purified sample contains functional carbonic anhydrase (CA) and to quantify its activity. After isolating CA from the P-Gel, an assay was needed to verify whether the extracted enzyme was functionally active. This required assessing the isolated protein&rsquo;s catalytic capability via a CA representative reaction. CA possesses two major catalytic activities: the hydration of carbon dioxide and the hydrolysis of ester bonds </span><span class="c2 c8">(Uda et al., 2015)</span><span class="c2">. Both reactions occur at the same active site, meaning that its esterase activity, which is easier to quantify, can serve as a reliable proxy for the enzyme&rsquo;s physiological carbon-fixation activity. Hence, the functionality of the isolated CA was evaluated using a colorimetric esterase assay to quantify the enzymatic activity of the isolated protein sample </span><span class="c2 c8">(Snoeck et al., 2024)</span><span class="c3 c2">.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">The activity assay works based on the hydrolysis of an ester substrate by CA, generating a product whose absorbance can be quantified spectrophotometrically. Specifically, CA catalyzes the cleavage of a colourless compound, </span><span class="c2 c9">p</span><span class="c2">-nitrophenyl acetate, into acetate and </span><span class="c2 c9">p</span><span class="c2">-nitrophenol. In alkaline solutions, </span><span class="c2 c9">p</span><span class="c3 c2">-nitrophenol becomes bright yellow and exhibits strong absorbance at 405 nm. By monitoring these colorimetric changes and comparing them against a standard curve of known absorbances and activity, the enzyme&#39;s activity in terms of units of activity can be measured.</span></p><h1 class="c11" id="h.bzi6nbz52uko"><span class="c18 c2">Techniques</span></h1><p class="c1"><span class="c25">Buffer Exchange</span></p><p class="c1"><span class="c3 c2">Following purification, CA-containing samples were buffer-exchanged into a zinc chloride buffer to restore the enzyme&rsquo;s native catalytic environment. Zinc ions form the active center of carbonic anhydrase, coordinating a hydroxide ion that enables CO&#8322; hydration. Size-exclusion columns were used to replace the original solution and salts with the zinc buffer, retaining the enzyme while removing small molecules. This ensured proper metal reconstitution and preservation of maximal catalytic activity.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">Colorimetric Activity Assay</span></p><p class="c1"><span class="c3 c2">Colorimetric detection was used to quantify CA activity through spectrophotometric measurements of a chromogenic nitrophenol reaction product. This technique relies on establishing a standard curve by reacting known concentrations of purified CA with an ester substrate that yields a colored product upon hydrolysis. The absorbance of this product is measured at a specific wavelength (405 nm) using a spectrophotometer, generating a calibration curve that relates absorbance to enzymatic activity. The same procedure is then applied to the extracted CA samples, and their absorbance values are compared to the standard curve to determine activity. Agreement between the measured and expected responses confirms that the extracted enzyme is functionally active and consistent with carbonic anhydrase standards.</span></p><h1 class="c11" id="h.cf5eykjcczlu"><span class="c2">Reagents</span></h1><table class="c24"><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Reagent Name</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Supplier</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Catalogue Number</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">CA Assay Buffer</span></p><p class="c5"><span class="c4 c2">CA Enzyme I</span></p><p class="c5"><span class="c4 c2">CA Inhibitor I</span></p><p class="c5"><span class="c4 c2">CA Substrate (ester substrate)</span></p><p class="c5"><span class="c4 c2">Dilution Buffer VI</span></p><p class="c5"><span class="c4 c2">Nitrophenol Standard</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Abcam</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">AB284550</span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Microplate (96 well)</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">ThermoFischer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c2 c8"><a class="c10" href="https://www.google.com/url?q=https://www.thermofisher.com/order/catalog/product/165305&amp;sa=D&amp;source=editors&amp;ust=1760085499752880&amp;usg=AOvVaw3AoiWKXhZ2ZhQbiSCpLB3B">165305</a></span></p></td></tr><tr class="c13"><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">Microplate Reader</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">ThermoFischer</span></p></td><td class="c0" colspan="1" rowspan="1"><p class="c5"><span class="c4 c2">VL0000D0</span></p></td></tr></table><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.laaf4pj3jepy"><span class="c18 c2">Methods</span></h1><p class="c1"><span class="c3 c2">Based on the results of the BCA assay, all samples showed no significant difference in total protein content yield from each condition. As a result, no further dilution was required, and it was assumed that each isolated sample (i.e. cell-based, plasmid-based CFPP, and P-Gel-based CFPP) contained comparable amounts of carbonic anhydrase (CA). Because protein concentrations were consistent across all conditions, it can be inferred that all activity measurements reflected true differences in catalytic activity rather than variations in protein abundance.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">Buffer Exchange</span></p><p class="c1"><span class="c3 c2">To run an activity assay, we needed to ensure compatible buffers that did not give false positive results. Hence, all samples were buffer-exchanged into a zinc chloride buffer to maximize the amount of active CA. Zinc ions form an essential component of the CA active site, where they coordinate a hydroxide ion that facilitates the reversible hydration of carbon dioxide. Metal loss during purification or dialysis can substantially reduce activity; therefore, reintroduction of Zn(2+) through buffer exchange ensured proper metal coordination and maintained optimal conditions for enzymatic function during activity measurements.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c25">Activity Assay </span></p><p class="c1"><span class="c2">Enzymatic activity was assessed using a colorimetric CA Activity Assay Kit (Abcam, ab284550) based on the hydrolysis of </span><span class="c2 c9">p</span><span class="c2">-nitrophenyl acetate. Samples and controls were prepared in duplicate in a 96-well clear microplate and adjusted to a final volume of 95 &micro;l with assay buffer. Reactions were initiated by adding 5 &micro;l of substrate solution and monitored at 405 nm for one hour at room temperature. Hydrolysis of </span><span class="c2 c9">p</span><span class="c2">-nitrophenyl acetate produced </span><span class="c2 c9">p</span><span class="c2">-nitrophenol, which becomes bright yellow under alkaline conditions and exhibits strong absorbance at 405 nm. A standard curve generated using 0 - 40 nmol </span><span class="c2 c9">p</span><span class="c2">-nitrophenol per well was used to calculate enzyme activity after subtraction of background and inhibitor controls. Activities were expressed in milliunits per millilitre according to the manufacturer&rsquo;s protocol, representing the true catalytic performance of the isolated CA.</span></p><h1 class="c11" id="h.8tjmhav2sb8w"><span class="c2 c18">Results </span></h1><p class="c1"><span class="c2">An esterase activity assay was used to quantify the amount of active carbonic anhydrase in one cell-based and two cell-free (CFPP and plasmid only) protein production systems. As shown in Figure 9, both CFPP systems had similar activity readings with no significant difference observed. In contrast, the </span><span class="c2 c9">E. coli </span><span class="c2">cell-based expression method showed markedly lower activity, indicating that the CFPP-generated enzyme retained higher catalytic functionality. Importantly, because protein concentrations were considered normalized across all samples following BCA quantification and dilution to a common protein concentration, these activity differences are likely not attributable to differences in protein amount. Rather, they likely reflect true variations in enzyme quality and folding efficiency. This suggests that cell-free environments provide a more favourable context for maintaining enzyme conformation and activity compared to traditional cell-based production. This is likely due to the absence of intracellular stress responses, as carbonic anhydrase has been shown to potentially be toxic to </span><span class="c2 c9">E. Coli</span><span class="c3 c2">&nbsp;systems and may trigger stress response mechanisms that denature the protein and stop its activity (Snoeck et al., 2024).</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 260.00px;"><img alt="" src="images/exp/image5.png" style="width: 624.00px; height: 260.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c3 c2">Figure 9: Carbonic anhydrase activity assay raw absorbance readings (left) and calculated concentrations (right).</span></p><p class="c1 c6"><span class="c3 c2"></span></p><h1 class="c11" id="h.p0dvpc6f03wo"><span class="c2">Works Cited</span></h1><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2 c8">Vandeventer, P. E., Mejia, J., Nadim, A., Johal, M. S., &amp; Niemz, A. (2013). DNA adsorption to and elution from silica surfaces: Influence of amino acid buffers. </span><span class="c2 c9 c8">The Journal of Physical Chemistry B</span><span class="c2 c8">, </span><span class="c2 c9 c8">117</span><span class="c2 c8">(37), 10742-10749. </span><span class="c20 c2 c8"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1021/jp405753m&amp;sa=D&amp;source=editors&amp;ust=1760085499758318&amp;usg=AOvVaw2k5dVw9RGRPNl_789IzShA">https://doi.org/10.1021/jp405753m</a></span><span class="c4 c2">&nbsp;</span></p><p class="c1"><span class="c4 c2">&zwnj;</span></p><p class="c1"><span class="c2 c8">Promega. (2025). </span><span class="c2 c9 c8">DNA Purification.</span><span class="c2 c8">&nbsp;</span><span class="c20 c2 c8"><a class="c10" href="https://www.google.com/url?q=https://www.promega.ca/resources/guides/nucleic-acid-analysis/dna-purification/&amp;sa=D&amp;source=editors&amp;ust=1760085499758666&amp;usg=AOvVaw2Ud1p_5EpNafqMZGN7HrF7">https://www.promega.ca/resources/guides/nucleic-acid-analysis/dna-purification/</a></span></p><p class="c1 c6"><span class="c27 c2 c8 c44"></span></p><p class="c1"><span class="c2 c8 c27">Park, N., Um, S. H., Funabashi, H., Xu, J., &amp; Luo, D. (2009). A cell-free protein-producing gel. </span><span class="c2 c35 c9 c8">Nature materials</span><span class="c27 c2 c8">, </span><span class="c2 c9 c8 c35">8</span><span class="c27 c2 c8">(5), 432&ndash;437. </span><span class="c20 c2 c48 c8"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1038/nmat2419&amp;sa=D&amp;source=editors&amp;ust=1760085499759049&amp;usg=AOvVaw2-XWQkehUAskLGmvQuKbZ1">https://doi.org/10.1038/nmat2419</a></span><span class="c27 c2 c8">&nbsp;</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2 c8">Lee, P. Y., Costumbrado, J., Hsu, C. Y., &amp; Kim, Y. H. (2012). </span><span class="c2 c9 c8">Agarose gel electrophoresis for the separation of DNA fragments</span><span class="c2 c8">. </span><span class="c2 c9 c8">Journal of Visualized Experiments</span><span class="c2 c8">, (62), 3923. </span><span class="c20 c2 c8"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.3791/3923&amp;sa=D&amp;source=editors&amp;ust=1760085499759440&amp;usg=AOvVaw29wVGPTAFQIRoJkuXk773g">https://doi.org/10.3791/3923</a></span><span class="c4 c2">&nbsp;</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2 c8">Qiagen. (n.d.). </span><span class="c2 c9 c8">Quantification of DNA.</span><span class="c2 c8">&nbsp;</span><span class="c20 c2 c8"><a class="c10" href="https://www.google.com/url?q=https://www.qiagen.com/us/knowledge-and-support/knowledge-hub/bench-guide/dna/analysing-dna/quantification-of-dna&amp;sa=D&amp;source=editors&amp;ust=1760085499759814&amp;usg=AOvVaw1xsQ9Ogj2GSDiKXBQhcBm9">https://www.qiagen.com/us/knowledge-and-support/knowledge-hub/bench-guide/dna/analysing-dna/quantification-of-dna</a></span><span class="c4 c2">&nbsp;</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2 c8">Park, N., Kahn, J. S., Rice, E. J., Hartman, M. R., Funabashi, H., Xu, J., Um, S. H., &amp; Luo, D. (2009). High-yield cell-free protein production from P-gel. Nature protocols, 4(12), 1759&ndash;1770. </span><span class="c20 c2 c8"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1038/nprot.2009.174&amp;sa=D&amp;source=editors&amp;ust=1760085499760345&amp;usg=AOvVaw3aF71DYMBjVKAvkdsxsFRr">https://doi.org/10.1038/nprot.2009.174</a></span><span class="c4 c2">&nbsp;</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2 c8">ThermoFisher Scientific. (2022, November 3). Trust your SYBR Green qPCR Data. Behind the Bench. </span><span class="c20 c2 c8"><a class="c10" href="https://www.google.com/url?q=https://www.thermofisher.com/blog/behindthebench/trust-your-sybr-green-qpcr-data/&amp;sa=D&amp;source=editors&amp;ust=1760085499760661&amp;usg=AOvVaw0BFjJF2ysY2B5JbIVdjDMv">https://www.thermofisher.com/blog/behindthebench/trust-your-sybr-green-qpcr-data/</a></span><span class="c4 c2">&nbsp;</span></p><p class="c1 c6"><span class="c4 c2"></span></p><p class="c1"><span class="c2">Addgene. (2016). </span><span class="c2 c9">Pouring LB agar plates</span><span class="c2">. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://www.addgene.org/protocols/pouring-lb-agar-plates/%23:~:text%3DThe%2520addition%2520of%2520an%2520antibiotic,carrying%2520the%2520antibiotic%2520resistance%2520gene&amp;sa=D&amp;source=editors&amp;ust=1760085499761052&amp;usg=AOvVaw35QFcBGqHk9LdR3l8i9Pq1">https://www.addgene.org/protocols/pouring-lb-agar-plates/#:~:text=The%20addition%20of%20an%20antibiotic,carrying%20the%20antibiotic%20resistance%20gene</a></span><span class="c3 c2">. </span></p><p class="c1 c32 c6"><span class="c2 c3"></span></p><p class="c1"><span class="c2">Asif, A., Mohsin, H., Tanvir, R., &amp; Rehman, Y. (2017). Revisiting the mechanisms involved in calcium chloride induced bacterial transformation. </span><span class="c2 c9">Frontiers in Microbiology</span><span class="c2">, 8. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.3389/fmicb.2017.02169&amp;sa=D&amp;source=editors&amp;ust=1760085499761414&amp;usg=AOvVaw0NDvVD8fiuVn-rN-u5-j4Y">https://doi.org/10.3389/fmicb.2017.02169</a></span></p><p class="c1 c6 c32"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Bruslind, L. (2021, January 3). </span><span class="c2 c9">9: Microbial growth</span><span class="c2">. Biology LibreTexts. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://bio.libretexts.org/Bookshelves/Microbiology/Microbiology_(Bruslind)/09%253A_Microbial_Growth%23:~:text%3DOnce%2520cells%2520have%2520accumulated%2520all,utilize%2520cells%2520from%2520this%2520phase&amp;sa=D&amp;source=editors&amp;ust=1760085499761881&amp;usg=AOvVaw0ncvC7JTWZtyDauHLNW6aF">https://bio.libretexts.org/Bookshelves/Microbiology/Microbiology_(Bruslind)/09%3A_Microbial_Growth#:~:text=Once%20cells%20have%20accumulated%20all,utilize%20cells%20from%20this%20phase</a></span><span class="c3 c2">. </span></p><p class="c1 c32 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Hancock Lab. (n.d.). CaCl2 Transformation of E. coli. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://cmdr.ubc.ca/bobh/method/cacl2-transformation-of-e-coli/&amp;sa=D&amp;source=editors&amp;ust=1760085499762150&amp;usg=AOvVaw1d0T6k2_gCHEFjUEFmwJzZ">https://cmdr.ubc.ca/bobh/method/cacl2-transformation-of-e-coli/</a></span></p><p class="c1 c32 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Kanamycin. (2008). </span><span class="c2 c9">Tuberculosis</span><span class="c2">, 88(2), 117&ndash;118. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1016/s1472-9792(08)70012-x&amp;sa=D&amp;source=editors&amp;ust=1760085499762395&amp;usg=AOvVaw1sqaOGwcX5M2fdWo5JGHIA">https://doi.org/10.1016/s1472-9792(08)70012-x</a></span></p><p class="c1 c32 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Kobayashi, A., Hirakawa, H., Hirata, T., Nishino, K., &amp; Yamaguchi, A. (2006). Growth phase-dependent expression of drug exporters in </span><span class="c2 c9">escherichia</span><span class="c2">&nbsp;</span><span class="c2 c9">coli</span><span class="c2">&nbsp;and its contribution to drug tolerance. </span><span class="c2 c9">Journal of Bacteriology</span><span class="c2">, 188(16), 5693&ndash;5703. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1128/jb.00217-06&amp;sa=D&amp;source=editors&amp;ust=1760085499762843&amp;usg=AOvVaw0_MA7s-WMsOycPYxUSyIty">https://doi.org/10.1128/jb.00217-06</a></span><span class="c3 c2">&nbsp;</span></p><p class="c1 c32 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Kroemer, T. (n.d.-a). </span><span class="c2 c9">A quick overview of SOC Medium &amp; Competent Cell Recovery Medium</span><span class="c2">. GoldBio. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://goldbio.com/articles/article/a-quick-overview-of-soc-medium-and-competent-cell-recovery-medium?srsltid%3DAfmBOopJKQXHJyTDQJ-OoV5zdjFvmWVkQ3HDF4nozuPTUQgRHhhCgKJM&amp;sa=D&amp;source=editors&amp;ust=1760085499763433&amp;usg=AOvVaw3rxnJAcTctjtxJTs98rqnp">https://goldbio.com/articles/article/a-quick-overview-of-soc-medium-and-competent-cell-recovery-medium?srsltid=AfmBOopJKQXHJyTDQJ-OoV5zdjFvmWVkQ3HDF4nozuPTUQgRHhhCgKJM</a></span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Kroemer, T. (n.d.-b). </span><span class="c2 c9">How does IPTG induction work?</span><span class="c2">. GoldBio. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://goldbio.com/articles/article/how-does-iptg-induction-work?srsltid%3DAfmBOoovAaVw5oeeXsdDi39EC26p6fd7SFBvXl5Ixas4qU9rRCxwwx-v&amp;sa=D&amp;source=editors&amp;ust=1760085499764134&amp;usg=AOvVaw1UypzlBGokqGb9BvTFZHxm">https://goldbio.com/articles/article/how-does-iptg-induction-work?srsltid=AfmBOoovAaVw5oeeXsdDi39EC26p6fd7SFBvXl5Ixas4qU9rRCxwwx-v</a></span><span class="c3 c2">&nbsp;</span></p><p class="c1 c32 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Martin, K. (n.d.). </span><span class="c2 c9">Introduction to competent cells</span><span class="c2">. GoldBio. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://goldbio.com/articles/article/Introduction-to-Competent-Cells?srsltid%3DAfmBOorI2qYeSrnS4xvfEI3p_PmxmgqPIeEZXnFCvc70yJt6P-XlGypc&amp;sa=D&amp;source=editors&amp;ust=1760085499764835&amp;usg=AOvVaw3sLK6ju5pmuR50pu0A3hdm">https://goldbio.com/articles/article/Introduction-to-Competent-Cells?srsltid=AfmBOorI2qYeSrnS4xvfEI3p_PmxmgqPIeEZXnFCvc70yJt6P-XlGypc</a></span><span class="c3 c2">&nbsp;</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Gilchrist, A., &amp; Nobbs, J. (2017). Colorimetry, Theory. Encyclopedia of Spectroscopy and Spectrometry, 328&ndash;333. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1016/b978-0-12-803224-4.00124-2&amp;sa=D&amp;source=editors&amp;ust=1760085499765354&amp;usg=AOvVaw1hwV-rBnkEhQOLMgtBiRJS">https://doi.org/10.1016/b978-0-12-803224-4.00124-2</a></span><span class="c3 c2">&nbsp;</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Snoeck, S., Guidi, C., &amp; De Mey, M. (2024). &ldquo;Metabolic burden&rdquo; explained: stress symptoms and its related responses induced by (over)expression of (heterologous) proteins in Escherichia coli. Microbial Cell Factories, 23(1), 96. </span><span class="c20 c2"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.1186/s12934-024-02370-9&amp;sa=D&amp;source=editors&amp;ust=1760085499766024&amp;usg=AOvVaw1SsN08dzEV-CYlDG0fS7zn">https://doi.org/10.1186/s12934-024-02370-9</a></span><span class="c3 c2">&nbsp;</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c2">Uda, N. R., Seibert, V., Stenner-Liewen, F., M&uuml;ller, P., Herzig, P., Gondi, G., Zeidler, R., van Dijk, M., Zippelius, A., &amp; Renner, C. (2015). Esterase activity of carbonic anhydrases serves as surrogate for selecting antibodies blocking hydratase activity. Journal of Enzyme Inhibition and Medicinal Chemistry, 30(6), 955&ndash;960. </span><span class="c2 c20"><a class="c10" href="https://www.google.com/url?q=https://doi.org/10.3109/14756366.2014.1001754&amp;sa=D&amp;source=editors&amp;ust=1760085499766865&amp;usg=AOvVaw0lNVZMTj0MHd_v08xjK565">https://doi.org/10.3109/14756366.2014.1001754</a></span><span class="c3 c2">&nbsp;</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c2">&nbsp;</span></p><h1 class="c11" id="h.otd13511gl9g"><span class="c18 c2">Appendix </span></h1><p class="c1"><span class="c4 c25">Appendix A</span></p><h1 class="c29 c33" id="h.7omhyrriyc51"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 349.50px; height: 268.29px;"><img alt="" src="images/exp/image10.png" style="width: 349.50px; height: 268.29px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h1><h1 class="c29 c33" id="h.14i3z7ys9qn6"><span class="c2 c43">Figu</span><span class="c2 c43">re 10: Thermal profile for annealing X-DNA. The rate of temperature decrease throughout this annealing process is 3 &deg;C s</span><span class="c2 c40 c43">-1</span><span class="c2 c43">&nbsp;(the maximum rate of Eppendorf Mastercycler gradient machin</span><span class="c2 c43">es). This thermal profile is described further in Table 1. This figure was replicated from </span><span class="c2 c41">Park et al., 2009.</span></h1><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Appendix B</span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 590.50px; height: 352.03px;"><img alt="" src="images/exp/image14.png" style="width: 590.50px; height: 352.03px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c3 c2">Figure 11: Standard curve for BCA assay with concentration of protein on the X-axis and activity on the y-axis.</span></p><p class="c1 c6"><span class="c3 c2"></span></p><p class="c1"><span class="c3 c25">Appendix C</span></p><p class="c29"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 410.08px; height: 341.50px;"><img alt="" src="images/exp/image7.png" style="width: 410.08px; height: 341.50px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c29"><span class="c2">Figure 12: Standard curve for activity assay, with enzyme activity on the x-axis and absorbance value on the y-axis.</span></p></body></html>