<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=vzxKn6nZtIGKCOgTKQG8-3e6S71Rvb7XAk8J16JjOPI);ol.lst-kix_gk4pb4odi6c8-0.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-0 0}.lst-kix_j2yh5ht7lqr7-0>li:before{content:"\0025cf   "}ol.lst-kix_n928stnuoycl-5.start{counter-reset:lst-ctn-kix_n928stnuoycl-5 0}.lst-kix_gk4pb4odi6c8-4>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-4}ol.lst-kix_8larvs5s5y96-6.start{counter-reset:lst-ctn-kix_8larvs5s5y96-6 0}ul.lst-kix_updun7l5fb5w-8{list-style-type:none}.lst-kix_j2yh5ht7lqr7-4>li:before{content:"\0025cb   "}.lst-kix_j2yh5ht7lqr7-5>li:before{content:"\0025a0   "}ul.lst-kix_updun7l5fb5w-7{list-style-type:none}ul.lst-kix_updun7l5fb5w-6{list-style-type:none}ul.lst-kix_updun7l5fb5w-5{list-style-type:none}.lst-kix_j2yh5ht7lqr7-1>li:before{content:"\0025cb   "}.lst-kix_j2yh5ht7lqr7-2>li:before{content:"\0025a0   "}.lst-kix_j2yh5ht7lqr7-3>li:before{content:"\0025cf   "}.lst-kix_n928stnuoycl-2>li{counter-increment:lst-ctn-kix_n928stnuoycl-2}ol.lst-kix_7x4neuo2ittw-0.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-0 0}.lst-kix_8larvs5s5y96-3>li{counter-increment:lst-ctn-kix_8larvs5s5y96-3}ol.lst-kix_n928stnuoycl-0.start{counter-reset:lst-ctn-kix_n928stnuoycl-0 0}ol.lst-kix_gk4pb4odi6c8-5.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-5 0}.lst-kix_7x4neuo2ittw-2>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-2}.lst-kix_7x4neuo2ittw-5>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-5}.lst-kix_gwvkwwabtx1g-0>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-0}.lst-kix_9csuuaujo8s6-0>li:before{content:"\0025cf   "}.lst-kix_9csuuaujo8s6-1>li:before{content:"\0025cb   "}ol.lst-kix_7x4neuo2ittw-5.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-5 0}.lst-kix_gk4pb4odi6c8-8>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-8}ol.lst-kix_v8deognbazpq-8.start{counter-reset:lst-ctn-kix_v8deognbazpq-8 0}.lst-kix_9csuuaujo8s6-7>li:before{content:"\0025cb   "}.lst-kix_8larvs5s5y96-6>li{counter-increment:lst-ctn-kix_8larvs5s5y96-6}.lst-kix_9csuuaujo8s6-5>li:before{content:"\0025a0   "}.lst-kix_9csuuaujo8s6-3>li:before{content:"\0025cf   "}.lst-kix_49jfd78g267r-5>li:before{content:"\0025a0   "}.lst-kix_49jfd78g267r-3>li:before{content:"\0025cf   "}.lst-kix_49jfd78g267r-7>li:before{content:"\0025cb   "}.lst-kix_7x4neuo2ittw-5>li:before{content:"(" counter(lst-ctn-kix_7x4neuo2ittw-5,lower-roman) ") "}.lst-kix_8larvs5s5y96-7>li{counter-increment:lst-ctn-kix_8larvs5s5y96-7}.lst-kix_7x4neuo2ittw-7>li:before{content:"" counter(lst-ctn-kix_7x4neuo2ittw-7,lower-latin) ". "}.lst-kix_gk4pb4odi6c8-0>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-0}ol.lst-kix_8larvs5s5y96-4.start{counter-reset:lst-ctn-kix_8larvs5s5y96-4 0}ol.lst-kix_gwvkwwabtx1g-3.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-3 0}.lst-kix_v8deognbazpq-7>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-7,lower-latin) ". "}.lst-kix_n928stnuoycl-5>li{counter-increment:lst-ctn-kix_n928stnuoycl-5}.lst-kix_gwvkwwabtx1g-7>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-7}.lst-kix_7x4neuo2ittw-6>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-6}ol.lst-kix_gk4pb4odi6c8-2.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-2 0}.lst-kix_gk4pb4odi6c8-7>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-7}.lst-kix_gwvkwwabtx1g-8>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-8}.lst-kix_gk4pb4odi6c8-1>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-1}ol.lst-kix_8larvs5s5y96-1.start{counter-reset:lst-ctn-kix_8larvs5s5y96-1 0}ol.lst-kix_7x4neuo2ittw-2.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-2 0}ul.lst-kix_j2yh5ht7lqr7-1{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-0{list-style-type:none}.lst-kix_v8deognbazpq-5>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-5,lower-roman) ". "}ul.lst-kix_j2yh5ht7lqr7-7{list-style-type:none}.lst-kix_v8deognbazpq-6>li{counter-increment:lst-ctn-kix_v8deognbazpq-6}ul.lst-kix_j2yh5ht7lqr7-6{list-style-type:none}.lst-kix_7x4neuo2ittw-1>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-1}ul.lst-kix_u2ne9d4jiezd-0{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-8{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-1{list-style-type:none}.lst-kix_v8deognbazpq-3>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-3,decimal) ". "}ul.lst-kix_j2yh5ht7lqr7-3{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-2{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-2{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-3{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-5{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-4{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-4{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-5{list-style-type:none}.lst-kix_v8deognbazpq-1>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-1,lower-latin) ". "}ol.lst-kix_gwvkwwabtx1g-1.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-1 0}ul.lst-kix_updun7l5fb5w-0{list-style-type:none}.lst-kix_j2yh5ht7lqr7-8>li:before{content:"\0025a0   "}ul.lst-kix_updun7l5fb5w-4{list-style-type:none}.lst-kix_j2yh5ht7lqr7-6>li:before{content:"\0025cf   "}ul.lst-kix_updun7l5fb5w-3{list-style-type:none}ul.lst-kix_updun7l5fb5w-2{list-style-type:none}ul.lst-kix_updun7l5fb5w-1{list-style-type:none}.lst-kix_7x4neuo2ittw-3>li:before{content:"(" counter(lst-ctn-kix_7x4neuo2ittw-3,decimal) ") "}.lst-kix_7x4neuo2ittw-1>li:before{content:"" counter(lst-ctn-kix_7x4neuo2ittw-1,lower-latin) ") "}ol.lst-kix_8larvs5s5y96-3.start{counter-reset:lst-ctn-kix_8larvs5s5y96-3 0}ol.lst-kix_v8deognbazpq-1.start{counter-reset:lst-ctn-kix_v8deognbazpq-1 0}.lst-kix_8larvs5s5y96-0>li{counter-increment:lst-ctn-kix_8larvs5s5y96-0}ul.lst-kix_49jfd78g267r-4{list-style-type:none}ul.lst-kix_49jfd78g267r-3{list-style-type:none}ul.lst-kix_49jfd78g267r-6{list-style-type:none}ul.lst-kix_49jfd78g267r-5{list-style-type:none}ul.lst-kix_49jfd78g267r-0{list-style-type:none}ul.lst-kix_49jfd78g267r-2{list-style-type:none}ul.lst-kix_49jfd78g267r-1{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-3.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-3 0}ul.lst-kix_49jfd78g267r-8{list-style-type:none}ul.lst-kix_49jfd78g267r-7{list-style-type:none}.lst-kix_6a7xh489yycv-1>li:before{content:"\0025cb   "}ul.lst-kix_u2ne9d4jiezd-6{list-style-type:none}.lst-kix_mqvzwi63covm-1>li:before{content:"\0025cb   "}ul.lst-kix_u2ne9d4jiezd-7{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-8{list-style-type:none}.lst-kix_mqvzwi63covm-2>li:before{content:"\0025a0   "}.lst-kix_wr7pxauz585j-3>li:before{content:"\0025cf   "}.lst-kix_v8deognbazpq-0>li{counter-increment:lst-ctn-kix_v8deognbazpq-0}.lst-kix_gk4pb4odi6c8-7>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-7,lower-latin) ") "}ol.lst-kix_n928stnuoycl-8.start{counter-reset:lst-ctn-kix_n928stnuoycl-8 0}.lst-kix_6a7xh489yycv-4>li:before{content:"\0025cb   "}.lst-kix_mqvzwi63covm-6>li:before{content:"\0025cf   "}.lst-kix_gk4pb4odi6c8-8>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-8,lower-roman) ") "}.lst-kix_wr7pxauz585j-7>li:before{content:"\0025cb   "}.lst-kix_6a7xh489yycv-5>li:before{content:"\0025a0   "}.lst-kix_wr7pxauz585j-0>li:before{content:"\0025cf   "}.lst-kix_wr7pxauz585j-8>li:before{content:"\0025a0   "}ol.lst-kix_v8deognbazpq-6{list-style-type:none}ol.lst-kix_v8deognbazpq-7{list-style-type:none}.lst-kix_6a7xh489yycv-8>li:before{content:"\0025a0   "}ol.lst-kix_v8deognbazpq-8{list-style-type:none}.lst-kix_mqvzwi63covm-5>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-1>li:before{content:"\0025cb   "}.lst-kix_updun7l5fb5w-2>li:before{content:"\0025a0   "}ol.lst-kix_v8deognbazpq-0{list-style-type:none}ol.lst-kix_n928stnuoycl-2.start{counter-reset:lst-ctn-kix_n928stnuoycl-2 0}ol.lst-kix_v8deognbazpq-1{list-style-type:none}ol.lst-kix_v8deognbazpq-2{list-style-type:none}ol.lst-kix_v8deognbazpq-3{list-style-type:none}ol.lst-kix_v8deognbazpq-4{list-style-type:none}ol.lst-kix_v8deognbazpq-5{list-style-type:none}.lst-kix_wr7pxauz585j-4>li:before{content:"\0025cb   "}.lst-kix_gwvkwwabtx1g-1>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-1}ol.lst-kix_n928stnuoycl-7.start{counter-reset:lst-ctn-kix_n928stnuoycl-7 0}.lst-kix_v8deognbazpq-7>li{counter-increment:lst-ctn-kix_v8deognbazpq-7}.lst-kix_u2ne9d4jiezd-4>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-3>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-7>li:before{content:"-  "}ul.lst-kix_nrg3fbd7jit6-8{list-style-type:none}.lst-kix_u2ne9d4jiezd-0>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-8>li:before{content:"-  "}ul.lst-kix_nrg3fbd7jit6-7{list-style-type:none}ul.lst-kix_nrg3fbd7jit6-6{list-style-type:none}ul.lst-kix_nrg3fbd7jit6-5{list-style-type:none}ul.lst-kix_nrg3fbd7jit6-4{list-style-type:none}ol.lst-kix_v8deognbazpq-6.start{counter-reset:lst-ctn-kix_v8deognbazpq-6 0}ul.lst-kix_nrg3fbd7jit6-3{list-style-type:none}ul.lst-kix_nrg3fbd7jit6-2{list-style-type:none}ul.lst-kix_nrg3fbd7jit6-1{list-style-type:none}.lst-kix_8ra287xp8x8-0>li:before{content:"\0025cf   "}.lst-kix_updun7l5fb5w-5>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-6>li:before{content:"\0025cf   "}.lst-kix_8ra287xp8x8-1>li:before{content:"\0025cb   "}.lst-kix_rjrspob5h5d6-2>li:before{content:"\0025a0   "}.lst-kix_8ra287xp8x8-4>li:before{content:"\0025cb   "}.lst-kix_rjrspob5h5d6-3>li:before{content:"\0025cf   "}ol.lst-kix_n928stnuoycl-8{list-style-type:none}.lst-kix_8ra287xp8x8-8>li:before{content:"\0025a0   "}ol.lst-kix_n928stnuoycl-1.start{counter-reset:lst-ctn-kix_n928stnuoycl-1 0}ol.lst-kix_n928stnuoycl-7{list-style-type:none}ol.lst-kix_n928stnuoycl-4{list-style-type:none}ol.lst-kix_v8deognbazpq-7.start{counter-reset:lst-ctn-kix_v8deognbazpq-7 0}.lst-kix_8ra287xp8x8-5>li:before{content:"\0025a0   "}ol.lst-kix_n928stnuoycl-3{list-style-type:none}ol.lst-kix_n928stnuoycl-6{list-style-type:none}ol.lst-kix_n928stnuoycl-5{list-style-type:none}.lst-kix_49jfd78g267r-1>li:before{content:"\0025cb   "}ol.lst-kix_n928stnuoycl-0{list-style-type:none}.lst-kix_49jfd78g267r-0>li:before{content:"\0025cf   "}ol.lst-kix_n928stnuoycl-2{list-style-type:none}ol.lst-kix_n928stnuoycl-1{list-style-type:none}.lst-kix_rjrspob5h5d6-6>li:before{content:"\0025cf   "}ul.lst-kix_nrg3fbd7jit6-0{list-style-type:none}.lst-kix_rjrspob5h5d6-7>li:before{content:"\0025cb   "}ol.lst-kix_gk4pb4odi6c8-4.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-4 0}ol.lst-kix_v8deognbazpq-0.start{counter-reset:lst-ctn-kix_v8deognbazpq-0 0}.lst-kix_9csuuaujo8s6-6>li:before{content:"\0025cf   "}.lst-kix_v8deognbazpq-4>li{counter-increment:lst-ctn-kix_v8deognbazpq-4}ol.lst-kix_gk4pb4odi6c8-7.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-7 0}.lst-kix_9csuuaujo8s6-2>li:before{content:"\0025a0   "}.lst-kix_n928stnuoycl-7>li{counter-increment:lst-ctn-kix_n928stnuoycl-7}.lst-kix_49jfd78g267r-8>li:before{content:"\0025a0   "}.lst-kix_8larvs5s5y96-0>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-0,decimal) ". "}.lst-kix_i82rd4591l9b-5>li:before{content:"\0025a0   "}ul.lst-kix_8ra287xp8x8-1{list-style-type:none}ul.lst-kix_8ra287xp8x8-2{list-style-type:none}.lst-kix_49jfd78g267r-4>li:before{content:"\0025cb   "}ul.lst-kix_8ra287xp8x8-0{list-style-type:none}.lst-kix_7x4neuo2ittw-6>li:before{content:"" counter(lst-ctn-kix_7x4neuo2ittw-6,decimal) ". "}.lst-kix_gwvkwwabtx1g-6>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-6}.lst-kix_7x4neuo2ittw-4>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-4}.lst-kix_i82rd4591l9b-1>li:before{content:"\0025cb   "}ol.lst-kix_v8deognbazpq-2.start{counter-reset:lst-ctn-kix_v8deognbazpq-2 0}ol.lst-kix_n928stnuoycl-6.start{counter-reset:lst-ctn-kix_n928stnuoycl-6 0}.lst-kix_n928stnuoycl-5>li:before{content:"(" counter(lst-ctn-kix_n928stnuoycl-5,decimal) ") "}.lst-kix_8larvs5s5y96-5>li{counter-increment:lst-ctn-kix_8larvs5s5y96-5}.lst-kix_v8deognbazpq-2>li{counter-increment:lst-ctn-kix_v8deognbazpq-2}ol.lst-kix_v8deognbazpq-5.start{counter-reset:lst-ctn-kix_v8deognbazpq-5 0}.lst-kix_gwvkwwabtx1g-4>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-4}ol.lst-kix_8larvs5s5y96-7.start{counter-reset:lst-ctn-kix_8larvs5s5y96-7 0}ol.lst-kix_v8deognbazpq-3.start{counter-reset:lst-ctn-kix_v8deognbazpq-3 0}ol.lst-kix_8larvs5s5y96-8.start{counter-reset:lst-ctn-kix_8larvs5s5y96-8 0}.lst-kix_v8deognbazpq-6>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-6,decimal) ". "}.lst-kix_pquqx3i5yuly-6>li:before{content:"\0025cf   "}.lst-kix_1lwup76ts655-6>li:before{content:"\0025cf   "}ol.lst-kix_n928stnuoycl-3.start{counter-reset:lst-ctn-kix_n928stnuoycl-3 0}ol.lst-kix_gk4pb4odi6c8-8.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-8 0}ul.lst-kix_1lwup76ts655-8{list-style-type:none}.lst-kix_gk4pb4odi6c8-4>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-4,decimal) ") "}.lst-kix_nrg3fbd7jit6-6>li:before{content:"\0025cf   "}.lst-kix_v8deognbazpq-2>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-2,lower-roman) ". "}.lst-kix_1lwup76ts655-2>li:before{content:"\0025a0   "}.lst-kix_n928stnuoycl-1>li:before{content:"" counter(lst-ctn-kix_n928stnuoycl-1,lower-roman) ") "}ul.lst-kix_1lwup76ts655-3{list-style-type:none}ul.lst-kix_1lwup76ts655-2{list-style-type:none}ul.lst-kix_1lwup76ts655-1{list-style-type:none}ul.lst-kix_8ra287xp8x8-7{list-style-type:none}ul.lst-kix_1lwup76ts655-0{list-style-type:none}.lst-kix_gk4pb4odi6c8-0>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-0,upper-roman) ". "}ul.lst-kix_8ra287xp8x8-8{list-style-type:none}ul.lst-kix_1lwup76ts655-7{list-style-type:none}ul.lst-kix_8ra287xp8x8-5{list-style-type:none}ul.lst-kix_1lwup76ts655-6{list-style-type:none}.lst-kix_j2yh5ht7lqr7-7>li:before{content:"\0025cb   "}ul.lst-kix_8ra287xp8x8-6{list-style-type:none}ul.lst-kix_1lwup76ts655-5{list-style-type:none}ul.lst-kix_8ra287xp8x8-3{list-style-type:none}ul.lst-kix_1lwup76ts655-4{list-style-type:none}ul.lst-kix_8ra287xp8x8-4{list-style-type:none}ol.lst-kix_n928stnuoycl-4.start{counter-reset:lst-ctn-kix_n928stnuoycl-4 0}.lst-kix_7x4neuo2ittw-2>li:before{content:"" counter(lst-ctn-kix_7x4neuo2ittw-2,lower-roman) ") "}.lst-kix_nrg3fbd7jit6-2>li:before{content:"\0025a0   "}.lst-kix_pquqx3i5yuly-2>li:before{content:"\0025a0   "}ul.lst-kix_mqvzwi63covm-1{list-style-type:none}ul.lst-kix_mqvzwi63covm-2{list-style-type:none}ul.lst-kix_mqvzwi63covm-0{list-style-type:none}ul.lst-kix_mqvzwi63covm-5{list-style-type:none}ul.lst-kix_mqvzwi63covm-6{list-style-type:none}ul.lst-kix_mqvzwi63covm-3{list-style-type:none}ul.lst-kix_mqvzwi63covm-4{list-style-type:none}.lst-kix_gwvkwwabtx1g-7>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-7,lower-latin) ". "}ol.lst-kix_8larvs5s5y96-0.start{counter-reset:lst-ctn-kix_8larvs5s5y96-0 0}.lst-kix_n928stnuoycl-0>li{counter-increment:lst-ctn-kix_n928stnuoycl-0}.lst-kix_gwvkwwabtx1g-6>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-6,decimal) ". "}.lst-kix_gwvkwwabtx1g-8>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-8,lower-roman) ". "}.lst-kix_gwvkwwabtx1g-5>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-5,lower-roman) ". "}ol.lst-kix_gwvkwwabtx1g-2.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-2 0}ol.lst-kix_v8deognbazpq-4.start{counter-reset:lst-ctn-kix_v8deognbazpq-4 0}.lst-kix_8larvs5s5y96-1>li{counter-increment:lst-ctn-kix_8larvs5s5y96-1}.lst-kix_gwvkwwabtx1g-0>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-0,upper-latin) ". "}.lst-kix_gwvkwwabtx1g-1>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-1,lower-latin) ". "}.lst-kix_gk4pb4odi6c8-6>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-6}.lst-kix_gwvkwwabtx1g-3>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-3,decimal) ". "}ol.lst-kix_gk4pb4odi6c8-6.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-6 0}.lst-kix_gwvkwwabtx1g-2>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-2,lower-roman) ". "}.lst-kix_gwvkwwabtx1g-4>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-4,lower-latin) ". "}ol.lst-kix_7x4neuo2ittw-6.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-6 0}.lst-kix_v8deognbazpq-8>li{counter-increment:lst-ctn-kix_v8deognbazpq-8}.lst-kix_v8deognbazpq-1>li{counter-increment:lst-ctn-kix_v8deognbazpq-1}ol.lst-kix_8larvs5s5y96-5.start{counter-reset:lst-ctn-kix_8larvs5s5y96-5 0}ol.lst-kix_gk4pb4odi6c8-8{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-7{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-6{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-1{list-style-type:none}.lst-kix_8larvs5s5y96-7>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-7,lower-latin) ". "}ol.lst-kix_gk4pb4odi6c8-0{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-5{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-4{list-style-type:none}.lst-kix_8larvs5s5y96-8>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-8,lower-roman) ". "}ol.lst-kix_gk4pb4odi6c8-3{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-2{list-style-type:none}.lst-kix_8larvs5s5y96-3>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-3,decimal) ". "}.lst-kix_8larvs5s5y96-2>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-2,lower-roman) ". "}.lst-kix_8larvs5s5y96-4>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-4,lower-latin) ". "}.lst-kix_gk4pb4odi6c8-2>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-2}.lst-kix_8larvs5s5y96-6>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-6,decimal) ". "}.lst-kix_8larvs5s5y96-5>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-5,lower-roman) ". "}.lst-kix_i82rd4591l9b-8>li:before{content:"\0025a0   "}ul.lst-kix_rjrspob5h5d6-7{list-style-type:none}ul.lst-kix_rjrspob5h5d6-8{list-style-type:none}.lst-kix_i82rd4591l9b-4>li:before{content:"\0025cb   "}.lst-kix_i82rd4591l9b-6>li:before{content:"\0025cf   "}.lst-kix_8larvs5s5y96-1>li:before{content:"" counter(lst-ctn-kix_8larvs5s5y96-1,lower-latin) ". "}ul.lst-kix_rjrspob5h5d6-0{list-style-type:none}ul.lst-kix_rjrspob5h5d6-1{list-style-type:none}.lst-kix_i82rd4591l9b-0>li:before{content:"\0025cf   "}.lst-kix_i82rd4591l9b-2>li:before{content:"\0025a0   "}ul.lst-kix_rjrspob5h5d6-2{list-style-type:none}ul.lst-kix_rjrspob5h5d6-3{list-style-type:none}ul.lst-kix_rjrspob5h5d6-4{list-style-type:none}ul.lst-kix_rjrspob5h5d6-5{list-style-type:none}ul.lst-kix_rjrspob5h5d6-6{list-style-type:none}ol.lst-kix_7x4neuo2ittw-4.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-4 0}ul.lst-kix_pquqx3i5yuly-8{list-style-type:none}.lst-kix_n928stnuoycl-4>li:before{content:"(" counter(lst-ctn-kix_n928stnuoycl-4,lower-roman) ") "}ul.lst-kix_pquqx3i5yuly-4{list-style-type:none}ul.lst-kix_pquqx3i5yuly-5{list-style-type:none}ul.lst-kix_pquqx3i5yuly-6{list-style-type:none}ul.lst-kix_pquqx3i5yuly-7{list-style-type:none}ul.lst-kix_9csuuaujo8s6-1{list-style-type:none}.lst-kix_8larvs5s5y96-8>li{counter-increment:lst-ctn-kix_8larvs5s5y96-8}ul.lst-kix_9csuuaujo8s6-0{list-style-type:none}ul.lst-kix_9csuuaujo8s6-3{list-style-type:none}ul.lst-kix_9csuuaujo8s6-2{list-style-type:none}.lst-kix_n928stnuoycl-6>li:before{content:"" counter(lst-ctn-kix_n928stnuoycl-6,lower-latin) ". "}.lst-kix_n928stnuoycl-8>li:before{content:"" counter(lst-ctn-kix_n928stnuoycl-8,decimal) ". "}ul.lst-kix_9csuuaujo8s6-5{list-style-type:none}ul.lst-kix_9csuuaujo8s6-4{list-style-type:none}ul.lst-kix_9csuuaujo8s6-7{list-style-type:none}ul.lst-kix_9csuuaujo8s6-6{list-style-type:none}.lst-kix_v8deognbazpq-5>li{counter-increment:lst-ctn-kix_v8deognbazpq-5}ul.lst-kix_pquqx3i5yuly-0{list-style-type:none}ul.lst-kix_pquqx3i5yuly-1{list-style-type:none}ul.lst-kix_pquqx3i5yuly-2{list-style-type:none}ul.lst-kix_pquqx3i5yuly-3{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-0.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-0 0}.lst-kix_gwvkwwabtx1g-2>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-2}.lst-kix_pquqx3i5yuly-3>li:before{content:"\0025cf   "}.lst-kix_7x4neuo2ittw-0>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-0}.lst-kix_pquqx3i5yuly-5>li:before{content:"\0025a0   "}ol.lst-kix_gwvkwwabtx1g-0{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-1{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-2{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-3{list-style-type:none}.lst-kix_7x4neuo2ittw-7>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-7}ol.lst-kix_gwvkwwabtx1g-4{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-5{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-6{list-style-type:none}.lst-kix_1lwup76ts655-7>li:before{content:"\0025cb   "}.lst-kix_pquqx3i5yuly-7>li:before{content:"\0025cb   "}ol.lst-kix_gk4pb4odi6c8-1.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-1 0}.lst-kix_6a7xh489yycv-0>li:before{content:"\0025cf   "}.lst-kix_nrg3fbd7jit6-7>li:before{content:"\0025cb   "}.lst-kix_1lwup76ts655-1>li:before{content:"\0025cb   "}.lst-kix_1lwup76ts655-3>li:before{content:"\0025cf   "}.lst-kix_n928stnuoycl-2>li:before{content:"" counter(lst-ctn-kix_n928stnuoycl-2,decimal) ") "}.lst-kix_nrg3fbd7jit6-5>li:before{content:"\0025a0   "}.lst-kix_gk4pb4odi6c8-3>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-3,lower-latin) ") "}ol.lst-kix_gwvkwwabtx1g-7{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-8{list-style-type:none}ol.lst-kix_7x4neuo2ittw-1.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-1 0}.lst-kix_1lwup76ts655-5>li:before{content:"\0025a0   "}.lst-kix_n928stnuoycl-0>li:before{content:"" counter(lst-ctn-kix_n928stnuoycl-0,lower-latin) ") "}.lst-kix_gk4pb4odi6c8-1>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-1,upper-latin) ". "}.lst-kix_nrg3fbd7jit6-1>li:before{content:"\0025cb   "}.lst-kix_pquqx3i5yuly-1>li:before{content:"\0025cb   "}.lst-kix_nrg3fbd7jit6-3>li:before{content:"\0025cf   "}.lst-kix_n928stnuoycl-4>li{counter-increment:lst-ctn-kix_n928stnuoycl-4}ul.lst-kix_mqvzwi63covm-7{list-style-type:none}ol.lst-kix_8larvs5s5y96-2.start{counter-reset:lst-ctn-kix_8larvs5s5y96-2 0}ul.lst-kix_mqvzwi63covm-8{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-5.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-5 0}.lst-kix_6a7xh489yycv-3>li:before{content:"\0025cf   "}.lst-kix_gk4pb4odi6c8-6>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-6,lower-roman) ") "}.lst-kix_6a7xh489yycv-2>li:before{content:"\0025a0   "}.lst-kix_gk4pb4odi6c8-5>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-5,lower-latin) ") "}ol.lst-kix_7x4neuo2ittw-3.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-3 0}.lst-kix_mqvzwi63covm-0>li:before{content:"\0025cf   "}.lst-kix_gk4pb4odi6c8-3>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-3}.lst-kix_wr7pxauz585j-1>li:before{content:"\0025cb   "}.lst-kix_rjrspob5h5d6-0>li:before{content:"\0025cf   "}.lst-kix_wr7pxauz585j-2>li:before{content:"\0025a0   "}.lst-kix_wr7pxauz585j-6>li:before{content:"\0025cf   "}.lst-kix_8larvs5s5y96-2>li{counter-increment:lst-ctn-kix_8larvs5s5y96-2}.lst-kix_mqvzwi63covm-7>li:before{content:"\0025cb   "}.lst-kix_6a7xh489yycv-6>li:before{content:"\0025cf   "}.lst-kix_updun7l5fb5w-4>li:before{content:"\0025cb   "}.lst-kix_6a7xh489yycv-7>li:before{content:"\0025cb   "}.lst-kix_updun7l5fb5w-3>li:before{content:"\0025cf   "}.lst-kix_mqvzwi63covm-4>li:before{content:"\0025cb   "}.lst-kix_wr7pxauz585j-5>li:before{content:"\0025a0   "}.lst-kix_mqvzwi63covm-3>li:before{content:"\0025cf   "}.lst-kix_updun7l5fb5w-0>li:before{content:"\0025cf   "}.lst-kix_u2ne9d4jiezd-5>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-2>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-6>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-1>li:before{content:"-  "}ol.lst-kix_gwvkwwabtx1g-4.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-4 0}ol.lst-kix_7x4neuo2ittw-5{list-style-type:none}ol.lst-kix_7x4neuo2ittw-6{list-style-type:none}ul.lst-kix_9csuuaujo8s6-8{list-style-type:none}.lst-kix_n928stnuoycl-3>li{counter-increment:lst-ctn-kix_n928stnuoycl-3}.lst-kix_rjrspob5h5d6-5>li:before{content:"\0025a0   "}ol.lst-kix_7x4neuo2ittw-3{list-style-type:none}.lst-kix_mqvzwi63covm-8>li:before{content:"\0025a0   "}ol.lst-kix_7x4neuo2ittw-4{list-style-type:none}.lst-kix_rjrspob5h5d6-4>li:before{content:"\0025cb   "}.lst-kix_8ra287xp8x8-2>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-7>li:before{content:"\0025cb   "}ol.lst-kix_7x4neuo2ittw-7{list-style-type:none}ol.lst-kix_7x4neuo2ittw-8{list-style-type:none}ul.lst-kix_wr7pxauz585j-6{list-style-type:none}ul.lst-kix_wr7pxauz585j-5{list-style-type:none}.lst-kix_8ra287xp8x8-3>li:before{content:"\0025cf   "}ul.lst-kix_wr7pxauz585j-4{list-style-type:none}.lst-kix_rjrspob5h5d6-1>li:before{content:"\0025cb   "}ul.lst-kix_wr7pxauz585j-3{list-style-type:none}ul.lst-kix_wr7pxauz585j-2{list-style-type:none}ol.lst-kix_7x4neuo2ittw-1{list-style-type:none}ul.lst-kix_wr7pxauz585j-1{list-style-type:none}ol.lst-kix_7x4neuo2ittw-2{list-style-type:none}.lst-kix_updun7l5fb5w-8>li:before{content:"\0025a0   "}ul.lst-kix_wr7pxauz585j-0{list-style-type:none}ol.lst-kix_7x4neuo2ittw-0{list-style-type:none}ul.lst-kix_i82rd4591l9b-8{list-style-type:none}.lst-kix_8ra287xp8x8-7>li:before{content:"\0025cb   "}ul.lst-kix_i82rd4591l9b-6{list-style-type:none}.lst-kix_7x4neuo2ittw-8>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-8}ul.lst-kix_i82rd4591l9b-7{list-style-type:none}ul.lst-kix_i82rd4591l9b-4{list-style-type:none}ul.lst-kix_i82rd4591l9b-5{list-style-type:none}.lst-kix_8ra287xp8x8-6>li:before{content:"\0025cf   "}ul.lst-kix_i82rd4591l9b-2{list-style-type:none}ul.lst-kix_wr7pxauz585j-8{list-style-type:none}.lst-kix_gwvkwwabtx1g-3>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-3}ul.lst-kix_i82rd4591l9b-3{list-style-type:none}ul.lst-kix_wr7pxauz585j-7{list-style-type:none}ul.lst-kix_i82rd4591l9b-0{list-style-type:none}ul.lst-kix_i82rd4591l9b-1{list-style-type:none}.lst-kix_49jfd78g267r-2>li:before{content:"\0025a0   "}.lst-kix_rjrspob5h5d6-8>li:before{content:"\0025a0   "}.lst-kix_gk4pb4odi6c8-5>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-5}.lst-kix_9csuuaujo8s6-4>li:before{content:"\0025cb   "}.lst-kix_9csuuaujo8s6-8>li:before{content:"\0025a0   "}.lst-kix_i82rd4591l9b-7>li:before{content:"\0025cb   "}.lst-kix_gwvkwwabtx1g-5>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-5}.lst-kix_49jfd78g267r-6>li:before{content:"\0025cf   "}.lst-kix_7x4neuo2ittw-4>li:before{content:"(" counter(lst-ctn-kix_7x4neuo2ittw-4,lower-latin) ") "}ol.lst-kix_7x4neuo2ittw-7.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-7 0}.lst-kix_i82rd4591l9b-3>li:before{content:"\0025cf   "}.lst-kix_v8deognbazpq-3>li{counter-increment:lst-ctn-kix_v8deognbazpq-3}.lst-kix_7x4neuo2ittw-8>li:before{content:"" counter(lst-ctn-kix_7x4neuo2ittw-8,lower-roman) ". "}.lst-kix_n928stnuoycl-6>li{counter-increment:lst-ctn-kix_n928stnuoycl-6}.lst-kix_n928stnuoycl-8>li{counter-increment:lst-ctn-kix_n928stnuoycl-8}.lst-kix_v8deognbazpq-8>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-8,lower-roman) ". "}.lst-kix_n928stnuoycl-7>li:before{content:"" counter(lst-ctn-kix_n928stnuoycl-7,lower-roman) ". "}.lst-kix_7x4neuo2ittw-3>li{counter-increment:lst-ctn-kix_7x4neuo2ittw-3}ol.lst-kix_gwvkwwabtx1g-6.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-6 0}ol.lst-kix_gwvkwwabtx1g-7.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-7 0}ol.lst-kix_8larvs5s5y96-6{list-style-type:none}ol.lst-kix_8larvs5s5y96-7{list-style-type:none}ol.lst-kix_8larvs5s5y96-8{list-style-type:none}.lst-kix_pquqx3i5yuly-4>li:before{content:"\0025cb   "}.lst-kix_8larvs5s5y96-4>li{counter-increment:lst-ctn-kix_8larvs5s5y96-4}.lst-kix_v8deognbazpq-4>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-4,lower-latin) ". "}.lst-kix_pquqx3i5yuly-8>li:before{content:"\0025a0   "}.lst-kix_1lwup76ts655-8>li:before{content:"\0025a0   "}.lst-kix_n928stnuoycl-3>li:before{content:"(" counter(lst-ctn-kix_n928stnuoycl-3,lower-latin) ") "}.lst-kix_nrg3fbd7jit6-8>li:before{content:"\0025a0   "}.lst-kix_gk4pb4odi6c8-2>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-2,decimal) ". "}ol.lst-kix_gwvkwwabtx1g-8.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-8 0}ul.lst-kix_6a7xh489yycv-0{list-style-type:none}ul.lst-kix_6a7xh489yycv-1{list-style-type:none}ul.lst-kix_6a7xh489yycv-2{list-style-type:none}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_v8deognbazpq-0>li:before{content:"" counter(lst-ctn-kix_v8deognbazpq-0,decimal) ". "}ul.lst-kix_6a7xh489yycv-3{list-style-type:none}.lst-kix_1lwup76ts655-4>li:before{content:"\0025cb   "}ul.lst-kix_6a7xh489yycv-4{list-style-type:none}ul.lst-kix_6a7xh489yycv-5{list-style-type:none}ul.lst-kix_6a7xh489yycv-6{list-style-type:none}.lst-kix_nrg3fbd7jit6-0>li:before{content:"\0025cf   "}ul.lst-kix_6a7xh489yycv-7{list-style-type:none}ol.lst-kix_8larvs5s5y96-0{list-style-type:none}ul.lst-kix_6a7xh489yycv-8{list-style-type:none}ol.lst-kix_8larvs5s5y96-1{list-style-type:none}ol.lst-kix_8larvs5s5y96-2{list-style-type:none}ol.lst-kix_8larvs5s5y96-3{list-style-type:none}ol.lst-kix_8larvs5s5y96-4{list-style-type:none}ol.lst-kix_8larvs5s5y96-5{list-style-type:none}.lst-kix_nrg3fbd7jit6-4>li:before{content:"\0025cb   "}.lst-kix_7x4neuo2ittw-0>li:before{content:"" counter(lst-ctn-kix_7x4neuo2ittw-0,decimal) ") "}ol.lst-kix_7x4neuo2ittw-8.start{counter-reset:lst-ctn-kix_7x4neuo2ittw-8 0}.lst-kix_pquqx3i5yuly-0>li:before{content:"\0025cf   "}.lst-kix_n928stnuoycl-1>li{counter-increment:lst-ctn-kix_n928stnuoycl-1}.lst-kix_1lwup76ts655-0>li:before{content:"\0025cf   "}ol{margin:0;padding:0}table td,table th{padding:0}.c9{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:122.2pt;border-top-color:#000000;border-bottom-style:solid}.c46{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:78.8pt;border-top-color:#000000;border-bottom-style:solid}.c28{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:137.2pt;border-top-color:#000000;border-bottom-style:solid}.c35{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:233.2pt;border-top-color:#000000;border-bottom-style:solid}.c18{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#ffffff;border-top-width:1pt;border-right-width:1pt;border-left-color:#ffffff;vertical-align:top;border-right-color:#ffffff;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:468pt;border-top-color:#ffffff;border-bottom-style:solid}.c29{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:204pt;border-top-color:#000000;border-bottom-style:solid}.c33{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#ffffff;border-top-width:1pt;border-right-width:1pt;border-left-color:#ffffff;vertical-align:top;border-right-color:#ffffff;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:156pt;border-top-color:#ffffff;border-bottom-style:solid}.c16{margin-left:36pt;padding-top:18pt;padding-left:0pt;padding-bottom:10pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c11{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c25{padding-top:16pt;padding-bottom:4pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c0{color:#434343;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Manrope";font-style:normal}.c26{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:16pt;font-family:"Arial";font-style:normal}.c15{padding-top:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c3{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c40{padding-top:16pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c13{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c14{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c47{color:#000000;text-decoration:none;vertical-align:baseline;font-size:15pt;font-style:normal}.c24{color:#000000;text-decoration:none;vertical-align:baseline;font-size:10pt;font-style:normal}.c4{color:#000000;text-decoration:none;vertical-align:baseline;font-size:11pt;font-style:normal}.c10{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:center}.c48{color:#ff0000;text-decoration:none;vertical-align:baseline;font-style:normal}.c17{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c39{margin-left:-3pt;border-spacing:0;border-collapse:collapse;margin-right:auto}.c38{border-spacing:0;border-collapse:collapse;margin-right:auto}.c7{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c42{color:#434343;text-decoration:none;vertical-align:baseline;font-style:normal}.c34{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c30{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;text-decoration:underline}.c12{margin-left:21.3pt;padding-left:0pt}.c20{margin-left:36pt;padding-left:0pt}.c8{font-weight:700;font-family:"Manrope"}.c1{color:inherit;text-decoration:inherit}.c6{padding:0;margin:0}.c45{margin-left:71.4pt;text-indent:0.6pt}.c41{margin-left:144pt;text-indent:36pt}.c2{font-weight:400;font-family:"Manrope"}.c51{margin-left:144pt}.c36{font-size:10pt}.c43{font-style:italic}.c44{color:#0000ee}.c19{height:0pt}.c31{padding-left:0pt}.c37{margin-left:36pt}.c32{page-break-after:avoid}.c49{height:22pt}.c27{height:16pt}.c50{margin-left:71.4pt}.c5{height:11pt}.c23{margin-left:72pt}.c22{margin-left:35.4pt}.c21{font-size:11pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c34 doc-content"><h2 class="c11 c27" id="h.5avputbjm7uj"><span class="c4 c2"></span></h2><ol class="c6 lst-kix_gk4pb4odi6c8-0 start" start="1"><li class="c11 c20 li-bullet-0"><h2 id="h.8ln8qcyjd886" style="display:inline"><span class="c4 c8">Modelling Software and Evaluation Metrics</span></h2></li></ol><p class="c3"><span class="c2">Over the past decades, DNA has been used as a fundamental building block for synthesizing nanostructures with precisely controlled physicochemical properties for a wide range of biomedical applications (Ke et al., 2018). In the early stages, the absence of computational design tools limited DNA nanostructure research to relatively simple, single-stranded constructs (Tian et al, 2022) &nbsp;but r</span><span class="c2">ecent advances in computer-aided design (CAD) software and increased computational power have enabled the creation of far more complex architectures, driving rapid growth in this field.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">For our project, we took advantage of the various CAD softwares as described below:</span></p><p class="c3 c5"><span class="c4 c2"></span></p><ol class="c6 lst-kix_7x4neuo2ittw-0 start" start="1"><li class="c15 c12 li-bullet-0"><span class="c4 c8">oxView</span></li></ol><p class="c3"><span class="c4 c2">oxView is a web-based application that enables researchers to build and edit custom 3D shapes of DNA nanostructures (Bohlin et al., 2022). oxView supports both lattice-based designs and free-form geometries with complex shapes. It also provides features such as customizable strand color palettes, the ability to merge structures created in different design tools, interrogation of individual atoms, and bulk editing capabilities (Bohlin et al., 2022).</span></p><p class="c3 c5"><span class="c4 c2"></span></p><ol class="c6 lst-kix_7x4neuo2ittw-0" start="2"><li class="c15 c12 li-bullet-0"><span class="c8">oxDNA</span></li></ol><p class="c3"><span class="c4 c2">Designs created in oxView can be exported to oxDNA, a coarse-grained molecular dynamics (MD) simulator that tracks the positions and forces acting on clusters of atoms within the structure (Doye et al., 2023). oxDNA updates particle motion using either Langevin or Brownian dynamics, depending on the desired physical modeling conditions. Simulation outputs include metrics such as the residue mean square deviation (RMSD) of atomic positions, which quantifies the average deviation between molecular conformations over time compared to a reference state. A high RMSD indicates greater structural fluctuations and may suggest lower stability. oxDNA can also compute the Gibbs free energy along the simulation trajectory, where lower, or more negative, values are generally associated with more thermodynamically stable structures.</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><ol class="c6 lst-kix_7x4neuo2ittw-0" start="3"><li class="c12 c15 li-bullet-0"><span class="c4 c8">NUPACK</span></li></ol><p class="c3"><span class="c4 c2">While oxView and oxDNA focus primarily on stability analysis, other software tools such as NUPACK are designed to analyze the equilibrium base-pairing properties of specific DNA nanostructure assemblies. In other words, NUPACK can be used to evaluate the probability that a given set of DNA sequences will self-assemble into the desired shape. The software can also estimate the free energy associated with the predicted structure, providing additional information into its stability.</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Gibbs free energy represents the energy available for a system to do work. At chemical equilibrium, a negative Gibbs free energy indicates that a reaction is spontaneous. By measuring the Gibbs free energy of the three DNA monomers, we can assess their ability to spontaneously form the desired shapes. RMSD, on the other hand, quantifies how much a molecule&#39;s trajectory deviates from a reference conformation. Typically, this is the conformation with the lowest Gibbs free energy. This reference conformation is obtained by performing an initial Monte Carlo simulation to &ldquo;relax&rdquo; the DNA structures in oxView.</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Despite the availability of powerful CAD software for sequence design, structural modeling, and stability analysis, there is currently no software capable of directly evaluating macroscopic properties of DNA nanostructures, such as tensile strength or diffusive behavior. Instead, customized computational workflows using python to fill this gap due to their extensive scientific libraries for numerical analysis, statistical modeling, and data visualization.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><ol class="c6 lst-kix_gk4pb4odi6c8-0" start="2"><li class="c11 c20 li-bullet-0"><h2 id="h.7hkh84fuhm6n" style="display:inline"><span class="c4 c8">Motivation</span></h2></li></ol><p class="c3"><span class="c4 c2">The computational simulation resources described above were utilized to achieve three goals:</span></p><p class="c3 c5"><span class="c4 c2"></span></p><ul class="c6 lst-kix_mqvzwi63covm-0 start"><li class="c15 c20 li-bullet-0"><span class="c4 c2">First, to identify the most stable monomer, to serve as a building block for our hydrogel, among the 3 potential shapes: X, Y, and T.</span></li><li class="c15 c20 li-bullet-0"><span class="c4 c2">Second, to optimize the selected monomer for maximum stability and the highest likelihood of correct self-assembly.</span></li><li class="c15 c20 li-bullet-0"><span class="c4 c2">Finally, to investigate the macroscopic properties of the DNA hydrogel, specifically examining the accumulation of the produced carbonic anhydrase (CA) within the network.</span></li></ul><p class="c15 c5"><span class="c4 c2"></span></p><ol class="c6 lst-kix_gk4pb4odi6c8-0" start="3"><li class="c16 li-bullet-0"><h2 id="h.y10n9kh0kegr" style="display:inline"><span class="c8 c21">Methods</span></h2></li></ol><ol class="c6 lst-kix_gwvkwwabtx1g-0 start" start="1"><li class="c15 c12 c32 li-bullet-0"><h3 id="h.50t9mbmrgy" style="display:inline"><span class="c8 c21">Monomer Conformation</span></h3></li></ol><ul class="c6 lst-kix_i82rd4591l9b-0 start"><li class="c3 c12 li-bullet-0"><span class="c2">oxView: Constructed 3D models of the X, Y, and T-shaped DNA monomers</span><span class="c4 c2">. &nbsp;Within oxView, a Monte Carlo simulation was then performed on these models to identify the most energetically favorable conformations, a process referred to as monomer relaxation.</span></li><li class="c3 c12 li-bullet-0"><span class="c4 c2">oxDNA: Used to conduct stability assessments on the previously relaxed monomer designs.</span></li><li class="c3 c12 li-bullet-0"><span class="c2">NUPACK: simulated the monomer&rsquo;s </span><span class="c2">folding and ligation under experimental thermal conditions, as well as define free energy of overall structure </span></li></ul><p class="c3 c5"><span class="c2 c21 c48"></span></p><p class="c3"><span class="c4 c2">The simulated values for each monomer can be found summarized in Table 1.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c8">Table 1. </span><span class="c2">The Gibbs free energy results for each monomer confirmation (X, Y, Z). </span></p><table class="c39"><tr class="c19"><td class="c46" colspan="1" rowspan="1"><p class="c17 c5"><span class="c4 c2"></span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">X</span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">Y</span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">T</span></p></td></tr><tr class="c19"><td class="c46" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">RMSD/ time</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 142.00px; height: 122.67px;"><img alt="" src="images/simulations/image9.png" style="width: 162.95px; height: 122.67px; margin-left: -4.99px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 142.00px; height: 113.33px;"><img alt="" src="images/simulations/image16.png" style="width: 161.73px; height: 122.10px; margin-left: -4.62px; margin-top: -8.76px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 142.00px; height: 113.33px;"><img alt="" src="images/simulations/image15.png" style="width: 160.89px; height: 120.47px; margin-left: -5.52px; margin-top: -7.13px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c19"><td class="c46" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">Mean RMSD</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">1.13</span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">1.25</span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">1.4</span></p></td></tr><tr class="c19"><td class="c46" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">Predicted Conformation</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 149.18px; height: 153.06px;"><img alt="" src="images/simulations/image10.png" style="width: 206.03px; height: 175.19px; margin-left: -15.19px; margin-top: -14.06px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 167.00px; height: 142.67px;"><img alt="" src="images/simulations/image12.png" style="width: 207.27px; height: 176.34px; margin-left: 0.00px; margin-top: -16.13px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 139.06px; height: 154.52px;"><img alt="" src="images/simulations/image17.png" style="width: 186.29px; height: 175.11px; margin-left: 0.00px; margin-top: -14.11px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c19"><td class="c46" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">Predicted Gibbs Free Energy</span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">-127.30 kcal/mol</span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">-95.41 kcal/mol</span></p></td><td class="c28" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">-91.81 kcal/mol</span></p></td></tr></table><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">From our simulations, we observed the X-DNA has the lowest Gibbs free energy, indicating it is the most thermodynamically stable and most likely to assemble into the desired structure during a one-pot reaction. The RMSD trajectories also show that the X-DNA exhibited the least deviation from its relaxed conformation in comparison to the others. Thus indicates that the X-DNA undergoes minimal conformational shifts out of all considered geometries and is the most likely to retain its intended structure.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c13 c5"><span class="c24 c2"></span></p><ol class="c6 lst-kix_gwvkwwabtx1g-0" start="2"><li class="c12 c32 c40 li-bullet-0"><h3 id="h.f8o2uko2q71h" style="display:inline"><span class="c8 c21 c42">P-gel arm GC + Length </span></h3></li></ol><p class="c3"><span class="c2">Since </span><span class="c4 c2">we validate the hypothesis that the X-DNA is the most stable and promising candidate among the monomers hence we chose to proceed with further optimization exclusively for the X-DNA. </span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">We focused on two main characteristics of the monomer: </span></p><ol class="c6 lst-kix_v8deognbazpq-0 start" start="1"><li class="c3 c20 li-bullet-0"><span class="c4 c2">The length of the X-DNA arms </span></li><li class="c3 c20 li-bullet-0"><span class="c4 c2">The GC content of the X-DNA arms</span></li></ol><p class="c3 c5"><span class="c4 c2"></span></p><p class="c15"><span class="c8">Length Optimization</span></p><p class="c3"><span class="c2">The length of the monomer arm was evaluated based on the number of base pairs (bp) it was composed of, including the overhang. We tested five conditions: 20 bp, 30bp, 40 bp, 50 bp, and 60 bp. To ensure statistical significance, for each target sequence length, we generated and tested 10 different sequences. The spontaneous formation of the monomer from these sequences were simulated using NUPACK, where the molecules&rsquo; Gibbs free energy values were recorded. </span><span class="c2">The graph in Figure 1 shows a linear downward trend, and we observe that shorter monomers have higher Gibbs free energy</span><span class="c4 c2">, and are therefore less stable. </span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Additionally, it should be noted that, compared to monomers with longer arms, those with shorter arms behave more like rigid rods than flexible ropes. Park et al. (2009) speculated that the increased rigidity of these monomers could stiffen the gel and ultimately lead to a decrease in overall protein yield. However, a major challenge associated with increasing the arm length of the monomer lies in maintaining an appropriate hydrogel mesh size. The mesh must remain relatively small to prevent ribosomes, DNA polymerase, and other cellular machinery from diffusing into the gel and becoming trapped within it. Having a small mesh size ensures that all protein production occurs within the lysate rather than inside the gel.</span></p><p class="c3"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 385.33px;"><img alt="" src="images/simulations/image14.png" style="width: 624.00px; height: 385.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c13 c37"><span class="c8">Figure 1. </span><span class="c4 c2">Violin plot of free energy of the monomer structure. </span></p><p class="c13 c37"><span class="c4 c2">Based on length of monomer arm sequence.</span></p><p class="c3 c37 c5"><span class="c4 c2"></span></p><p class="c3 c5"><span class="c2 c24"></span></p><p class="c3"><span class="c8">GC Content Optimization </span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Similar to the arm length optimization, we generated at least 10 unique X-DNA sequences for each of the following 5 different GC content levels: 40%, 50%, 60%, 70%, and 80%. Unique X-DNA arm sequences were generated via Python script(s) developed by our team.These sequences were subsequently analyzed using NUPACK to assess the overall Gibbs free energy of each monomer. The graph in Figure 2, which shows the average free energy for each GC content condition, indicates that monomers with higher GC content have lower Gibbs free energy and are therefore more stable. However, monomers with higher GC content are also more prone to forming intramolecular bonds and undesirable internal loop structures, which may hinder the self-assembly process (Liu et al., 2014). We can observe that there is indeed a gradual slowing of the average free energy decrease as you continue to increase the GC content (%). Therefore, we chose a GC content of approximately 60-70% for our X-DNA monomer to maximize stability but still ensure that relatively no internal loops were forming.</span></p><p class="c13"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 385.33px;"><img alt="" src="images/simulations/image13.png" style="width: 624.00px; height: 385.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c13"><span class="c8">Figure 2. </span><span class="c4 c2">Violin plot of free energy of the monomer structure. </span></p><p class="c13"><span class="c4 c2">Based on GC content of monomer arm sequence.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><ol class="c6 lst-kix_gwvkwwabtx1g-0" start="3"><li class="c20 c25 li-bullet-0"><h3 id="h.y1zg9hy39ru8" style="display:inline"><span class="c42 c8 c21">Diffusion</span></h3></li></ol><p class="c3"><span class="c2">The hydroge</span><span class="c4 c2">l will be submerged in E. coli lysate to start the protein production process. Given the hydrogel&rsquo;s porosity and the concentration gradient between the E. coli cellular machinery and the produced CA in the lysate versus within the hydrogel, both components may diffuse into and accumulate within the hydrogel mesh. Such accumulation may reduce production efficiency. In this study, we aim to evaluate the diffusive properties of CA, holoenzyme RNA polymerase, ribosomes, and CA mRNA to determine whether significant aggregation occurs within the hydrogel mesh.</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c3"><span class="c2">To begin the process, we first assessed whether these biological molecules could diffuse into the hydrogel mesh based on their sizes. Using the number of 20 base pairs per arm for the X-DNA, the mesh size was calculated to be 13.6 nm (Vesely, 2004b). The size of human CA II mRNA, determined from whole genome sequencing, was found to be 15.16 &micro;m (Roth et al. 2018, </span><span class="c2 c43">CA2 Carbonic Anhydrase 2 [Homo Sapiens (Human)] - Gene - NCBI</span><span class="c4 c2">, n.d.). On the other hand, E. coli RNA polymerase has a molecular weight of 449 kDa (Finn, 2000), which corresponds to an approximate diameter of 12.98 nm (Fluidic Sciences, 2025). The E. coli ribosome has a diameter of 26 nm, and the size of human CA II protein is approximately 2.01 nm (Milo et al. Nucl. Acids Res, 2010).</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">From the size of the molecules, it is apparent that only RNA polymerase and CA II are small enough to potentially diffuse into the hydrogel mesh. However, experimental measurements indicate that the actual mesh size of X-DNA hydrogels is closer to 12.3 &plusmn; 1.4 nm (Um et al., 2006). Since larger molecules have a lower probability of diffusing through the mesh, and the size of E. coli RNA polymerase is very close to the upper limit of this measured range (Axpe et al., 2019), its diffusion into the network is likely minimal and can be considered negligible. Moreover, once RNA polymerase produces the mRNA templates for protein synthesis, the mRNA can be reused multiple times in the lysate. Therefore, even if the concentration of RNA polymerase in the lysate decreases, the overall protein yield would not be significantly affected.</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">In contrast, due to its much smaller size, CA II can readily pass through the mesh and diffuse into the DNA polymer network. If CA II were to accumulate within the hydrogel, it could lower the yield in the lysate solution. To evaluate whether it would be beneficial for us to extract CA II from the gel, we modeled the diffusion of the protein into the polymer network by numerically solving the Fickian diffusion equation.</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c15"><span class="c4 c8">System and Constraints</span></p><p class="c3"><span class="c2 c4">The p-gel is a rectangular prism, so CA can diffuse into the hydrogel through all six faces. Because there are only three unique face types, we model transport across those three faces and invoke symmetry by multiplying each contribution by two to obtain the total influx across each axis.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Given the very high solubility of CA (Fisher et al., 2012) and the long reaction time, we assume the lysate becomes well mixed. Thus, the CA concentration in the bulk lysate and at the hydrogel&ndash;lysate interface is spatially uniform. In addition, because CA has a long half-life (Ye &amp; Lu, 2014) relative to the experiment duration, we neglect enzymatic degradation over the entire run.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c15"><span class="c4 c8">Model Construction:</span></p><p class="c3"><span class="c4 c2">Considering that the dimensions of our hydrogel are on the order of millimeters, which is much larger than the nanometer-scale size of CA, we assume that the CA will not diffuse through the entire hydrogel. Under this assumption, and given that there is no outlet, generation, or consumption of CA within the gel, the change in CA concentration inside the hydrogel can be modeled using Fick&rsquo;s second law:</span></p><p class="c3 c5 c23"><span class="c4 c2"></span></p><p class="c3 c51"><img src="images/simulations/image1.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[1]</span></p><p class="c3 c5 c22"><span class="c4 c2"></span></p><p class="c14"><span class="c2">Since our hydrogel has a uniform mesh size and the reaction occurs under constant temperature and viscosity, we assume that the diffusion coefficient of CA is constant. To estimate the effective diffusion coefficient of CA in the DNA polymer network, we used an online calculator developed by Dr. Nate Richbourg [</span><span class="c2 c30"><a class="c1" href="https://www.google.com/url?q=https://hydrogeldesign.org/the-model/mesh-transport/&amp;sa=D&amp;source=editors&amp;ust=1760081344121771&amp;usg=AOvVaw1CF_u3UJ6uUeVoRkLWroF0">16</a></span><span class="c4 c2">]. This tool first applies the Stokes&ndash;Einstein equation to approximate the free diffusion coefficient of CA in solution.</span></p><p class="c3 c41"><img src="images/simulations/image2.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[2]</span></p><p class="c14"><span class="c4 c2">The value for the effective diffusion coefficient was then adjusted for the hydrogel mesh using obstruction theory and free void volume theory:</span></p><p class="c14 c22"><img src="images/simulations/image3.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3]</span></p><p class="c14"><span class="c4 c2">Given that CA is not significantly smaller than the mesh size, its diffusion is largely influenced by the solution-mediated transport through the mesh, with the effective diffusion coefficient primarily determined by obstruction effects.</span></p><p class="c14"><span class="c4 c2">There are several important polymer properties that affect the ability of a solute to diffuse through the hydrogel mesh. The first is junction functionality, which describes the number of chains converging at each junction point. By taking the center of the X-DNA as the junction point, our p-gel has a junction functionality of 4. The degree of polymerization between junctions is defined as the number of repeating units between each junction. In our system, each sugar phosphate backbone in DNA is treated as a unit. Once the X-DNA strands assemble into a mesh, there are 40 base pairs between junctions, giving a degree of polymerization of 40. Since each nucleotide in DNA is linked to its neighbors through two phosphodiester bonds, the number of backbone bonds per unit is 2.</span></p><p class="c14"><span class="c4 c2">Based on values reported by Um et al., 2006 hydrogels made with X-DNA monomers exhibit a monomer incorporation efficiency of approximately 83.5 &plusmn; 2.1%, corresponding to a swollen polymer volume fraction of 0.835. For the solute, the hydrodynamic radius is taken as the Stokes radius of carbonic anhydrase, 2.01nm (Milo et al., 2009). The polymer fibers, represented by DNA, have a radius of 1nm (Milo et al., 2009). The average bond length of the polymeric repeating unit in the hydrogel mesh was calculated by summing the average bond lengths for all atoms in a nucleotide and dividing by four, yielding a value of 0.137 nm (Heyrovska, 2008).</span></p><p class="c14"><span class="c2">Diffusion of CA into the mesh occurs during protein production, so the system temperature is assumed to be a physiological temperature of 37 &deg;C. With these parameters defined, the Flory characteristic of the polymer can be calculated using the following equation:</span></p><p class="c14 c45"><img src="images/simulations/image4.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3]</span></p><p class="c14"><span class="c2">The persistence length of DNA is approximately 50 nm, while the effective bond length can be approximated as the distance between nucleotides, 0.34 nm (Milo et al., 2009b, Wang, 2021). Using these parameters, the effective diffusion coefficient of carbonic anhydrase through the DNA hydrogel mesh was calculated to be </span><span class="c2">103 &mu;m&sup2;/s</span><span class="c4 c2">.</span></p><p class="c14"><span class="c4 c2">To validate these results, we compared our free diffusion (D&#8320;) value and calculated mesh size with values reported in the literature. The mesh size obtained from the transport calculator was 12.9 nm, which falls within the range reported by Park et al. (12.3 &plusmn; 1.4 nm). Additionally, the free diffusion coefficient of carbonic anhydrase &beta; reported in the literature is 107 &mu;m&sup2;/s (Milo et al., 2009b). This is slightly lower than the D&#8320; value of 127 &mu;m&sup2;/s predicted by mesh transport theory. The discrepancy may be attributed to structural differences, as the &beta; isoform is composed of two subunits and is therefore larger than the &alpha; isoform. Since the &alpha; isoform is smaller, it can move through the hydrogel mesh more easily, resulting in a higher diffusion coefficient.</span></p><p class="c14"><span class="c4 c2">With the effective diffusion constant determined, we proceeded to solve the diffusion equation numerically to model the change in CA concentration within the hydrogel mesh. Since the hydrogel dimensions are effectively infinite relative to the molecular size of CA, we assume that the enzyme movement only occurs in 1 direction, which is into the gel. Furthermore, we focused on the total amount of CA diffused into the gel rather than its spatial distribution, which allowed us to reduce the system to a one-dimensional model for each of the axes as shown in Equation 4, 5, and 6.</span></p><p class="c14 c45"><span class="c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="images/simulations/image5.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4]</span></p><p class="c14 c45"><span class="c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="images/simulations/image6.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[5]</span></p><p class="c14 c45"><span class="c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="images/simulations/image7.png"><span class="c4 c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[6]</span></p><p class="c14"><span class="c2">To solve the equation computationally, discretization methods were applied. The spatial dimension was discretized using the central difference approach, which combines the forward and backward difference approaches to achieve a more accurate approximation of derivatives (Equation 7). Here </span><span class="c2 c43">u</span><span class="c4 c2">&nbsp;represent the concentration of CA.</span></p><p class="c14 c50"><span class="c2">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 230.77px; height: 44.71px;"><img alt="{&quot;mathml&quot;:&quot;&lt;math style=\&quot;font-family:stix;font-size:16px;\&quot; xmlns=\&quot;http://www.w3.org/1998/Math/MathML\&quot;&gt;&lt;mstyle mathsize=\&quot;16px\&quot;&gt;&lt;mfrac&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mo&gt;&amp;#x2202;&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo&gt;&amp;#x2202;&lt;/mo&gt;&lt;msup&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;mfenced&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;/mfenced&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mfrac&gt;&lt;mrow&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;mo&gt;-&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;-&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo&gt;&amp;#x2206;&lt;/mo&gt;&lt;msup&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;/mstyle&gt;&lt;/math&gt;&quot;,&quot;truncated&quot;:false}" src="images/simulations/image18.png" style="width: 230.77px; height: 44.71px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title="fraction numerator partial differential squared u over denominator partial differential x squared end fraction open parentheses u subscript i to the power of j close parentheses equals fraction numerator u subscript i plus 1 end subscript to the power of j minus 2 u subscript i to the power of j plus u subscript i minus 1 end subscript to the power of j over denominator increment x squared end fraction"></span><span class="c4 c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [7]</span></p><p class="c14"><span class="c4 c2">For the temporal integration, the system was solved using the implicit multistep variable-order (BDF) method via Python&rsquo;s solve_ivp module. Because the system spans a large time frame, the PDE is extremely stiff. Explicit methods, such as Euler&rsquo;s method or the Runge-Kutta methods (RK23, RK45, DOP853), result in divergence and numerical instability. The BDF method was chosen because it provides enhanced stability for stiff problems while maintaining a relatively low computational cost. This implicit approach approximates the derivative at the current time step by using a backward extrapolation from previous time points, allowing larger time steps without losing stability. The curves showing the concentration profile of CA inside the gel for each of the axes was plotted. Figure 3 illustrates the simulated concentration profile of CA inside of a 1.25mm x 1.25mm x 5mm gel.</span></p><table class="c38"><tr class="c19"><td class="c33" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">a)</span></p><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 194.00px; height: 147.18px;"><img alt="" src="images/simulations/image8.png" style="width: 194.00px; height: 147.18px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c33" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">b)</span></p><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 194.00px; height: 145.18px;"><img alt="" src="images/simulations/image8.png" style="width: 194.00px; height: 145.18px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td><td class="c33" colspan="1" rowspan="1"><p class="c17"><span class="c4 c2">c)</span></p><p class="c17"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 194.00px; height: 144.18px;"><img alt="" src="images/simulations/image11.png" style="width: 194.00px; height: 144.18px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c49"><td class="c18" colspan="3" rowspan="1"><p class="c17"><span class="c8">Figure 3</span><span class="c4 c2">. Simulated concentration profile of CA inside a 1.25mm x 1.25mm x 5mm gel</span></p></td></tr></table><p class="c14"><span class="c4 c2">The system was solved in python and the full code can be found in the Supplementary Information section. Since diffusion systems are constrained by stability requirements, the time step was chosen to scale with &Delta;x&sup2; / D, ensuring that the numerical solution remains accurate and stable. The total CA concentration within the mesh for each hydrogel size is shown in Table 2.</span></p><p class="c14"><span class="c4 c2">Table 2. Showing the total amount of CA mass expected to accumulated inside the gel</span></p><table class="c38"><tr class="c19"><td class="c29" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">Hydrogel Dimension</span></p></td><td class="c35" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">Total Mass of CA Inside the Gel</span></p></td></tr><tr class="c19"><td class="c29" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">1.25mm x 1.25mm x 5mm</span></p></td><td class="c35" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">0.021mg</span></p></td></tr><tr class="c19"><td class="c29" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">1.00mm x 1.00mm x 5mm</span></p></td><td class="c35" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">0.019mg</span></p></td></tr><tr class="c19"><td class="c29" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">0.75mm x 0.75mm x 5mm</span></p></td><td class="c35" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">0.017mg</span></p></td></tr><tr class="c19"><td class="c29" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">0.5mm x 0.5mm x 5mm</span></p></td><td class="c35" colspan="1" rowspan="1"><p class="c10"><span class="c4 c2">0.015mg</span></p></td></tr></table><p class="c14 c5"><span class="c4 c2"></span></p><h2 class="c14 c32" id="h.gur8npn26ok6"><span class="c47 c2">Data</span></h2><ul class="c6 lst-kix_49jfd78g267r-0 start"><li class="c3 c20 li-bullet-0"><span class="c2 c36">P-GEL final data: </span><span class="c30 c44"><a class="c1" href="https://www.google.com/url?q=https://docs.google.com/spreadsheets/d/1GABRQLDVuGh6-3HYVJcTVszux_3Eq_NxuNCkJ2zR8Y4/edit?gid%3D1541399363%23gid%3D1541399363&amp;sa=D&amp;source=editors&amp;ust=1760081344139615&amp;usg=AOvVaw1g1YxnR3ApYU6CqVjntHnm">P-Gel Final Data - External</a></span></li><li class="c3 c20 li-bullet-0"><span class="c4 c2">P-GEL scripts:</span></li></ul><ul class="c6 lst-kix_49jfd78g267r-1 start"><li class="c3 c31 c23 li-bullet-0"><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://drive.google.com/drive/folders/1ALouxTPTtbia92AaGWR598KuBBVyecz2&amp;sa=D&amp;source=editors&amp;ust=1760081344139896&amp;usg=AOvVaw26ONqrS99pFPMkZ0A8g5a2">https://drive.google.com/drive/folders/1ALouxTPTtbia92AaGWR598KuBBVyecz2</a></span><span class="c4 c2">&nbsp;</span></li><li class="c3 c23 c31 li-bullet-0"><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://github.com/UBC-BIOMOD-Computational/TreeSAPP_Cost-Analysis/blob/main/GC_content_seq.py&amp;sa=D&amp;source=editors&amp;ust=1760081344140157&amp;usg=AOvVaw0RDwf_BV3M1N5PjXX4Rtv8">https://github.com/UBC-BIOMOD-Computational/TreeSAPP_Cost-Analysis/blob/main/GC_content_seq.py</a></span><span class="c4 c2">&nbsp;</span></li></ul><h2 class="c11" id="h.3nw4lvu0fql"><span class="c47 c2">References:</span></h2><p class="c3"><span class="c4 c2">Ke, Yonggang, et al. &ldquo;Structural DNA nanotechnology: Artificial nanostructures for Biomedical Research.&rdquo; Annual Review of Biomedical Engineering, vol. 20, no. 1, 4 June 2018, pp. 375&ndash;401, https://doi.org/10.1146/annurev-bioeng-062117-120904. </span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Park, Nokyoung et al. &ldquo;High-yield cell-free protein production from P-gel.&rdquo; Nature protocols vol. 4,12 (2009): 1759-70. doi:10.1038/nprot.2009.174</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c8">OXDNA: </span></p><p class="c3"><span class="c2">Erik Poppleton, Roger Romero, Aatmik Mallya, Lorenzo Rovigatti, Petr &Scaron;ulc, OxDNA.org: a public webserver for coarse-grained simulations of DNA and RNA nanostructures, Nucleic Acids Research, 2021;, gkab324, </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1093/nar/gkab324&amp;sa=D&amp;source=editors&amp;ust=1760081344141208&amp;usg=AOvVaw32OpJl1Y4_VT5RXs9qPRRt">https://doi.org/10.1093/nar/gkab324</a></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">B. E. K. Snodin, F. Randisi, M. Mosayebi, P. &Scaron;ulc, J. S. Schreck, F. Romano, T. E. Ouldridge, R. Tsukanov, E. Nir, A. A. Louis, J. P. K. Doye, J. Chem. Phys. 142, 234901 (2015) (DOI: 10.1063/1.4921957)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">P. &Scaron;ulc, F. Romano, T. E. Ouldridge, L. Rovigatti, J. P. K. Doye, A. A. Louis, J. Chem. Phys. 137, 135101 (2012) (DOI: 10.1063/1.475413)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">L. Rovigatti, P. &Scaron;ulc, I. Z. Reguly, F Romano, J. Comput. Chem. 36, 1 (2015) (DOI: 10.1002/jcc.23763)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">T. E. Ouldridge, A. A. Louis and J. P. K. Doye, J. Chem. Phys, 134, 085101 (2011) (DOI: 10.1063/1.3552946)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c8">OXVIEW:</span></p><p class="c3"><span class="c2">Bohlin, J., Matthies, M., Poppleton, E. et al. Design and simulation of DNA, RNA and hybrid protein&ndash;nucleic acid nanostructures with oxView. Nat Protoc 17, 1762&ndash;1788 (2022). </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1038/s41596-022-00688-5&amp;sa=D&amp;source=editors&amp;ust=1760081344142515&amp;usg=AOvVaw0UR_hJM2uGUbclxg8lE0fx">https://doi.org/10.1038/s41596-022-00688-5</a></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Erik Poppleton, Joakim Bohlin, Michael Matthies, Shuchi Sharma, Fei Zhang, Petr &Scaron;ulc, Design, optimization and analysis of large DNA and RNA nanostructures through interactive visualization, editing and molecular simulation, Nucleic Acids Research, Volume 48, Issue 12, 09 July 2020, Page e72, </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1093/nar/gkaa417&amp;sa=D&amp;source=editors&amp;ust=1760081344142993&amp;usg=AOvVaw3FO5woRvNZNy416gMRYt74">https://doi.org/10.1093/nar/gkaa417</a></span><span class="c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c8">NUPACK:</span></p><p class="c3"><span class="c4 c2">M. E. Fornace, J. Huang, C. T. Newman, N. J. Porubsky, M. B. Pierce, N. A. Pierce. NUPACK: analysis and design of nucleic acid structures, devices, and systems. ChemRxiv, 2022.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">J. N. Zadeh, C. D. Steenberg, J. S. Bois, B. R. Wolfe, M. B. Pierce, A. R. Khan, R. M. Dirks, N. A. Pierce. NUPACK: analysis and design of nucleic acid systems. J Comput Chem, 32:170&ndash;173, 2011. (pdf)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">M. E. Fornace, N. J. Porubsky, and N. A. Pierce. A unified dynamic programming framework for the analysis of interacting nucleic acid strands: enhanced models, scalability, and speed. ACS Synth Bio, 9:2665-2678, 2020. (pdf, supp info)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">R. M. Dirks, J. S. Bois, J. M. Schaeffer, E. Winfree, and N. A. Pierce. Thermodynamic analysis of interacting nucleic acid strands. SIAM Rev, 49:65-88, 2007. (pdf)</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c8">GENERAL:</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Liu, Lei et al. &ldquo;The self-assembled behavior of DNA bases on the interface.&rdquo; </span><span class="c2 c43">International journal of molecular sciences</span><span class="c4 c2">&nbsp;vol. 15,2 1901-14. 27 Jan. 2014, doi:10.3390/ijms15021901</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Ke, Y., Castro, C., &amp; Choi, J. H. (2018). Structural DNA nanotechnology: Artificial nanostructures for biomedical research. Annual Review of Biomedical Engineering, 20(1), 375&ndash;401. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1146/annurev-bioeng-062117-120904&amp;sa=D&amp;source=editors&amp;ust=1760081344144617&amp;usg=AOvVaw39s-76DMG1z179KzV07Eew">https://doi.org/10.1146/annurev-bioeng-062117-120904</a></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Tian, T., Li, Y. &amp; Lin, Y. Prospects and challenges of dynamic DNA nanostructures in biomedical applications. Bone Res 10, 40 (2022). </span><span class="c2 c7"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1038/s41413-022-00212-1&amp;sa=D&amp;source=editors&amp;ust=1760081344144890&amp;usg=AOvVaw058NWqh1Z-K3NTRF56urH0">https://doi.org/10.1038/s41413-022-00212-1</a></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Bohlin, J., Matthies, M., Poppleton, E., Procyk, J., Mallya, A., Yan, H., &amp; &Scaron;ulc, P. (2022). Design and simulation of DNA, RNA and hybrid protein&ndash;nucleic acid nanostructures with oxView. Nature Protocols, 17(8), 1762&ndash;1788. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1038/s41596-022-00688-5&amp;sa=D&amp;source=editors&amp;ust=1760081344145242&amp;usg=AOvVaw0lI7FSXL-5diNgYnh6NKDz">https://doi.org/10.1038/s41596-022-00688-5</a></span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Doye, J. P. K., Fowler, H., Pre&scaron;ern, D., Bohlin, J., Rovigatti, L., Romano, F., &Scaron;ulc, P., Wong, C. K., Louis, A. A., Schreck, J. S., Engel, M. C., Matthies, M., Benson, E., Poppleton, E., &amp; Snodin, B. E. K. (2023). The OxDNA Coarse-Grained model as a tool to simulate DNA origami. Methods in Molecular Biology, 93&ndash;112. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1007/978-1-0716-3028-0_6&amp;sa=D&amp;source=editors&amp;ust=1760081344145730&amp;usg=AOvVaw0Ys67dfBJU6V_ICI0FAd5o">https://doi.org/10.1007/978-1-0716-3028-0_6</a></span><span class="c4 c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Calculating the root mean square deviation of atomic structures &mdash; MDAnalysis User Guide &nbsp;documentation. (n.d.). </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://userguide.mdanalysis.org/stable/examples/analysis/alignment_and_rms/rmsd.html&amp;sa=D&amp;source=editors&amp;ust=1760081344146054&amp;usg=AOvVaw1I8yzn9U5fm82G8O1OlbbQ">https://userguide.mdanalysis.org/stable/examples/analysis/alignment_and_rms/rmsd.html</a></span><span class="c4 c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Libretexts. (2023, January 30). Gibbs (Free) energy. Chemistry LibreTexts. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Thermodynamics/Energies_and_Potentials/Free_Energy/Gibbs_(Free)_Energy&amp;sa=D&amp;source=editors&amp;ust=1760081344146532&amp;usg=AOvVaw0WI7ncbDDwYyqgYhnhFRwk">https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Thermodynamics/Energies_and_Potentials/Free_Energy/Gibbs_(Free)_Energy</a></span><span class="c4 c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Vesely, P. (2004b). Molecular biology of the cell. By Bruce Alberts, Alexander Johnson, Julian Lewis, Martin Raff, Keith Roberts and Peter Walter. ISBN 0&#8208;8153&#8208;3218&#8208;1; hardback; 1,616 pages; $110.00 Garland Science Inc., New York, 2002. Scanning, 26(3), 153. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1002/sca.4950260309&amp;sa=D&amp;source=editors&amp;ust=1760081344146927&amp;usg=AOvVaw3qtw-qS6hW0jCF9ovXV3nb">https://doi.org/10.1002/sca.4950260309</a></span><span class="c4 c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Roth, E., Azaria, A. G., Girshevitz, O., Bitler, A., &amp; Garini, Y. (2018). Measuring the conformation and persistence length of Single-Stranded DNA using a DNA origami structure. Nano Letters, 18(11), 6703&ndash;6709. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1021/acs.nanolett.8b02093&amp;sa=D&amp;source=editors&amp;ust=1760081344147316&amp;usg=AOvVaw3BGvUBmv8bgg1qoy1hnhxh">https://doi.org/10.1021/acs.nanolett.8b02093</a></span><span class="c4 c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">(</span><span class="c2 c43">CA2 Carbonic Anhydrase 2 [Homo Sapiens (Human)] - Gene - NCBI</span><span class="c4 c2">, n.d.) </span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Finn, R. D. (2000). Escherichia coli RNA polymerase core and holoenzyme structures. The EMBO Journal, 19(24), 6833&ndash;6844. https://doi.org/10.1093/emboj/19.24.6833</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Fluidic Sciences. (2025, October 3). What is Hydrodynamic Radius | Molecular Weight to Hydrodynamic Radius Converter. Fluidic Sciences Ltd. https://fluidic.com/molecular-weight-to-hydrodynamic-radius-converter/</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Milo R, Jorgensen P, Moran U, Weber G, Springer M. BioNumbers--the database of key numbers in molecular and cell biology. Nucleic Acids Res. 2010 Jan;38(Database issue):D750-3. doi: 10.1093/nar/gkp889. Epub 2009 Oct 23. PMID: 19854939; PMCID: PMC2808940.</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Um, S. H., Lee, J. B., Park, N., Kwon, S. Y., Umbach, C. C., &amp; Luo, D. (2006). Enzyme-catalysed assembly of DNA hydrogel. Nature Materials, 5(10), 797&ndash;801. https://doi.org/10.1038/nmat1741</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Axpe, E., Chan, D., Offeddu, G. S., Chang, Y., Merida, D., Hernandez, H. L., &amp; Appel, E. A. (2019). A multiscale model for solute diffusion in hydrogels. Macromolecules, 52(18), 6889&ndash;6897. https://doi.org/10.1021/acs.macromol.9b00753</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Fisher, Z., Boone, C. D., Biswas, S. M., Venkatakrishnan, B., Aggarwal, M., Tu, C., Agbandje-McKenna, M., Silverman, D., &amp; McKenna, R. (2012). Kinetic and structural characterization of thermostabilized mutants of human carbonic anhydrase II. Protein Engineering Design and Selection, 25(7), 347&ndash;355. https://doi.org/10.1093/protein/gzs027</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c4 c2">Ye, X., &amp; Lu, Y. (2014). CO2 absorption into catalyzed potassium carbonate&ndash;bicarbonate solutions: Kinetics and stability of the enzyme carbonic anhydrase as a biocatalyst. Chemical Engineering Science, 116, 567&ndash;575. https://doi.org/10.1016/j.ces.2014.05.040</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Heyrovska, R. (2008). Structures of the Molecular Components in DNA and RNA with Bond Lengths Interpreted as Sums of Atomic Covalent Radii. The Open Structural Biology Journal, 2(1), 1&ndash;7. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.2174/1874199100802010001&amp;sa=D&amp;source=editors&amp;ust=1760081344149673&amp;usg=AOvVaw0UJ9frW9n0_OivTNVkZhPK">https://doi.org/10.2174/1874199100802010001</a></span><span class="c4 c2">&nbsp;</span></p><p class="c3 c5"><span class="c4 c2"></span></p><p class="c3"><span class="c2">Wang, M. D. (2021). Ruler of life. Nature Physics, 17(8), 976. </span><span class="c7 c2"><a class="c1" href="https://www.google.com/url?q=https://doi.org/10.1038/s41567-021-01300-5&amp;sa=D&amp;source=editors&amp;ust=1760081344149894&amp;usg=AOvVaw3GzqzQSEywsrEczsq3i_R6">https://doi.org/10.1038/s41567-021-01300-5</a></span><span class="c4 c2">&nbsp;</span></p><h2 class="c11" id="h.ps8j04cb17oi"><span class="c26">END</span></h2></body></html>