<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">ol.lst-kix_gk4pb4odi6c8-0.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-0 0}.lst-kix_j2yh5ht7lqr7-0>li:before{content:"\0025cf   "}.lst-kix_gk4pb4odi6c8-4>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-4}ul.lst-kix_updun7l5fb5w-8{list-style-type:none}.lst-kix_gwvkwwabtx1g-7>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-7,lower-latin) ". "}.lst-kix_j2yh5ht7lqr7-4>li:before{content:"\0025cb   "}.lst-kix_j2yh5ht7lqr7-5>li:before{content:"\0025a0   "}ul.lst-kix_updun7l5fb5w-7{list-style-type:none}ul.lst-kix_updun7l5fb5w-6{list-style-type:none}.lst-kix_gwvkwwabtx1g-6>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-6,decimal) ". "}.lst-kix_gwvkwwabtx1g-8>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-8,lower-roman) ". "}ul.lst-kix_updun7l5fb5w-5{list-style-type:none}.lst-kix_gwvkwwabtx1g-5>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-5,lower-roman) ". "}.lst-kix_j2yh5ht7lqr7-1>li:before{content:"\0025cb   "}.lst-kix_j2yh5ht7lqr7-2>li:before{content:"\0025a0   "}.lst-kix_j2yh5ht7lqr7-3>li:before{content:"\0025cf   "}ol.lst-kix_gwvkwwabtx1g-2.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-2 0}.lst-kix_gwvkwwabtx1g-0>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-0,upper-latin) ". "}.lst-kix_gwvkwwabtx1g-1>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-1,lower-latin) ". "}.lst-kix_gk4pb4odi6c8-6>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-6}.lst-kix_gwvkwwabtx1g-3>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-3,decimal) ". "}ol.lst-kix_gk4pb4odi6c8-6.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-6 0}.lst-kix_gwvkwwabtx1g-2>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-2,lower-roman) ". "}.lst-kix_gwvkwwabtx1g-4>li:before{content:"" counter(lst-ctn-kix_gwvkwwabtx1g-4,lower-latin) ". "}ol.lst-kix_gk4pb4odi6c8-5.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-5 0}ol.lst-kix_gk4pb4odi6c8-8{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-7{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-6{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-1{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-0{list-style-type:none}.lst-kix_gwvkwwabtx1g-0>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-0}ol.lst-kix_gk4pb4odi6c8-5{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-4{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-3{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-2{list-style-type:none}.lst-kix_9csuuaujo8s6-0>li:before{content:"\0025cf   "}.lst-kix_9csuuaujo8s6-1>li:before{content:"\0025cb   "}.lst-kix_gk4pb4odi6c8-2>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-2}.lst-kix_gk4pb4odi6c8-8>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-8}.lst-kix_9csuuaujo8s6-7>li:before{content:"\0025cb   "}.lst-kix_9csuuaujo8s6-5>li:before{content:"\0025a0   "}ul.lst-kix_rjrspob5h5d6-7{list-style-type:none}ul.lst-kix_rjrspob5h5d6-8{list-style-type:none}.lst-kix_9csuuaujo8s6-3>li:before{content:"\0025cf   "}ul.lst-kix_rjrspob5h5d6-0{list-style-type:none}ul.lst-kix_rjrspob5h5d6-1{list-style-type:none}ul.lst-kix_rjrspob5h5d6-2{list-style-type:none}ul.lst-kix_rjrspob5h5d6-3{list-style-type:none}ul.lst-kix_rjrspob5h5d6-4{list-style-type:none}ul.lst-kix_rjrspob5h5d6-5{list-style-type:none}ul.lst-kix_rjrspob5h5d6-6{list-style-type:none}.lst-kix_gk4pb4odi6c8-0>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-0}ul.lst-kix_pquqx3i5yuly-8{list-style-type:none}ul.lst-kix_pquqx3i5yuly-4{list-style-type:none}ul.lst-kix_pquqx3i5yuly-5{list-style-type:none}ul.lst-kix_pquqx3i5yuly-6{list-style-type:none}ul.lst-kix_pquqx3i5yuly-7{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-3.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-3 0}ul.lst-kix_9csuuaujo8s6-1{list-style-type:none}ul.lst-kix_9csuuaujo8s6-0{list-style-type:none}ul.lst-kix_9csuuaujo8s6-3{list-style-type:none}ul.lst-kix_9csuuaujo8s6-2{list-style-type:none}ul.lst-kix_9csuuaujo8s6-5{list-style-type:none}ul.lst-kix_9csuuaujo8s6-4{list-style-type:none}ul.lst-kix_9csuuaujo8s6-7{list-style-type:none}ul.lst-kix_9csuuaujo8s6-6{list-style-type:none}.lst-kix_gwvkwwabtx1g-7>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-7}ul.lst-kix_pquqx3i5yuly-0{list-style-type:none}ul.lst-kix_pquqx3i5yuly-1{list-style-type:none}ul.lst-kix_pquqx3i5yuly-2{list-style-type:none}ul.lst-kix_pquqx3i5yuly-3{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-2.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-2 0}.lst-kix_gk4pb4odi6c8-7>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-7}ol.lst-kix_gwvkwwabtx1g-0.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-0 0}.lst-kix_gwvkwwabtx1g-8>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-8}.lst-kix_gk4pb4odi6c8-1>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-1}.lst-kix_gwvkwwabtx1g-2>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-2}.lst-kix_pquqx3i5yuly-3>li:before{content:"\0025cf   "}.lst-kix_pquqx3i5yuly-5>li:before{content:"\0025a0   "}ol.lst-kix_gwvkwwabtx1g-0{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-1{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-1{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-0{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-2{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-3{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-4{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-5{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-6{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-7{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-6{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-0{list-style-type:none}.lst-kix_1lwup76ts655-7>li:before{content:"\0025cb   "}ul.lst-kix_j2yh5ht7lqr7-8{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-1{list-style-type:none}.lst-kix_pquqx3i5yuly-7>li:before{content:"\0025cb   "}ul.lst-kix_j2yh5ht7lqr7-3{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-2{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-2{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-3{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-5{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-4{list-style-type:none}ul.lst-kix_j2yh5ht7lqr7-4{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-5{list-style-type:none}ol.lst-kix_gk4pb4odi6c8-1.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-1 0}.lst-kix_6a7xh489yycv-0>li:before{content:"\0025cf   "}.lst-kix_1lwup76ts655-1>li:before{content:"\0025cb   "}.lst-kix_1lwup76ts655-3>li:before{content:"\0025cf   "}ol.lst-kix_gwvkwwabtx1g-1.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-1 0}.lst-kix_gk4pb4odi6c8-3>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-3,lower-latin) ") "}ul.lst-kix_updun7l5fb5w-0{list-style-type:none}ol.lst-kix_gwvkwwabtx1g-7{list-style-type:none}.lst-kix_j2yh5ht7lqr7-8>li:before{content:"\0025a0   "}ol.lst-kix_gwvkwwabtx1g-8{list-style-type:none}.lst-kix_1lwup76ts655-5>li:before{content:"\0025a0   "}ul.lst-kix_updun7l5fb5w-4{list-style-type:none}.lst-kix_j2yh5ht7lqr7-6>li:before{content:"\0025cf   "}ul.lst-kix_updun7l5fb5w-3{list-style-type:none}ul.lst-kix_updun7l5fb5w-2{list-style-type:none}.lst-kix_gk4pb4odi6c8-1>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-1,upper-latin) ". "}ul.lst-kix_updun7l5fb5w-1{list-style-type:none}.lst-kix_pquqx3i5yuly-1>li:before{content:"\0025cb   "}ol.lst-kix_gwvkwwabtx1g-5.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-5 0}.lst-kix_6a7xh489yycv-3>li:before{content:"\0025cf   "}ol.lst-kix_gk4pb4odi6c8-3.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-3 0}.lst-kix_gk4pb4odi6c8-6>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-6,lower-roman) ") "}.lst-kix_6a7xh489yycv-1>li:before{content:"\0025cb   "}.lst-kix_6a7xh489yycv-2>li:before{content:"\0025a0   "}.lst-kix_gk4pb4odi6c8-5>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-5,lower-latin) ") "}ul.lst-kix_u2ne9d4jiezd-6{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-7{list-style-type:none}ul.lst-kix_u2ne9d4jiezd-8{list-style-type:none}.lst-kix_gk4pb4odi6c8-3>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-3}.lst-kix_wr7pxauz585j-1>li:before{content:"\0025cb   "}.lst-kix_wr7pxauz585j-3>li:before{content:"\0025cf   "}.lst-kix_rjrspob5h5d6-0>li:before{content:"\0025cf   "}.lst-kix_gk4pb4odi6c8-7>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-7,lower-latin) ") "}.lst-kix_wr7pxauz585j-2>li:before{content:"\0025a0   "}.lst-kix_wr7pxauz585j-6>li:before{content:"\0025cf   "}.lst-kix_6a7xh489yycv-4>li:before{content:"\0025cb   "}.lst-kix_gk4pb4odi6c8-8>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-8,lower-roman) ") "}.lst-kix_wr7pxauz585j-7>li:before{content:"\0025cb   "}.lst-kix_updun7l5fb5w-4>li:before{content:"\0025cb   "}.lst-kix_6a7xh489yycv-5>li:before{content:"\0025a0   "}.lst-kix_6a7xh489yycv-6>li:before{content:"\0025cf   "}.lst-kix_wr7pxauz585j-0>li:before{content:"\0025cf   "}.lst-kix_wr7pxauz585j-8>li:before{content:"\0025a0   "}.lst-kix_6a7xh489yycv-7>li:before{content:"\0025cb   "}.lst-kix_6a7xh489yycv-8>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-3>li:before{content:"\0025cf   "}.lst-kix_updun7l5fb5w-1>li:before{content:"\0025cb   "}.lst-kix_updun7l5fb5w-2>li:before{content:"\0025a0   "}.lst-kix_wr7pxauz585j-5>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-0>li:before{content:"\0025cf   "}.lst-kix_wr7pxauz585j-4>li:before{content:"\0025cb   "}.lst-kix_gwvkwwabtx1g-1>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-1}.lst-kix_u2ne9d4jiezd-4>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-5>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-2>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-3>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-6>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-7>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-0>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-1>li:before{content:"-  "}.lst-kix_u2ne9d4jiezd-8>li:before{content:"-  "}ol.lst-kix_gwvkwwabtx1g-4.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-4 0}.lst-kix_8ra287xp8x8-0>li:before{content:"\0025cf   "}.lst-kix_updun7l5fb5w-5>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-6>li:before{content:"\0025cf   "}ul.lst-kix_9csuuaujo8s6-8{list-style-type:none}.lst-kix_rjrspob5h5d6-5>li:before{content:"\0025a0   "}.lst-kix_rjrspob5h5d6-4>li:before{content:"\0025cb   "}.lst-kix_8ra287xp8x8-1>li:before{content:"\0025cb   "}.lst-kix_8ra287xp8x8-2>li:before{content:"\0025a0   "}.lst-kix_updun7l5fb5w-7>li:before{content:"\0025cb   "}ul.lst-kix_wr7pxauz585j-6{list-style-type:none}.lst-kix_rjrspob5h5d6-2>li:before{content:"\0025a0   "}.lst-kix_8ra287xp8x8-3>li:before{content:"\0025cf   "}.lst-kix_8ra287xp8x8-4>li:before{content:"\0025cb   "}ul.lst-kix_wr7pxauz585j-5{list-style-type:none}ul.lst-kix_wr7pxauz585j-4{list-style-type:none}.lst-kix_rjrspob5h5d6-1>li:before{content:"\0025cb   "}.lst-kix_rjrspob5h5d6-3>li:before{content:"\0025cf   "}ul.lst-kix_wr7pxauz585j-3{list-style-type:none}ul.lst-kix_wr7pxauz585j-2{list-style-type:none}ul.lst-kix_wr7pxauz585j-1{list-style-type:none}.lst-kix_updun7l5fb5w-8>li:before{content:"\0025a0   "}ul.lst-kix_wr7pxauz585j-0{list-style-type:none}.lst-kix_8ra287xp8x8-7>li:before{content:"\0025cb   "}.lst-kix_8ra287xp8x8-8>li:before{content:"\0025a0   "}.lst-kix_8ra287xp8x8-5>li:before{content:"\0025a0   "}.lst-kix_8ra287xp8x8-6>li:before{content:"\0025cf   "}ul.lst-kix_wr7pxauz585j-8{list-style-type:none}.lst-kix_gwvkwwabtx1g-3>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-3}ul.lst-kix_wr7pxauz585j-7{list-style-type:none}.lst-kix_rjrspob5h5d6-6>li:before{content:"\0025cf   "}.lst-kix_rjrspob5h5d6-7>li:before{content:"\0025cb   "}ol.lst-kix_gk4pb4odi6c8-4.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-4 0}.lst-kix_rjrspob5h5d6-8>li:before{content:"\0025a0   "}.lst-kix_gk4pb4odi6c8-5>li{counter-increment:lst-ctn-kix_gk4pb4odi6c8-5}.lst-kix_9csuuaujo8s6-6>li:before{content:"\0025cf   "}.lst-kix_9csuuaujo8s6-4>li:before{content:"\0025cb   "}.lst-kix_9csuuaujo8s6-8>li:before{content:"\0025a0   "}ol.lst-kix_gk4pb4odi6c8-7.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-7 0}.lst-kix_9csuuaujo8s6-2>li:before{content:"\0025a0   "}.lst-kix_gwvkwwabtx1g-5>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-5}ul.lst-kix_8ra287xp8x8-1{list-style-type:none}ul.lst-kix_8ra287xp8x8-2{list-style-type:none}ul.lst-kix_8ra287xp8x8-0{list-style-type:none}.lst-kix_gwvkwwabtx1g-6>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-6}.lst-kix_gwvkwwabtx1g-4>li{counter-increment:lst-ctn-kix_gwvkwwabtx1g-4}ol.lst-kix_gwvkwwabtx1g-6.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-6 0}ol.lst-kix_gwvkwwabtx1g-7.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-7 0}.lst-kix_pquqx3i5yuly-4>li:before{content:"\0025cb   "}.lst-kix_pquqx3i5yuly-6>li:before{content:"\0025cf   "}.lst-kix_1lwup76ts655-6>li:before{content:"\0025cf   "}ol.lst-kix_gk4pb4odi6c8-8.start{counter-reset:lst-ctn-kix_gk4pb4odi6c8-8 0}.lst-kix_pquqx3i5yuly-8>li:before{content:"\0025a0   "}.lst-kix_1lwup76ts655-8>li:before{content:"\0025a0   "}ul.lst-kix_1lwup76ts655-8{list-style-type:none}.lst-kix_gk4pb4odi6c8-2>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-2,decimal) ". "}.lst-kix_gk4pb4odi6c8-4>li:before{content:"(" counter(lst-ctn-kix_gk4pb4odi6c8-4,decimal) ") "}ol.lst-kix_gwvkwwabtx1g-8.start{counter-reset:lst-ctn-kix_gwvkwwabtx1g-8 0}.lst-kix_1lwup76ts655-2>li:before{content:"\0025a0   "}ul.lst-kix_1lwup76ts655-3{list-style-type:none}ul.lst-kix_1lwup76ts655-2{list-style-type:none}ul.lst-kix_1lwup76ts655-1{list-style-type:none}ul.lst-kix_8ra287xp8x8-7{list-style-type:none}ul.lst-kix_1lwup76ts655-0{list-style-type:none}.lst-kix_gk4pb4odi6c8-0>li:before{content:"" counter(lst-ctn-kix_gk4pb4odi6c8-0,upper-roman) ". "}ul.lst-kix_8ra287xp8x8-8{list-style-type:none}ul.lst-kix_6a7xh489yycv-0{list-style-type:none}ul.lst-kix_1lwup76ts655-7{list-style-type:none}ul.lst-kix_8ra287xp8x8-5{list-style-type:none}ul.lst-kix_6a7xh489yycv-1{list-style-type:none}ul.lst-kix_1lwup76ts655-6{list-style-type:none}ul.lst-kix_8ra287xp8x8-6{list-style-type:none}.lst-kix_j2yh5ht7lqr7-7>li:before{content:"\0025cb   "}ul.lst-kix_6a7xh489yycv-2{list-style-type:none}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}ul.lst-kix_1lwup76ts655-5{list-style-type:none}ul.lst-kix_8ra287xp8x8-3{list-style-type:none}ul.lst-kix_6a7xh489yycv-3{list-style-type:none}.lst-kix_1lwup76ts655-4>li:before{content:"\0025cb   "}ul.lst-kix_1lwup76ts655-4{list-style-type:none}ul.lst-kix_8ra287xp8x8-4{list-style-type:none}ul.lst-kix_6a7xh489yycv-4{list-style-type:none}ul.lst-kix_6a7xh489yycv-5{list-style-type:none}ul.lst-kix_6a7xh489yycv-6{list-style-type:none}ul.lst-kix_6a7xh489yycv-7{list-style-type:none}ul.lst-kix_6a7xh489yycv-8{list-style-type:none}.lst-kix_pquqx3i5yuly-2>li:before{content:"\0025a0   "}.lst-kix_pquqx3i5yuly-0>li:before{content:"\0025cf   "}.lst-kix_1lwup76ts655-0>li:before{content:"\0025cf   "}ol{margin:0;padding:0}table td,table th{padding:0}.c27{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:468pt;border-top-color:#000000;border-bottom-style:solid}.c11{margin-left:36pt;padding-top:16pt;padding-left:0pt;padding-bottom:4pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c16{margin-left:36pt;padding-top:18pt;padding-left:0pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c21{margin-left:71.4pt;padding-top:12pt;text-indent:0.6pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c14{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left;height:16pt}.c4{margin-left:36pt;padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;height:11pt}.c2{margin-left:35.4pt;padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left;height:11pt}.c3{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Times New Roman";font-style:normal}.c24{color:#434343;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Times New Roman";font-style:normal}.c0{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c1{-webkit-text-decoration-skip:none;color:#1155cc;font-weight:400;text-decoration:underline;text-decoration-skip-ink:none;font-size:12pt;font-family:"Times New Roman"}.c6{margin-left:36pt;padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c28{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Arial";font-style:normal}.c7{margin-left:35.4pt;padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c12{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Times New Roman";font-style:normal}.c30{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Arial";font-style:normal}.c8{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c23{padding-top:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c26{margin-left:36pt;border-spacing:0;border-collapse:collapse;margin-right:auto}.c25{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c32{background-color:#ffffff;max-width:468pt;padding:72pt 72pt 72pt 72pt}.c5{font-size:12pt;font-weight:400;font-family:"Times New Roman"}.c15{margin-left:144pt;text-indent:36pt}.c17{padding:0;margin:0}.c13{margin-left:72pt;padding-left:0pt}.c10{color:inherit;text-decoration:inherit}.c22{background-color:#ffff00}.c9{height:11pt}.c18{height:0pt}.c29{font-style:italic}.c31{color:#0000ff}.c20{margin-left:35.4pt}.c19{margin-left:72pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c32 doc-content"><h2 class="c14" id="h.5avputbjm7uj"><span class="c3"></span></h2><ol class="c17 lst-kix_gk4pb4odi6c8-0 start" start="1"><li class="c16 li-bullet-0"><h2 id="h.8ln8qcyjd886" style="display:inline"><span class="c12">Introduction</span></h2></li></ol><p class="c8 c20"><span class="c5">Over the past decades, DNA has been used as a fundamental building block for synthesizing nanostructures with precisely controlled physicochemical properties for a wide range of biomedical applications [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.annualreviews.org/content/journals/10.1146/annurev-bioeng-062117-120904&amp;sa=D&amp;source=editors&amp;ust=1758169724556861&amp;usg=AOvVaw1eCDRF-IrBGz6bBmznnSPQ">1</a></span><span class="c5">]. In the early stages, the absence of computational design tools limited DNA nanostructure research to relatively simple, single-stranded constructs [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.nature.com/articles/s41413-022-00212-1?fromPaywallRec%3Dfalse&amp;sa=D&amp;source=editors&amp;ust=1758169724557455&amp;usg=AOvVaw14xK18EjQq_-G8X9b5dHX8">2</a></span><span class="c3">]. Recent advances in computer-aided design (CAD) software and increased computational power have enabled the creation of far more complex architectures, driving rapid growth in this field.</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c5">Several open-source software tools have been developed to streamline the DNA nanostructure design process and improve the stability and quality of the final constructs. One example is oxView, a web-based application that enables researchers to build and edit custom 3D shapes of DNA nanostructures [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.nature.com/articles/s41596-022-00688-5&amp;sa=D&amp;source=editors&amp;ust=1758169724558975&amp;usg=AOvVaw1cakdyY0HR-De0VNZah47c">3</a></span><span class="c5">]. oxView supports both lattice-based designs and free-form geometries with complex shapes. It also provides features such as customizable strand color palettes, the ability to merge structures created in different design tools, interrogation of individual atoms, and bulk editing capabilities [</span><span class="c5 c31">3</span><span class="c3">].</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c5">Designs created in oxView can be exported to oxDNA, a coarse-grained molecular dynamics (MD) simulator that tracks the positions and forces acting on clusters of atoms within the structure [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://link.springer.com/protocol/10.1007/978-1-0716-3028-0_6%23Sec5&amp;sa=D&amp;source=editors&amp;ust=1758169724560514&amp;usg=AOvVaw3HQzc0rU31IV03kp6uWs7g">4</a></span><span class="c5">]. oxDNA updates particle motion using either Langevin or Brownian dynamics, depending on the desired physical modeling conditions. Simulation outputs include metrics such as the root mean square deviation (RMSD) of atomic positions, which quantifies the average deviation between molecular conformations over time [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://userguide.mdanalysis.org/stable/examples/analysis/alignment_and_rms/rmsd.html&amp;sa=D&amp;source=editors&amp;ust=1758169724561482&amp;usg=AOvVaw1AGd9DWkCzD1BeNnw28wj9">5</a></span><span class="c5">]. A high RMSD indicates greater structural fluctuations and may suggest reduced stability. oxDNA can also compute the Gibbs free energy along the simulation trajectory, where lower, or more negative, values are generally associated with more thermodynamically stable structures [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://chem.libretexts.org/Bookshelves/Physical_and_Theoretical_Chemistry_Textbook_Maps/Supplemental_Modules_(Physical_and_Theoretical_Chemistry)/Thermodynamics/Energies_and_Potentials/Free_Energy/Gibbs_(Free)_Energy&amp;sa=D&amp;source=editors&amp;ust=1758169724562513&amp;usg=AOvVaw3dNKzE2zzSGtlRlY0PEC6o">6</a></span><span class="c3">].</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c5">While oxView and oxDNA focus primarily on design and stability analysis, other software tools such as NUPACK are designed to analyze the equilibrium base-pairing properties of specific DNA nanostructure assemblies [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://docs.nupack.org/&amp;sa=D&amp;source=editors&amp;ust=1758169724563343&amp;usg=AOvVaw2cIBMd4m0Cqt--AMP8Dsnu">7</a></span><span class="c3">]. In other words, NUPACK can be used to evaluate the probability that a given set of DNA sequences will self-assemble into the desired shape. The software can also estimate the free energy associated with the predicted structure, providing additional information into its stability.</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c5">Despite the availability of powerful CAD software for sequence design, structural modeling, and stability analysis, there is currently no software capable of directly evaluating macroscopic properties of DNA nanostructures, such as tensile strength or diffusive behavior. Instead, customized computational workflows using programming languages such as MATLAB, Python, or C++ are used to fill this gap due to their extensive scientific libraries for numerical analysis, statistical modeling, and data visualization.</span></p><ol class="c17 lst-kix_gk4pb4odi6c8-0" start="2"><li class="c16 li-bullet-0"><h2 id="h.7hkh84fuhm6n" style="display:inline"><span class="c12">Background/Motivation</span></h2></li></ol><p class="c8 c20"><span class="c3">The computational simulation resources described above were utilized to achieve three goals:</span></p><ul class="c17 lst-kix_6a7xh489yycv-0 start"><li class="c13 c23 li-bullet-0"><span class="c3">First, to identify the most stable monomer, to serve as a building block for our hydrogel, among the 3 potential shapes: X, Y, and T.</span></li><li class="c23 c13 li-bullet-0"><span class="c3">Second, to optimize the selected monomer for maximum stability and the highest likelihood of correct self-assembly.</span></li><li class="c23 c13 li-bullet-0"><span class="c5">Finally, to investigate the macroscopic properties of the DNA hydrogel, specifically examining the accumulation of the produced carbonic anhydrase (CA) within the network.</span></li></ul><ol class="c17 lst-kix_gk4pb4odi6c8-0" start="3"><li class="c16 li-bullet-0"><h2 id="h.y10n9kh0kegr" style="display:inline"><span class="c12">Method</span></h2></li></ol><ol class="c17 lst-kix_gwvkwwabtx1g-0 start" start="1"><li class="c11 li-bullet-0"><h3 id="h.50t9mbmrgy" style="display:inline"><span class="c24">Monomer Conformation</span></h3></li></ol><p class="c8"><span class="c0">Stability assessments on different monomer designs were evaluated using the molecular dynamics (MD) software oxDNA, based on Langevin dynamics principles. MD simulations predict the motion of each atom over time, allowing us to track the trajectory of the molecule and compute the associated Gibbs free energy and Root Mean Square Deviation (RMSD) . These metrics were then used to assess and compare the structural stability of the DNA monomers.</span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8"><span class="c0">Gibbs free energy represents the energy available for a system to do work. At chemical equilibrium, a negative Gibbs free energy indicates that a reaction is spontaneous []. By measuring the Gibbs free energy of the three DNA monomers, we can assess their ability to spontaneously form the desired shapes. RMSD, on the other hand, quantifies how much a molecule&#39;s trajectory deviates from a reference conformation. Typically, this is the conformation with the lowest Gibbs free energy. This reference conformation is obtained by performing an initial Monte Carlo simulation to &ldquo;relax&rdquo; the DNA structures in oxView.</span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8"><span class="c0">The Gibbs free energy results from our simulation are depicted in Figure ___.</span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8"><span class="c0">&lt;Insert figure&gt;</span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8"><span class="c0">Figure _____ shows that X-DNA has the lowest Gibbs free energy, indicating it is the most thermodynamically stable and most likely to assemble into the desired structure during a one-pot reaction.</span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8"><span class="c0">Observation of RMSD trajectory for each monomer shown in Figure____ further supports this. The X-DNA exhibited the least deviation from its relaxed conformation, indicating that the X-DNA undergoes minimal conformational shifts and is the most likely to retain its intended structure.</span></p><p class="c8 c9"><span class="c0"></span></p><p class="c8 c9"><span class="c0"></span></p><ol class="c17 lst-kix_gwvkwwabtx1g-0" start="2"><li class="c11 li-bullet-0"><h3 id="h.f8o2uko2q71h" style="display:inline"><span class="c24">P-gel arm GC + Length </span></h3></li></ol><p class="c6"><span class="c3">Our workflow was separated into two main processes: re-assessing the stability of DNA monomers with different geometry, and optimizing the stability of the monomers by adjusting the number of base pairs in the monomer&rsquo;s arms and the GC content of each strand of the monomer.</span></p><p class="c4"><span class="c3"></span></p><p class="c6"><span class="c12">Assessing Stability:</span></p><p class="c6"><span class="c3">We first constructed 3 DNA structures: X-shaped monomer, Y-shaped monomer, and T-shaped monomer using oxView. The structures were relaxed into the conformations with lowest Gibbs free energy level. The relaxed structures were then imputed into oxDNA where molecular dynamics simulations using a Langevin thermostat were carried out. The trajectory of the molecule was recorded, and was analysed for the average level of Gibbs free energy and RMSD. The structure with the lowest Gibbs free energy across the trajectory and the lowest RMSD is considered to be the most stable and selected for further optimization.</span></p><p class="c4"><span class="c3"></span></p><p class="c6"><span class="c12">Optimization:</span></p><p class="c6"><span class="c5">The goal of the optimization process was to improve the structural stability of our monomers by varying two physical parameters: the number of base pairs per arm and the GC content. We first developed a Python script to generate 10 unique sequences for each experimental condition, varying arm length and GC content. These sequences were then analyzed using NUPACK, a software tool for modeling nucleic acid thermodynamics. For each monomer, we evaluated Gibbs free energy and the probability of correct base pairing. Monomers with lower Gibbs free energy and higher pairing probabilities were considered more stable. To assess the influence of each parameter on stability, we plotted the average Gibbs free energy across different arm lengths and GC contents. A flowchart summarizing this workflow is shown in Figure ___.</span></p><p class="c4"><span class="c3"></span></p><p class="c4"><span class="c3"></span></p><p class="c4"><span class="c3"></span></p><p class="c6"><span class="c3">Optimization</span></p><p class="c6"><span class="c3">Based on our stability analysis, we validate the hypothesis that the X-DNA is the most stable and promising candidate among the monomers hence we chose to proceed with optimization exclusively for the X-DNA. We then shift the focus on two main characteristics of the monomer: the length of the monomer&rsquo;s arms and its GC content.</span></p><p class="c4"><span class="c3"></span></p><p class="c6"><span class="c3">The length of the monomer arm was evaluated based on the number of base pairs (bp) it contained. We tested six conditions: 20 bp, 40 bp, 60 bp, 80 bp, 100 bp, and 120 bp.The spontaneous formation of the monomer from thesesequences were simulated using NUPACK, where the molecules&rsquo; Gibbs free energy values were recorded. A graph displaying the average free energy for each condition was generated, and the optimal arm length was identified to be the one with the lowest Gibbs free energy value. The graph in Figure ___ shows a linear downward trend, denoting that shorter monomers have lower Gibbs free energy. However, shorter X-DNA strands may also be more rigid. Park et al. (2009) suggested that increased hydrogel rigidity could negatively impact protein production, reducing the overall yield of functional protein. For this reason, we concluded that a strand length of 40-50 bp provides sufficient flexibility while maintaining relatively low Gibbs free energy.</span></p><p class="c4"><span class="c3"></span></p><p class="c4"><span class="c3"></span></p><table class="c26"><tr class="c18"><td class="c27" colspan="1" rowspan="1"><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 562.00px; height: 344.00px;"><img alt="" src="images/image12.png" style="width: 562.00px; height: 344.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 562.00px; height: 341.33px;"><img alt="" src="images/image10.png" style="width: 562.00px; height: 341.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c18"><td class="c27" colspan="1" rowspan="1"><p class="c25"><span class="c3">Figure </span></p></td></tr></table><p class="c4"><span class="c3"></span></p><p class="c4"><span class="c3"></span></p><p class="c6"><span class="c3">Similar to the arm length optimization, we generated unique X-DNA sequences for each of the followingdifferent GC content levels: 50%, 60%, 70%, and 80%. However, &nbsp;to ensure statistical significance, for each GC content level, we tested for 10 different sequences. Unique X-DNA arm sequences were generated by a Python script developed by our team. &nbsp;These sequences were subsequently analyzed using NUPACK to assess the overall Gibbs free energy of each monomer. The graph in Figure ___, which shows the average free energy for each GC content condition, indicates that monomers with higher GC content have lower Gibbs free energy and are therefore more stable. However, monomers with higher GC content are also more prone to forming intramolecular bonds and undesirable internal loop structures, which may hinder the self-assembly process []. To address this concern, we chose a moderate GC content of approximately 60&ndash;70% for our X-DNA monomer to maximize stability but still ensure that relatively no internal loops were forming..</span></p><p class="c4"><span class="c3"></span></p><p class="c4"><span class="c3"></span></p><table class="c26"><tr class="c18"><td class="c27" colspan="1" rowspan="1"><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 455.50px; height: 274.76px;"><img alt="" src="images/image9.png" style="width: 455.50px; height: 274.76px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c25"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 562.00px; height: 341.33px;"><img alt="" src="images/image8.png" style="width: 562.00px; height: 341.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p></td></tr><tr class="c18"><td class="c27" colspan="1" rowspan="1"><p class="c25"><span class="c3">Figure </span></p></td></tr></table><p class="c4"><span class="c3"></span></p><p class="c4"><span class="c3"></span></p><p class="c6"><span class="c3">After our evaluation, the final design for our monomer crosslinker is an X shaped DNA molecule with 40-50 base pairs for each arm, and a GC content of approximately 60-70%.</span></p><p class="c4"><span class="c3"></span></p><ol class="c17 lst-kix_gwvkwwabtx1g-0" start="3"><li class="c11 li-bullet-0"><h3 id="h.y1zg9hy39ru8" style="display:inline"><span class="c24">Diffusion</span></h3></li></ol><p class="c8 c20"><span class="c3">The hydrogel will be submerged in E. coli lysate to facilitate protein production. Given the hydrogel&rsquo;s porosity and the concentration gradient between the E. coli cellular machinery and the produced CA in the lysate versus within the hydrogel, both components may diffuse into and accumulate within the hydrogel mesh. Such accumulation may reduce production efficiency. In this study, we aim to evaluate the diffusive properties of CA, holoenzyme RNA polymerase, ribosomes, and CA mRNA to determine whether significant aggregation occurs within the hydrogel mesh.</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c5">To begin the process, we first assessed whether these biological molecules could diffuse into the hydrogel mesh based on their sizes. Using the number of 20 base pairs per arm for the X-DNA, the mesh size was calculated to be 13.6 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.google.ca/books/edition/Molecular_Biology_of_the_Cell/_NkpygAACAAJ?hl%3Den&amp;sa=D&amp;source=editors&amp;ust=1758169724589121&amp;usg=AOvVaw31gx1QX24TpmcRd7vkFs6_">8</a></span><span class="c5">, p.177]. The size of human CA II mRNA, determined from whole genome sequencing, was found to be 15.16 &micro;m [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://pubs.acs.org/doi/pdf/10.1021/acs.nanolett.8b02093?ref%3Darticle_openPDF&amp;sa=D&amp;source=editors&amp;ust=1758169724589631&amp;usg=AOvVaw0FMTqcwLtwv683m8wVKK-3">9</a></span><span class="c5">, </span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.ncbi.nlm.nih.gov/gene/760%23gene-expression&amp;sa=D&amp;source=editors&amp;ust=1758169724589809&amp;usg=AOvVaw07TZK1sFe1JGC7_F5yilWF">10</a></span><span class="c5">]. On the other hand, E. coli RNA polymerase has a molecular weight of 449 kDa [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC305883/&amp;sa=D&amp;source=editors&amp;ust=1758169724590188&amp;usg=AOvVaw31l-DgWRwwjDyXtPtz_XdS">11</a></span><span class="c5">], which corresponds to an approximate diameter of 12.98 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://fluidic.com/molecular-weight-to-hydrodynamic-radius-converter/&amp;sa=D&amp;source=editors&amp;ust=1758169724590504&amp;usg=AOvVaw24GqICkvbm9Po1rjJPEV2U">12</a></span><span class="c5">]. The E. coli ribosome has a diameter of 26 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://bionumbers.hms.harvard.edu/bionumber.aspx?s%3Dn%26v%3D8%26id%3D100121&amp;sa=D&amp;source=editors&amp;ust=1758169724590799&amp;usg=AOvVaw1hEqLWTCJgUtyLPmohXTOV">13</a></span><span class="c5">], and the size of human CA II protein is approximately 2.01 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://bionumbers.hms.harvard.edu/bionumber.aspx?s%3Dn%26v%3D1%26id%3D104773&amp;sa=D&amp;source=editors&amp;ust=1758169724591146&amp;usg=AOvVaw3befoK_AGgJWA5_auInI9O">14</a></span><span class="c5">]</span><span class="c3">.</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c5">From the size of the molecules, it is apparent that only RNA polymerase and CA II are small enough to potentially diffuse into the hydrogel mesh. However, experimental measurements indicate that the actual mesh size of X-DNA hydrogels is closer to 12.3 &plusmn; 1.4 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.nature.com/articles/nmat1741&amp;sa=D&amp;source=editors&amp;ust=1758169724592207&amp;usg=AOvVaw2FNJDRxbJgBcGUBMIVh_SU">15</a></span><span class="c5">]. Since larger molecules have a lower probability of diffusing through the mesh, and the size of E. coli RNA polymerase is very close to the upper limit of this measured range [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://pubs.acs.org/doi/pdf/10.1021/acs.macromol.9b00753?ref%3Darticle_openPDF&amp;sa=D&amp;source=editors&amp;ust=1758169724592840&amp;usg=AOvVaw18M-oA3pkYGYXxaG-CeRxh">16</a></span><span class="c3">], its diffusion into the network is likely minimal and can be considered negligible. Moreover, once RNA polymerase produces the mRNA templates for protein synthesis, these mRNA molecules can be reused multiple times in the lysate. Therefore, even if the concentration of RNA polymerase in the lysate decreases, the overall protein yield would not be significantly affected.</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c8 c20"><span class="c3">In contrast, due to its much smaller size, CA II can readily pass through the mesh and diffuse into the DNA polymer network. If CA II were to accumulate within the hydrogel, it could lower the yield in the lysate solution. To evaluate whether it would be beneficial for us to extract CA II from the gel, we modeled the diffusion of the protein into the polymer network by numerically solving the Fickian diffusion equation.</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c6"><span class="c12">System and Constraints:</span></p><p class="c7"><span class="c3">The p-gel is a rectangular prism as represented in Figure___, so CA can diffuse into the hydrogel through all six faces. Because there are only three unique face types, we model transport across those three faces and invoke symmetry by multiplying each contribution by two to obtain the total influx across each axis.</span></p><p class="c2"><span class="c3"></span></p><p class="c7"><span class="c3">Figure____</span></p><p class="c2"><span class="c3"></span></p><p class="c7"><span class="c5">Given the very high solubility of CA [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC3378272/?utm_source%3Dchatgpt.com&amp;sa=D&amp;source=editors&amp;ust=1758169724596677&amp;usg=AOvVaw2F-j5A_1B4JCHCbYV4hKVA">14</a></span><span class="c5">] and the long reaction time, we assume the lysate becomes well mixed. Thus, the CA concentration in the bulk lysate and at the hydrogel&ndash;lysate interface is spatially uniform. In addition, because CA has a long half-life [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://www.sciencedirect.com/science/article/pii/S0009250914002668&amp;sa=D&amp;source=editors&amp;ust=1758169724597439&amp;usg=AOvVaw2IvIB5X-dDW5ua1tH8kbZR">15</a></span><span class="c3">] relative to the experiment duration, we neglect enzymatic degradation over the entire run.</span></p><p class="c8 c9"><span class="c3"></span></p><p class="c8"><span class="c5">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c12">Model Building:</span></p><ul class="c17 lst-kix_j2yh5ht7lqr7-0 start"><li class="c8 c13 li-bullet-0"><span class="c3">Writing equation to describe the change in CA concentration in the gel: Simple 1D diffusion, no reaction</span></li><li class="c8 c13 li-bullet-0"><span class="c3">Discretizing the spatial dimension: central diff</span></li><li class="c8 c13 li-bullet-0"><span class="c3">Discretizing the time dimension:</span></li><li class="c8 c13 li-bullet-0"><span class="c3">Implemented the algorithm and solve the system numerically</span></li><li class="c8 c13 li-bullet-0"><span class="c3">Visualize the result</span></li></ul><p class="c8 c20"><span class="c5">Considering that the dimensions of our hydrogel are on the order of millimeters, which is much larger than the nanometer-scale size of CA, we assume that the CA will not diffuse through the entire hydrogel. Under this assumption, and given that there is no outlet, generation, or consumption of CA within the gel, the change in CA concentration inside the hydrogel can be modeled using Fick&rsquo;s second law:</span></p><p class="c8 c9 c19"><span class="c3"></span></p><p class="c8 c15"><img src="images/image1.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[1]</span></p><p class="c8 c9 c20"><span class="c3"></span></p><p class="c7"><span class="c5">Since our hydrogel has a uniform mesh size and the reaction occurs under constant temperature and viscosity, we assume that the diffusion coefficient of CA is constant. To estimate the effective diffusion coefficient of CA in the DNA polymer network, we used an online calculator [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://hydrogeldesign.org/the-model/mesh-transport/&amp;sa=D&amp;source=editors&amp;ust=1758169724604675&amp;usg=AOvVaw21P2hBVndLS1NdEYnYd8yM">16</a></span><span class="c5">]. This tool first applies the Stokes&ndash;Einstein equation to approximate the free diffusion coefficient of CA in solution.</span></p><p class="c8 c15"><img src="images/image2.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[2]</span></p><p class="c7"><span class="c3">The value for the effective diffusion coefficient was then adjusted for the hydrogel mesh using obstruction theory and free void volume theory.</span></p><p class="c7"><img src="images/image3.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c7"><span class="c3">[3]</span></p><p class="c7"><span class="c3">Given that CA is not significantly smaller than the mesh size, its diffusion is largely influenced by the solution-mediated transport through the mesh, with the effective diffusion coefficient primarily determined by obstruction effects.</span></p><p class="c7"><span class="c3">There are several important polymer properties that affect the ability of a solute to diffuse through the hydrogel mesh. The first is junction functionality, which describes the number of chains converging at each junction point. By taking the center of the X-DNA as the junction point, our p-gel has a junction functionality of 4. The degree of polymerization between junctions is defined as the number of repeating units between each junction. In our system, each sugar phosphate backbone in DNA is treated as a unit. Once the X-DNA strands assemble into a mesh, there are 40 base pairs between junctions, giving a degree of polymerization of 40. Since each nucleotide in DNA is linked to its neighbors through two phosphodiester bonds, the number of backbone bonds per unit is 2.</span></p><p class="c7"><span class="c5">Based on values reported by Um et al., hydrogels made with X-DNA monomers exhibit a monomer incorporation efficiency of approximately 83.5 &plusmn; 2.1%, corresponding to a swollen polymer volume fraction of 0.835. For the solute, the hydrodynamic radius is taken as the Stokes radius of carbonic anhydrase, 2.01nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://bionumbers.hms.harvard.edu/bionumber.aspx?id%3D104773%26ver%3D1%26trm%3Dcarbonic%2Banhydrase%2Bii%2Bradius%26org%3D&amp;sa=D&amp;source=editors&amp;ust=1758169724618957&amp;usg=AOvVaw0Y11gvOsnfBIhcu0n8c0b8">17</a></span><span class="c5">]. The polymer fibers, represented by DNA, have a radius of 1nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://bionumbers.hms.harvard.edu/bionumber.aspx?id%3D100666%26ver%3D6%26trm%3Ddna%2Bdiameter%26org%3D&amp;sa=D&amp;source=editors&amp;ust=1758169724619377&amp;usg=AOvVaw3NKQYgl2ASt05t2igfRjYo">18</a></span><span class="c5">]. The average bond length of the polymeric repeating unit in the hydrogel mesh was calculated by summing the average bond lengths for all atoms in a nucleotide and dividing by four, yielding a value of 0.137 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://arxiv.org/pdf/0708.1271&amp;sa=D&amp;source=editors&amp;ust=1758169724620029&amp;usg=AOvVaw2whR8MTiLnxOEGafW7K7jA">19</a></span><span class="c3">].</span></p><p class="c2"><span class="c3"></span></p><p class="c7"><span class="c5">Diffusion of CA into the mesh occurs during protein production, so the system temperature is assumed to be physiological temperature of 37 &deg;C. With these parameters defined, the Flory characteristic of the polymer can be calculated using the following equation:</span></p><p class="c21"><img src="images/image4.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[3]</span></p><p class="c7"><span class="c5">The persistence length of DNA is approximately 50 nm, while the effective bond length can be approximated as the distance between nucleotides, 0.34 nm [</span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://bionumbers.hms.harvard.edu/bionumber.aspx?id%3D103112%26ver%3D7%26trm%3Ddna%2Bpersistence%2Blength%26org%3D&amp;sa=D&amp;source=editors&amp;ust=1758169724625247&amp;usg=AOvVaw2AKSZdwpbgZPcfoHNX8GaM">20</a></span><span class="c5">, </span><span class="c1"><a class="c10" href="https://www.google.com/url?q=https://pmc.ncbi.nlm.nih.gov/articles/PMC11031260/&amp;sa=D&amp;source=editors&amp;ust=1758169724625462&amp;usg=AOvVaw2u5QNW16upmIxI_S7TD1BG">21</a></span><span class="c5">]. Using these parameters, the effective diffusion coefficient of carbonic anhydrase through the DNA hydrogel mesh was calculated to be </span><span class="c5 c22">103 &mu;m&sup2;/s</span><span class="c3">.</span></p><p class="c7"><span class="c3">To validate these results, we compared our free diffusion (D&#8320;) value and calculated mesh size with values reported in the literature. The mesh size obtained from the transport calculator was 12.9 nm, which falls within the range reported by Park et al. (12.3 &plusmn; 1.4 nm). Additionally, the free diffusion coefficient of carbonic anhydrase &beta; reported in the literature is 107 &mu;m&sup2;/s. This is slightly lower than the D&#8320; value of 127 &mu;m&sup2;/s predicted by mesh transport theory. The discrepancy may be attributed to structural differences, as the &beta; isoform is composed of two subunits and is therefore larger than the &alpha; isoform. Since the &alpha; isoform is smaller, it can move through the hydrogel mesh more easily, resulting in a higher diffusion coefficient.</span></p><p class="c7"><span class="c3">With the effective diffusion constant determined, we proceeded to solve the diffusion equation numerically to model the change in CA concentration within the hydrogel mesh. Since the hydrogel dimensions are effectively infinite relative to the molecular size of CA, we assume that the enzyme movement only occurs in 1 direction, which is into the gel. Furthermore, we focused on the total amount of CA diffused into the gel rather than its spatial distribution, which allowed us to reduce the system to a one-dimensional model for each of the axes as shown in Equation 4, 5, and 6.</span></p><p class="c21"><span class="c5">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="images/image5.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[4]</span></p><p class="c21"><span class="c5">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="images/image6.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[5]</span></p><p class="c21"><span class="c5">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><img src="images/image7.png"><span class="c3">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;[6]</span></p><p class="c7"><span class="c5">To solve the equation computationally, discretization methods were applied. The spatial dimension was discretized using the central difference approach, which combines the forward and backward difference approaches to achieve a more accurate approximation of derivatives (Equation 7). Here </span><span class="c5 c29">u</span><span class="c3">&nbsp;represent the concentration of CA.</span></p><p class="c21"><span class="c5">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp;</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 230.77px; height: 44.71px;"><img alt="{&quot;mathml&quot;:&quot;&lt;math style=\&quot;font-family:stix;font-size:16px;\&quot; xmlns=\&quot;http://www.w3.org/1998/Math/MathML\&quot;&gt;&lt;mstyle mathsize=\&quot;16px\&quot;&gt;&lt;mfrac&gt;&lt;mrow&gt;&lt;msup&gt;&lt;mo&gt;&amp;#x2202;&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo&gt;&amp;#x2202;&lt;/mo&gt;&lt;msup&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;mfenced&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;/mfenced&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mfrac&gt;&lt;mrow&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;mo&gt;-&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;msup&gt;&lt;msub&gt;&lt;mi&gt;u&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mo&gt;-&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;mrow&gt;&lt;mo&gt;&amp;#x2206;&lt;/mo&gt;&lt;msup&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/mfrac&gt;&lt;/mstyle&gt;&lt;/math&gt;&quot;,&quot;truncated&quot;:false}" src="images/image11.png" style="width: 230.77px; height: 44.71px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title="fraction numerator partial differential squared u over denominator partial differential x squared end fraction open parentheses u subscript i to the power of j close parentheses equals fraction numerator u subscript i plus 1 end subscript to the power of j minus 2 u subscript i to the power of j plus u subscript i minus 1 end subscript to the power of j over denominator increment x squared end fraction"></span><span class="c3">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; [7]</span></p><p class="c7"><span class="c3">For the temporal dimension, the system was integrated using the classical fourth-order Runge-Kutta (RK4) method. Compared to lower order RK, the fourth order approximation is closer to the actual value. RK4 estimates the slope of the solution of the differential equation four times with the next slope being the estimate of the value of the function midway through the interval. This process is carried out for each time step and combined into a weighted average, effectively canceling error terms up to order h&#8308;, similar to a fourth order Taylor Expansion. This yields a local truncation error of O(h&#8309;) and a global error of O(h&#8308;), providing significantly higher accuracy and stability than Euler or other lower-order methods for non-stiff problems. The discretized form of the RK4 method is presented in Equation 8.</span></p><p class="c2"><span class="c3"></span></p><p class="c7"><span class="c3">The system was solved in python and the full code can be found in the Supplementary Information section. Since diffusion systems are constrained by stability requirements, the time step was chosen to scale with &Delta;x&sup2; / D, ensuring that the numerical solution remains accurate and stable. The total CA concentration within the mesh after ____ was found to be ______, as demonstrated in Figure ________.</span></p><p class="c8 c9"><span class="c28"></span></p><p class="c8 c9"><span class="c30"></span></p></body></html>